---
title: "News"
output: html_document
---

```{r, include=FALSE}
# required to use 'Font Awesome' (FA) icons (Website, Twitter, GitHub, etc)
htmltools::tagList(rmarkdown::html_dependency_font_awesome())
```


```{r, echo=FALSE}
# Instructions:
# - Add a new row to the dataframe tbl_news for each entry. The output order is defined by the date from newest to oldest.
# - To add custom linebreaks in text sections, use the HTML tag <br> for linebreaks.
# - To add 'Font Awesome' (FA) icons, you can use the following HTML code and suggested icons:
#     Website: <i class='fa fa-desktop'></i>
#     Twitter: <i class='fa fa-twitter'></i>
#     GitHub:  <i class='fa fa-github'></i>
#     Google Scholar: <i class='fa fa-newspaper-o'></i>
# - For additional icons (e.g. LinkedIn), see the following website:
# https://www.w3schools.com/icons/fontawesome_icons_webapp.asp

tbl_news <- data.frame(matrix(vector(), 0, 3, 
                              dimnames = list(c(), c("date","title","news"))), 
                       stringsAsFactors = FALSE)

# tbl_news[nrow(tbl_news) + 1,] <- list("1.6.2018","Test", "Test<br> Test<br><br> Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test")
# tbl_news[nrow(tbl_news) + 1,] <- list("31.5.2018","Test", "Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test")
# tbl_news[nrow(tbl_news) + 1,] <- list("30.5.2018","Test", "Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test")
# tbl_news[nrow(tbl_news) + 1,] <- list("29.5.2018","Test", "Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test")
tbl_news[nrow(tbl_news) + 1,] <- list("01.06.2018","New website is published!","After a major collaborative effort from everyone in the lab, the new github-based lab website is finished.")
tbl_news[nrow(tbl_news) + 1,] <- list("18.06.2018","New bioRxiv preprint!","Weber LM, Nowicka G, Soneson C and Robinson MD: [diffcyt: Differential discovery in high-dimensional cytometry via high-resolution clustering](https://doi.org/10.1101/349738)")
tbl_news[nrow(tbl_news) + 1,] <- list("27.06.2018","New bioRxiv preprint!","Parker HR, Orjuela S, Martinho Oliveira A, Cereatti F, Sauter M, Heinrich H, Tanzi G, Weber A, Komminoth P, Vavricka S, Albanese L, Buffoli F, Robinson MD, Marra G: [The proto CpG island methylator phenotype of sessile serrated adenoma/polyps](https://doi.org/10.1101/357178)")
tbl_news[nrow(tbl_news) + 1,] <- list("26.07.2018","New F1000Research publication!","Duò A, Robinson MD and Soneson C: [A systematic performance evaluation of clustering methods for single-cell RNA-seq data](https://doi.org/10.12688/f1000research.15666.1)")
tbl_news[nrow(tbl_news) + 1,] <- list("28.07.2018","New bioRxiv preprint!","Soneson C, Love MI, Patro R, Hussain S, Malhotra D and Robinson MD: [A junction coverage compatibility score to quantify the reliability of transcript abundance estimates and annotation catalogs](https://doi.org/10.1101/378539)")
tbl_news[nrow(tbl_news) + 1,] <- list("06.08.2018","New BMC Genomics publication!","Bick JT, Flöter VL, Robinson MD, Bauersachs S, Ulbrich SE.
[Small RNA-seq analysis of single porcine blastocysts revealed that maternal estradiol-17beta exposure does not affect miRNA isoform (isomiR) expression](https://doi.org/10.1186/s12864-018-4954-9)")
tbl_news[nrow(tbl_news) + 1,] <- list("17.10.2018","New PeerJ preprint!","Van den Berge K, Hembach K, Soneson C, Tiberi S, Clement L, Love MI, Patro R, Robinson MD.
[RNA sequencing data: hitchhiker’s guide to expression analysis](https://doi.org/10.7287/peerj.preprints.27283v1)")
tbl_news[nrow(tbl_news) + 1,] <- list("21.01.2019","New Life Science Alliance publication!","Soneson C, Love MI, Patro R, Hussain S, Malhotra D and Robinson MD: [A junction coverage compatibility score to quantify the reliability of transcript abundance estimates and annotation catalogs](http://www.life-science-alliance.org/content/2/1/e201800175)")
tbl_news[nrow(tbl_news) + 1,] <- list("12.03.2019","New bioRxiv preprint!","Soneson C, Yao Y, Bratus-Neuenschwander A, Patrignani A, Robinson MC and Hussain S: [A comprehensive examination of Nanopore native RNA sequencing for characterization of complex transcriptomes](https://doi.org/10.1101/574525)")
tbl_news[nrow(tbl_news) + 1,] <- list("13.03.2019","New bioRxiv preprint!","Orjuela S, Huang R, Hembach K, Robinson MD, Soneson C: [ARMOR: an Automated Reproducible MOdular workflow for preprocessing and differential analysis of RNA-seq data](https://doi.org/10.1101/575951)")
tbl_news[nrow(tbl_news) + 1,] <- list("17.05.2019","New G3 publication!","Orjuela S, Huang R, Hembach K, Robinson MD, Soneson C: [ARMOR: an Automated Reproducible MOdular workflow for preprocessing and differential analysis of RNA-seq data](https://www.g3journal.org/content/early/2019/05/14/g3.119.400185.long)")
tbl_news[nrow(tbl_news) + 1,] <- list("25.06.2019","A little flurry of publications ..","D’Agati G, Cabello EM, Frontzek K, Rushing EJ, Klemm R, Robinson MD, White RM, Mosimann C, Burger A: [Active receptor tyrosine kinases, but not Brachyury, are sufficient to trigger chordoma in zebrafish.](https://dmm.biologists.org/content/early/2019/06/19/dmm.039545.long) 

Weber LM, Saelens W, Cannoodt R, Soneson C, Hapfelmeier A, Gardner PP, Boulesteix AL, Saeys Y, Robinson MD: [Essential guidelines for computational method benchmarking.](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1738-8)

Weber LM, Nowicka M, Soneson C, Robinson MD: [diffcyt: Differential discovery in high-dimensional cytometry via high-resolution clustering](https://www.nature.com/articles/s42003-019-0415-5)")
tbl_news[nrow(tbl_news) + 1,] <- list("26.07.2019","New bioRxiv preprint!","Crowell HL, Soneson C, Germain P-L, Calini D, Collin L, Raposo C, Malhotra D, Robinson MD:  [On the discovery of population-specific state transitions from multi-sample multi-condition single-cell RNA sequencing data](https://doi.org/10.1101/713412")
tbl_news[nrow(tbl_news) + 1,] <- list("02.08.2019","New Nature Communications publication!","Soneson C, Yao Y, Bratus-Neuenschwander A, Patrignani A, Robinson MC and Hussain S: [A comprehensive examination of Nanopore native RNA sequencing for characterization of complex transcriptomes](https://www.nature.com/articles/s41467-019-11272-z)")
```

[//]: # Add Mark's twitter feed
<a class="twitter-timeline" data-height="400" href="https://twitter.com/markrobinsonca">Tweets by markrobinsonca</a> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>

```{r, echo=FALSE, message=FALSE, results="asis", warning=FALSE}
# R code to output HTML code

library(knitr)
library(kableExtra)
library(dplyr)
library(tidyr)

# Separate title and news into different rows
tbl_news <- tbl_news %>% gather(type,text,-date) %>%
  mutate(date=as.Date(date,"%d.%m.%Y")) %>%
  # Sort by date (from newest to oldest) and type(1.title, 2.news)
  arrange(desc(date),desc(type)) %>%
  # Convert the date into German format and remove date in news rows
  mutate(date=ifelse("news"==type,"",format(date, "%d.%m.%Y"))) %>%
  select(date,text)
# remove column names
colnames(tbl_news) <- NULL

# output HTML code using 'kableExtra' package
# note: requires 'results="asis"' in code chunk header, and 'print' command at the end
for (i in seq(1,nrow(tbl_news),by=2)) {
  tmp <- tbl_news[c(i,i+1),]
  rownames(tmp) <- NULL # Rownames have to be removed or are plotted as separate column
  kable(tmp, escape = FALSE) %>% 
    kable_styling(bootstrap_options = "condensed", full_width = FALSE) %>% 
    column_spec(column = 1, width = "3cm") %>% 
    column_spec(column = 2, width = "20cm") %>% 
    row_spec(row = 1, bold = TRUE, font_size = 18) %>% 
    collapse_rows(columns = 1, valign = "top") %>% 
    # hack to remove row borders between table cells (by adding additional options to HTML code for table cells)
    gsub('<td style="', '<td style="border:none;', .) %>% 
    print
}
```


