<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>People</title>

<script src="site_libs/header-attrs-2.8/header-attrs.js"></script>
<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/flatly.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>








<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-inverse  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Statistical Bioinformatics @SIB @UZH</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="research.html">
    <span class="fas fa-area-chart"></span>
     
    research
  </a>
</li>
<li>
  <a href="publications.html">
    <span class="fas fa-file-text-o"></span>
     
    publications
  </a>
</li>
<li>
  <a href="tools.html">
    <span class="fas fa-code"></span>
     
    resources
  </a>
</li>
<li>
  <a href="people.html">
    <span class="fas fa-user"></span>
     
    people
  </a>
</li>
<li>
  <a href="teaching.html">
    <span class="fas fa-graduation-cap"></span>
     
    teaching
  </a>
</li>
<li>
  <a href="more_info.html">
    <span class="fas fa-info-circle"></span>
     
    more info
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">People</h1>

</div>


<table class="table table-condensed" style="width: auto !important; margin-left: auto; margin-right: auto;">
<tbody>
<tr>
<td style="border:none;text-align:left;width: 3cm; font-weight: bold;font-size: 18px;vertical-align: top !important;" rowspan="4">
<img src="img/thumbnail/mark.jpg" />
</td>
<td style="border:none;text-align:left;width: 20cm; font-weight: bold;font-size: 18px;">
Mark D Robinson
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
<i class="fa fa-twitter"></i> Twitter <a href="https://twitter.com/markrobinsonca">@markrobinsonca</a><br><i class="fa fa-github"></i> GitHub <a href="https://github.com/markrobinsonuzh">markrobinsonuzh</a><br><i class="fa fa-newspaper-o"></i> <a href="https://scholar.google.ch/citations?user=XPfrRQEAAAAJ">Google Scholar</a>
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
My research interests are diverse, but more-or-less encompass the general application of statistical methods and data science to experimental data with biological applications. Often, this is within the context of genomics data types, but we are interested in methodological challenges and robust solutions in data, generally. We also try to be modern scientists, with a focus on reproducibility (repos for code) and open science (preprints).
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
PhD in Medical Biology (2008), University of Melbourne (Walter and Eliza Hall Institute)<br>MSc in Statistics (2001), University of British Columbia<br>BSc in Applied Mathematics and Statistics (1999), University of Guelph
</td>
</tr>
</tbody>
</table>
<table class="table table-condensed" style="width: auto !important; margin-left: auto; margin-right: auto;">
<tbody>
<tr>
<td style="border:none;text-align:left;width: 3cm; font-weight: bold;font-size: 18px;vertical-align: top !important;" rowspan="4">
<img src="img/thumbnail/simone.png" />
</td>
<td style="border:none;text-align:left;width: 20cm; font-weight: bold;font-size: 18px;">
Simone Tiberi
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
<i class="fa fa-desktop"></i> <a href="https://sites.google.com/view/simonetiberi">Website</a><br><i class="fa fa-twitter"></i> Twitter <a href="https://twitter.com/tiberi_simone">@tiberi_simone</a><br><i class="fa fa-github"></i> GitHub <a href="https://github.com/SimoneTiberi">SimoneTiberi</a><br><i class="fa fa-newspaper-o"></i> <a href="https://scholar.google.com/citations?hl=en&amp;user=58nWpq8AAAAJ&amp;view_op=list_works&amp;gmla=AJsN-F4oKYjm4oTFUio_EFmbXkpF5n0dgQOv-C9pfBGgyBq4ffqGAeJ2R6HwC1VNmMuSkj_NTMa-BQ5H7RyLxqZ6hr5yUsTej9kccy4vFeVjIAIgGy1uIH8">Google Scholar</a>
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
I am a Postdoc working on the development of cutting-edge statistical methods in bioinformatics, mostly for bulk and single-cell RNA-seq data.Previously, I was a PhD student at the Department of Statistics at the University of Warwick, where I worked on stochastic Bayesian hierarchical models to investigate transcription in single cells.In general terms, my interests are broad and lie in the development of statistical methods for applications in biology.
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
PhD in Statistics (2017), The University of Warwick<br>MSc in Statistics (2012), The University of Padua<br>BSc in Statistics (2010), Sapienza University of Rome
</td>
</tr>
</tbody>
</table>
<table class="table table-condensed" style="width: auto !important; margin-left: auto; margin-right: auto;">
<tbody>
<tr>
<td style="border:none;text-align:left;width: 3cm; font-weight: bold;font-size: 18px;vertical-align: top !important;" rowspan="4">
<img src="img/thumbnail/izaskun.jpg" />
</td>
<td style="border:none;text-align:left;width: 20cm; font-weight: bold;font-size: 18px;">
Izaskun Mallona
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
<i class="fa fa-desktop"></i> <a href="http://imallona.bitbucket.io">Website</a><br><i class="fa fa-newspaper-o"></i> <a href="https://scholar.google.ch/citations?user=kMfxCIYAAAAJ">Google Scholar</a>
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
I am mainly interested in gene expression modulation, as it lays the ground for any cell fate. The questions which excite my curiosity the most are: How are gene expression programs coded? and maintained? how do they adapt during development? and disease? To mine the regulation metastructure genome-wide I’m fond of integrating gene expression and epigenomic data, and to develop methods for doing so.
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
BSc in Computer Engineering (2014), Open University of Catalonia<br>PhD (2012), Technical University of Cartagena<br>MSc in Agriculture Engineering (2008), Technical University of Cartagena<br>MSc in Biology (2007), University of Murcia
</td>
</tr>
</tbody>
</table>
<table class="table table-condensed" style="width: auto !important; margin-left: auto; margin-right: auto;">
<tbody>
<tr>
<td style="border:none;text-align:left;width: 3cm; font-weight: bold;font-size: 18px;vertical-align: top !important;" rowspan="4">
<img src="img/thumbnail/pierreluc.jpg" />
</td>
<td style="border:none;text-align:left;width: 20cm; font-weight: bold;font-size: 18px;">
Pierre-Luc Germain
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
<i class="fa fa-github"></i> GitHub <a href="https://github.com/plger">plger</a><br><i class="fa fa-newspaper-o"></i> <a href="https://scholar.google.com/citations?user=788GnysAAAAJ">Google Scholar</a>
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
I gradually shifted from the humanities to bioinformatics towards the end of my PhD, worked a lot in the past on linking chromatin alterations to gene expression, and currently work mostly on the analysis of short RNA-seq and single-cell RNA-seq, as well as pre- and post-transcriptional regulation of gene expression especially in the context of neurodevelopment. I have a shared position with the ETH’s D-HEST Institute for Neurosciences.
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
PhD in Philosophy of science (2014), SEMM and University of Milan<br>MA in Philosophy (2009), University of Montreal<br>BA in Philosophy (2007)
</td>
</tr>
</tbody>
</table>
<table class="table table-condensed" style="width: auto !important; margin-left: auto; margin-right: auto;">
<tbody>
<tr>
<td style="border:none;text-align:left;width: 3cm; font-weight: bold;font-size: 18px;vertical-align: top !important;" rowspan="4">
<img src="img/thumbnail/imran.jpg" />
</td>
<td style="border:none;text-align:left;width: 20cm; font-weight: bold;font-size: 18px;">
Imran Fanaswala
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
<i class="fa fa-twitter"></i> Twitter <a href="https://twitter.com/IFanaswala">@IFanaswala</a><br>
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
Imran Fanaswala has worked in academia and industry for over a decade; his passion lies in value-driven research (i.e. addressing unmet needs through research as well as bringing ideas to market). His work has spanned various fields (robotics, phylogenetics, DNA-assembly, neuroscience) but often within the machine-learning/AI context (and more recently, “ModelOps”). He strives for readable code, succint prose, and meaningful research.
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
MSc in Computational Biology (2014), ETH Zürich (thesis at RIKEN) <br>BSc in Computer Science (2006), American University of Sharjah <br>
</td>
</tr>
</tbody>
</table>
<table class="table table-condensed" style="width: auto !important; margin-left: auto; margin-right: auto;">
<tbody>
<tr>
<td style="border:none;text-align:left;width: 3cm; font-weight: bold;font-size: 18px;vertical-align: top !important;" rowspan="3">
<img src="img/thumbnail/kathi.jpg" />
</td>
<td style="border:none;text-align:left;width: 20cm; font-weight: bold;font-size: 18px;">
Katharina Hembach
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
I have a background in bioinformatics and I am a shared PhD student with the <a href="http://www.imls.uzh.ch/en/research/polymenidou.html">Polymenidou lab</a>. They focus on understanding the molecular mechanisms of neurodegenerative diseases such as Amyotrophic Lateral Sclerosis (ALS) and Frontotemporal Lobar Degeneration (FTLD). I help with the analysis and interpretation of different sequencing data, such as RNA-seq, CLIP-seq, and iCLIP. My main project is the development of a pipeline for the identification and quantification of alternative splicing events, specifically microexons and cryptic exons, from RNA-seq data.
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
MSc in Bioinformatics (2015), LMU and TU Munich<br>BSc in Bioinformatics (2012), LMU and TU Munich
</td>
</tr>
</tbody>
</table>
<table class="table table-condensed" style="width: auto !important; margin-left: auto; margin-right: auto;">
<tbody>
<tr>
<td style="border:none;text-align:left;width: 3cm; font-weight: bold;font-size: 18px;vertical-align: top !important;" rowspan="4">
<img src="img/thumbnail/stephan.jpg" />
</td>
<td style="border:none;text-align:left;width: 20cm; font-weight: bold;font-size: 18px;">
Stephan Schmeing
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
<i class="fa fa-twitter"></i> Twitter <a href="https://twitter.com/StephanSchmeing">@StephanSchmeing</a><br><i class="fa fa-github"></i> GitHub <a href="https://github.com/schmeing">schmeing</a><br>
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
As a PhD student coming from a particle physics background I am currently working on a more realistic Illumina sequence simulator called ReSequenceR. It is able to create a profile from a bam file that is then used to simulate paired-read fastq files. Furthermore I will work on an assembly and population genomics of the Indo-Pacific bottlenose dolphin (tursiops aduncus).
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
MSc in Astro, Particle and Nuclear physics (2014), TU Munich<br>BSc in Physics (2011), TU Munich
</td>
</tr>
</tbody>
</table>
<table class="table table-condensed" style="width: auto !important; margin-left: auto; margin-right: auto;">
<tbody>
<tr>
<td style="border:none;text-align:left;width: 3cm; font-weight: bold;font-size: 18px;vertical-align: top !important;" rowspan="3">
<img src="img/thumbnail/vladimir.jpg" />
</td>
<td style="border:none;text-align:left;width: 20cm; font-weight: bold;font-size: 18px;">
Vladimir Barbosa C. de Souza
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
I am a PhD candidate in Bioinformatics, and I am working with PacBio data (third-generation sequencing). As there is just few documentation about the data, one of the focus of my research is to figure out their features, mainly about quality scores. At the moment I am working on a project to develop a new fast and accurate method to create CCS reads (circular consensus sequences).
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
MSc in Applied Statistics (2015), University of Vicosa<br>BSc in Biology, specialization Genetics (2013), University of Vicosa
</td>
</tr>
</tbody>
</table>
<table class="table table-condensed" style="width: auto !important; margin-left: auto; margin-right: auto;">
<tbody>
<tr>
<td style="border:none;text-align:left;width: 3cm; font-weight: bold;font-size: 18px;vertical-align: top !important;" rowspan="4">
<img src="img/thumbnail/stefan.jpg" />
</td>
<td style="border:none;text-align:left;width: 20cm; font-weight: bold;font-size: 18px;">
Stefan Milosavljevic
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
<i class="fa fa-linkedin-square"></i> <a href="https://www.linkedin.com/in/stefan-milosavljevic-07361b8a/">LinkedIn</a> <br><i class="fa fa-github"></i> GitHub <a href="https://github.com/supermaxiste">supermaxiste</a><br>
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
I have a shared PhD position with the <a href="http://www.ieu.uzh.ch/en/research/evolbiol/ecogenomics.html">Shimizu lab</a>. I’ve got a background in biology and bioinformatics and I’m currently working on the rapid evolution of a plant species called <em>Arabidopsis</em> <em>kamchatica</em>. My project focuses on epigenetic modifications over short timescales and my general aim is to assess the impact and the importance of DNA methylation in evolution over short periods (rapid evolution).
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
MSc in Computational Biology and Bioinformatics (2017), ETH Zürich and University of Zürich<br>BSc in Biology (2015), Université de Lausanne
</td>
</tr>
</tbody>
</table>
<table class="table table-condensed" style="width: auto !important; margin-left: auto; margin-right: auto;">
<tbody>
<tr>
<td style="border:none;text-align:left;width: 3cm; font-weight: bold;font-size: 18px;vertical-align: top !important;" rowspan="4">
<img src="img/thumbnail/helena.png" />
</td>
<td style="border:none;text-align:left;width: 20cm; font-weight: bold;font-size: 18px;">
Helena Lucia Crowell
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
<i class="fa fa-twitter"></i> Twitter <a href="https://twitter.com/CrowellHL">@CrowellHL</a><br>
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
In collaboration with the <a href="http://www.bodenmillerlab.com">Bodenmiller lab</a> lab, I have worked on correcting for spillover in CyTOF data. The preprint to this project is available on <a href="https://doi.org/10.1101/185744">bioRxiv</a> and has been published in <a href="https://doi.org/10.1016/j.cels.2018.02.010">cell systems</a>. Alongside our compensation approach, I have made Finck et al.’s <a href="https://doi.org/10.1002/cyto.a.22271">bead-based normalization</a> and Zunder et al.’s <a href="https://doi.org/10.1038/nprot.2015.020">single-cell deconvolution</a> available in <a href="https://bioconductor.org/packages/release/bioc/html/CATALYST.html">CATALYST</a> (<strong>C</strong>ytometry d<strong>AT</strong>a an<strong>ALYS</strong>is <strong>T</strong>ools): a <a href="https://bioconductor.org">Bioconductor</a> package providing tools for preprocessing and analysis of cytometry data; and have further integrated M. Nowicka’s <a href="https://f1000research.com/articles/6-748/v2">CyTOF workflow</a> for differential analysis into the package. As a PhD student, I am currently working on extending this framework for differential discovery to scRNA-seq data with complex experimental designs.
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
MSc in Computational Biology &amp; Bioinformatics (2018), ETH Zürich<br>BSc in Biochemistry (2015), University of Heidelberg
</td>
</tr>
</tbody>
</table>
<table class="table table-condensed" style="width: auto !important; margin-left: auto; margin-right: auto;">
<tbody>
<tr>
<td style="border:none;text-align:left;width: 3cm; font-weight: bold;font-size: 18px;vertical-align: top !important;" rowspan="3">
<img src="img/thumbnail/almut.jpg" />
</td>
<td style="border:none;text-align:left;width: 20cm; font-weight: bold;font-size: 18px;">
Almut Lütge
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
I’m a PhD student with a background in Molecular Biotechnology, but already during my master I focused on Bioinformatics. I’m interested in all kinds of genomic analysis, but mainly worked with transcriptomic data, so far. My PhD project is about methods in single cell RNAseq and part of a Sinergia collaboration on defining the identity and differentiation pathways of the immune-stimulating fibroblastic tumor stroma.
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
MSc in Molecular Biotechnology (2018), University of Heidelberg<br>BSc Molecular Biotechnology (2013), University of Heidelberg
</td>
</tr>
</tbody>
</table>
<table class="table table-condensed" style="width: auto !important; margin-left: auto; margin-right: auto;">
<tbody>
<tr>
<td style="border:none;text-align:left;width: 3cm; font-weight: bold;font-size: 18px;vertical-align: top !important;" rowspan="4">
<img src="img/thumbnail/Anthony.png" />
</td>
<td style="border:none;text-align:left;width: 20cm; font-weight: bold;font-size: 18px;">
Anthony Sonrel
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
<i class="fa fa-linkedin-square"></i> <a href="https://www.linkedin.com/in/anthony-sonrel-612505114/">LinkedIn</a> <br>
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
I am a PhD student that joined the Robinson Lab as part of the Life Science Zürich Graduate School. Through my past projects in academia and industry I worked on various topics such as NGS, GWAS summary statistics, epigenetic analyses (ChIP/ATAC-seq) and deep neural networks. In my current project, I’m evaluating and developing tools (mainly R packages) for single-cell RNA sequencing datasets. More specifically, I’m interested in developing new methodologies and new effective tools.
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
MSc in Molecular Life Sciences - Bioinformatics (2018), University of Lausanne<br>BSc of Science in Biology (2015), University of Lausanne
</td>
</tr>
</tbody>
</table>
<table class="table table-condensed" style="width: auto !important; margin-left: auto; margin-right: auto;">
<tbody>
<tr>
<td style="border:none;text-align:left;width: 3cm; font-weight: bold;font-size: 18px;vertical-align: top !important;" rowspan="3">
<img src="img/thumbnail/Reto.jpg" />
</td>
<td style="border:none;text-align:left;width: 20cm; font-weight: bold;font-size: 18px;">
Reto Gerber
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
I am a Research Assistant in the Robinson lab. In my lastest project I developed a method for differential abundance analysis in cytometry where a covariate is subject to censoring (Preprint on <a href="https://doi.org/10.1101/2020.11.09.374447">bioRxiv</a>). Currently I develop a R shiny web app helping in exploration and visualization of open access publication profiles of individual authors. In past projects I worked among other things on docking of biomolecules and signalling networks in cells, before turning more to the statistical aspect of bioinformatics.
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
MSc of Science in Computational Biology and Bioinformatics (2020), ETH, University of Zürich<br>BSc of Science in Biology (2017), University of Zürich
</td>
</tr>
</tbody>
</table>
<table class="table table-condensed" style="width: auto !important; margin-left: auto; margin-right: auto;">
<tbody>
<tr>
<td style="border:none;text-align:left;width: 3cm; font-weight: bold;font-size: 18px;vertical-align: top !important;" rowspan="3">
<img src="img/thumbnail/elyas.jpeg" />
</td>
<td style="border:none;text-align:left;width: 20cm; font-weight: bold;font-size: 18px;">
Elyas Heidari
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
I am a Master student of Computational Biology &amp; Bioinformatics at ETH university. I am interested in developing analysis pipelines for single-cell multi-omics data. Currently, I am exploring single-cell B/T-cell receptor data to understand the diversity in immune repertoire and phenotypes. Prior to joining Robinson lab, I worked on developing a number of machine/deep learning tools for biomedical data. I have developed <a href="https://baio-lab.github.io/muvis/index.html">MUVIS</a> for general medical data analysis and <a href="http://seqlearner.readthedocs.io/">SeqLearner</a> for semi-supervised learning on protein sequences.
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
BSc of Science in Applied Mathematics (2019), Sharif University of Technology<br>BSc of Science in Computer Engineering (2019), Sharif University of Technology<br>
</td>
</tr>
</tbody>
</table>
<table class="table table-condensed" style="width: auto !important; margin-left: auto; margin-right: auto;">
<tbody>
<tr>
<td style="border:none;text-align:left;width: 3cm; font-weight: bold;font-size: 18px;vertical-align: top !important;" rowspan="3">
<img src="img/thumbnail/emanuel.jpg" />
</td>
<td style="border:none;text-align:left;width: 20cm; font-weight: bold;font-size: 18px;">
Emanuel Sonder
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
As a Master student in the Computational Biology and Bioinformatics joint program of ETH and UZH, I am currently working on my master thesis. The project is about the integration of multi-omics single-cell data to describe patterns of epigenetic heterogeneity, such as partially methylated domains (PMDs) and their influence on gene expression. In previous projects I have worked on different topics, amongst others, on a pan-cancer analysis of somatic mutations and DNA methylation alterations in the epigenetic machinery. I have a broad interest in the integration and statistical analysis of (single cell) data from different biological modalities.
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
BSc in Biology (2018), ETH Zürich
</td>
</tr>
</tbody>
</table>
<table class="table table-condensed" style="width: auto !important; margin-left: auto; margin-right: auto;">
<tbody>
<tr>
<td style="border:none;text-align:left;width: 3cm; font-weight: bold;font-size: 18px;vertical-align: top !important;" rowspan="4">
<img src="img/thumbnail/joel.jpg" />
</td>
<td style="border:none;text-align:left;width: 20cm; font-weight: bold;font-size: 18px;">
Joël Meili
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
<i class="fa fa-github"></i> GitHub <a href="https://github.com/joelmeili">joelmeili</a><br>
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
I am a master’s student in the Biostatistics program at UZH. Currently I am working on my master’s thesis which aims to propose a novel method that compares RNA velocities to identify cell clusters and genes within the clusters with different RNA velocities. The main aim is to compare experimental conditions (e.g. healthy vs. disease). However, the framework shoud also allow for comparisions of cell clusters from individual samples (e.g. B cells vs. T cells).
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
BSc in Engineering and Management (2019), Zurich University of Applied Sciences
</td>
</tr>
</tbody>
</table>
<table class="table table-condensed" style="width: auto !important; margin-left: auto; margin-right: auto;">
<tbody>
<tr>
<td style="border:none;text-align:left;width: 3cm; font-weight: bold;font-size: 18px;vertical-align: top !important;" rowspan="4">
<img src="img/thumbnail/aalajami.jpg" />
</td>
<td style="border:none;text-align:left;width: 20cm; font-weight: bold;font-size: 18px;">
Ahmad Al Ajami
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
<i class="fa fa-linkedin-square"></i> <a href="https://www.linkedin.com/in/ahmad-al-ajami-2ba577168/">LinkedIn</a> <br>
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
I am a master’s student in the Computational Biology and Bioinformatics joint program of ETHZ and UZH. I am currently working on my master’s thesis where I am continuing work that I conducted during my internship at Roche. The project is about implementation and extension of existing reference-based methods for cell annotation in single-cell RNA sequencing. Throughout the years, I have developed particular interest in immunology, data analysis and integration of single-cell data from different biological modalities.
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
BSc in Genetics and Bioinformatics (2018), Bahcesehir University
</td>
</tr>
</tbody>
</table>
<table class="table table-condensed" style="width: auto !important; margin-left: auto; margin-right: auto;">
<tbody>
<tr>
<td style="border:none;text-align:left;width: 3cm; font-weight: bold;font-size: 18px;vertical-align: top !important;" rowspan="4">
<img src="img/thumbnail/dominique.jpg" />
</td>
<td style="border:none;text-align:left;width: 20cm; font-weight: bold;font-size: 18px;">
Dominique Paul
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
<i class="fa fa-desktop"></i> <a href="http://www.thisiscrispin.com/">Website</a><br><i class="fa fa-twitter"></i> Twitter <a href="https://twitter.com/DominiqueCAPaul">@dominiquecapaul</a><br><i class="fa fa-github"></i> GitHub <a href="https://github.com/DominiquePaul">dominiquepaul</a><br>
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
I am a Research Assistant at the Robinson lab. Currently I am working on a project with the lab team and researchers from the the university hospital to understand the effects of a new treatment method on systemic scleroris. I mainly work with and am interested in methods for single-cell RNA-seq data
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
Master student in Statistics, ETH Zurich
</td>
</tr>
</tbody>
</table>
<table class="table table-condensed" style="width: auto !important; margin-left: auto; margin-right: auto;">
<tbody>
<tr>
<td style="border:none;text-align:left;width: 3cm; font-weight: bold;font-size: 18px;vertical-align: top !important;" rowspan="3">
<img src="img/thumbnail/cpy.jpg" />
</td>
<td style="border:none;text-align:left;width: 20cm; font-weight: bold;font-size: 18px;">
Peiying Cai
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
As a Master student in the Biostatistics program of UZH, I am currently working on my master thesis. The project aims to propose a novel methodology for differential analyses from spatial transcriptomics data, that identifies spatially variable (SV) genes as well as the specific areas of the tissue affected by SV. Furthermore, the method will also investigate how spatial expression patterns vary between conditions (e.g., healthy vs. disease). Then, ideally the framework could be extended to do the same things with cell types.
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
BSc of Science in Statistics (2019), Xiamen University
</td>
</tr>
</tbody>
</table>
<div id="alumni" class="section level2">
<h2>Alumni</h2>
<ul>
<li>Stephany Orjuela (PhD student; author of <a href="https://github.com/markrobinsonuzh/DAMEfinder">DAMEfinder</a>; co-author of <a href="https://github.com/csoneson/ARMOR">ARMOR</a>; now Scientific software developer at <a href="https://oncobit.com/">Oncobit</a>)</li>
<li>Will Macnair (Postdoc; author of <a href="https://github.com/wmacnair/SampleQC/">SampleQC</a>; now Data Scientist at <a href="https://www.roche.ch/">Hoffman La Roche</a>)</li>
<li>Ruizhu (Fiona) Huang (PhD student; author of <a href="https://github.com/fionarhuang/treeclimbR">treeclimbR</a>, <a href="https://www.bioconductor.org/packages/release/bioc/html/TreeSummarizedExperiment.html">TreeSummarizeExperiment</a>, co-author of <a href="https://github.com/csoneson/ARMOR">ARMOR</a>; now Biostatistician at Cardio-CARE)</li>
<li><a href="http://merlin.up.poznan.pl/kmmis2/index.php?id=158&amp;id2=105">Joanna Zyprych-Walczak</a> (visiting professor)</li>
<li><a href="http://csoneson.github.io/">Charlotte Soneson</a> (postdoc, investigated <a href="https://f1000research.com/articles/4-1521/">various</a> <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-015-0862-3">aspects</a> of <a href="https://www.nature.com/articles/nmeth.4612">RNA-seq</a> <a href="http://www.life-science-alliance.org/content/2/1/e201800175">data</a> and <a href="https://www.nature.com/articles/nmeth.3805">method</a> <a href="https://academic.oup.com/bioinformatics/article/34/4/691/4345646">benchmarking</a>; now at <a href="https://www.fmi.ch">FMI</a>)</li>
<li>Helen Lindsay (postdoc, author of <a href="https://bioconductor.org/packages/release/bioc/html/CrispRVariants.html">CrispRVariants</a>; now at <a href="https://www.epfl.ch/">EPFL</a>)</li>
<li>Charity Law (postdoc, worked on differential splicing; now at <a href="https://www.wehi.edu.au/">WEHI</a>)</li>
<li>Andrea Riebler (postdoc, author of <a href="https://genomebiology.biomedcentral.com/articles/10.1186/gb-2014-15-2-r35">BayMeth</a> in <a href="http://bioconductor.org/packages/release/bioc/html/Repitools.html">Repitools</a> package; now at <a href="https://www.ntnu.edu/imf">NTNU</a>)</li>
<li>Ian Morilla (postdoc, involved in differential expression analysis using de novo RNA-seq assemblies; now at <a href="https://www.inserm.fr/">INSERM</a>)</li>
<li><a href="https://lmweber.github.io/">Lukas Weber</a> (PhD student, <a href="https://onlinelibrary.wiley.com/doi/full/10.1002/cyto.a.23030">compared CyTOF clustering algorithms</a>, author of <a href="https://www.nature.com/articles/s42003-019-0415-5">diffcyt</a>, reviewed <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1738-8">benchmarking</a>; now at <a href="https://www.jhsph.edu/departments/biostatistics/index.html">JHU</a>)</li>
<li>Xiaobei Zhou (PhD student; author of <a href="https://academic.oup.com/nar/article/42/11/e91/1427925">edgeR robust</a>)</li>
<li>Malgorzata Nowicka (PhD student, author of <a href="https://bioconductor.org/packages/DRIMSeq">DRIMSeq</a> and <a href="https://bioconductor.org/packages/cytofWorkflow/">cytofWorkflow</a>; now at <a href="https://www.roche.ch/">Hoffman La Roche</a>)</li>
<li>Hadi Gharibi (PhD student, analyzed BS-seq data; now at <a href="http://ut.ac.ir/en">Uni. Tehran</a>)</li>
<li>Andrea Komljenovic (PhD student, analyzed DNA methylation data; now at <a href="https://www.unil.ch/dee/home/menuinst/people/graduate-students.html">UNIL</a>)</li>
<li>Olga Nikolayeva (PhD student, analyzed PacBio DNA-seq data)</li>
<li>Lourdes Rosano González (MSc student, analyzed transformations of CyTOF data; now at <a href="http://www.nexus.ethz.ch/">ETH NEXUS</a>)</li>
<li>Katarina Matthes (MSc student, compared <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-015-0862-3">differential splicing methods</a>)</li>
<li>Dania Machlab (MSc student, analyzed allele-specific methylation from BS-seq data; now at <a href="https://www.fmi.ch/">FMI</a>)</li>
<li>Buildman Biyong (MSc student, established components of CrispRVariantsLite)</li>
<li>Angelo Duò (MSc student, <a href="https://f1000research.com/articles/7-1141/v2">compared scRNA-seq clustering algorithms</a>; now at <a href="https://www.scailyte.com/">Scailyte</a>)</li>
<li>Amr Elriedy (MSc student, analyzed isoform detection using PacBio cDNA-seq; now at University of Oxford)</li>
<li>Romy Schleiss (MSc student, compared normalization for CyTOF data)</li>
</ul>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
