<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>People</title>

<script src="site_libs/header-attrs-2.27/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/flatly.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-6.4.2/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.4.2/css/v4-shims.min.css" rel="stylesheet" />

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-inverse  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Statistical Bioinformatics @SIB @UZH</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="research.html">
    <span class="fa fa-area-chart"></span>
     
    research
  </a>
</li>
<li>
  <a href="publications.html">
    <span class="fa fa-file-text-o"></span>
     
    publications
  </a>
</li>
<li>
  <a href="tools.html">
    <span class="fa fa-code"></span>
     
    resources
  </a>
</li>
<li>
  <a href="people.html">
    <span class="fa fa-user"></span>
     
    people
  </a>
</li>
<li>
  <a href="teaching.html">
    <span class="fa fa-graduation-cap"></span>
     
    teaching
  </a>
</li>
<li>
  <a href="more_info.html">
    <span class="fa fa-info-circle"></span>
     
    more info
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">People</h1>

</div>


<table class="table table-condensed" style="width: auto !important; margin-left: auto; margin-right: auto;">
<tbody>
<tr>
<td style="border:none;text-align:left;width: 3cm; font-weight: bold;font-size: 18px;vertical-align: top !important;" rowspan="4">
<img src="img/thumbnail/mark.jpg" />
</td>
<td style="border:none;text-align:left;width: 20cm; font-weight: bold;font-size: 18px;">
Mark D Robinson
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
<i class="fa fa-twitter"></i> Twitter <a
href="https://twitter.com/markrobinsonca">@markrobinsonca</a><br><i class="fa fa-github"></i>
GitHub <a
href="https://github.com/markrobinsonuzh">markrobinsonuzh</a><br><i class="fa fa-newspaper-o"></i>
<a href="https://scholar.google.ch/citations?user=XPfrRQEAAAAJ">Google
Scholar</a>
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
My research interests are diverse, but more-or-less encompass the
general application of statistical methods and data science to
experimental data with biological applications. Often, this is within
the context of genomics data types, but we are interested in
methodological challenges and robust solutions in data, generally. We
also try to be modern scientists, with a focus on reproducibility (repos
for code) and open science (preprints).
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
PhD in Medical Biology (2008), University of Melbourne (Walter and Eliza
Hall Institute)<br>MSc in Statistics (2001), University of British
Columbia<br>BSc in Applied Mathematics and Statistics (1999), University
of Guelph
</td>
</tr>
</tbody>
</table>
<table class="table table-condensed" style="width: auto !important; margin-left: auto; margin-right: auto;">
<tbody>
<tr>
<td style="border:none;text-align:left;width: 3cm; font-weight: bold;font-size: 18px;vertical-align: top !important;" rowspan="4">
<img src="img/thumbnail/izaskun.jpg" />
</td>
<td style="border:none;text-align:left;width: 20cm; font-weight: bold;font-size: 18px;">
Izaskun Mallona
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
<i class="fa fa-desktop"></i> <a
href="http://imallona.bitbucket.io">Website</a><br><i class="fa fa-newspaper-o"></i>
<a href="https://scholar.google.ch/citations?user=kMfxCIYAAAAJ">Google
Scholar</a>
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
I am mainly interested in gene expression modulation, as it lays the
ground for any cell fate. The questions which excite my curiosity the
most are: How are gene expression programs coded? and maintained? how do
they adapt during development? and disease? To mine the regulation
metastructure genome-wide I’m fond of integrating gene expression and
epigenomic data, and to develop methods for doing so.
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
BSc in Computer Engineering (2014), Open University of Catalonia<br>PhD
(2012), Technical University of Cartagena<br>MSc in Agriculture
Engineering (2008), Technical University of Cartagena<br>MSc in Biology
(2007), University of Murcia
</td>
</tr>
</tbody>
</table>
<table class="table table-condensed" style="width: auto !important; margin-left: auto; margin-right: auto;">
<tbody>
<tr>
<td style="border:none;text-align:left;width: 3cm; font-weight: bold;font-size: 18px;vertical-align: top !important;" rowspan="4">
<img src="img/thumbnail/pierreluc.jpg" />
</td>
<td style="border:none;text-align:left;width: 20cm; font-weight: bold;font-size: 18px;">
Pierre-Luc Germain
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
<i class="fa fa-github"></i> GitHub <a
href="https://github.com/plger">plger</a><br><i class="fa fa-newspaper-o"></i>
<a href="https://scholar.google.com/citations?user=788GnysAAAAJ">Google
Scholar</a>
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
I gradually shifted from the humanities to bioinformatics towards the
end of my PhD, worked a lot in the past on linking chromatin alterations
to gene expression, and currently work mostly on the analysis of short
RNA-seq and single-cell RNA-seq, as well as pre- and
post-transcriptional regulation of gene expression especially in the
context of neurodevelopment. I have a shared position with the ETH’s
D-HEST Institute for Neurosciences.
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
PhD in Philosophy of science (2014), SEMM and University of Milan<br>MA
in Philosophy (2009), University of Montreal<br>BA in Philosophy (2007)
</td>
</tr>
</tbody>
</table>
<table class="table table-condensed" style="width: auto !important; margin-left: auto; margin-right: auto;">
<tbody>
<tr>
<td style="border:none;text-align:left;width: 3cm; font-weight: bold;font-size: 18px;vertical-align: top !important;" rowspan="3">
<img src="img/thumbnail/almut.jpg" />
</td>
<td style="border:none;text-align:left;width: 20cm; font-weight: bold;font-size: 18px;">
Almut Lütge
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
I’m a PhD student with a background in Molecular Biotechnology, but
already during my master I focused on Bioinformatics. I’m interested in
all kinds of genomic analysis, but mainly worked with transcriptomic
data, so far. My PhD project is about methods in single cell RNAseq and
part of a Sinergia collaboration on defining the identity and
differentiation pathways of the immune-stimulating fibroblastic tumor
stroma.
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
MSc in Molecular Biotechnology (2018), University of Heidelberg<br>BSc
Molecular Biotechnology (2013), University of Heidelberg
</td>
</tr>
</tbody>
</table>
<table class="table table-condensed" style="width: auto !important; margin-left: auto; margin-right: auto;">
<tbody>
<tr>
<td style="border:none;text-align:left;width: 3cm; font-weight: bold;font-size: 18px;vertical-align: top !important;" rowspan="4">
<img src="img/thumbnail/Anthony.png" />
</td>
<td style="border:none;text-align:left;width: 20cm; font-weight: bold;font-size: 18px;">
Anthony Sonrel
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
<i class="fa fa-linkedin-square"></i> <a
href="https://www.linkedin.com/in/anthony-sonrel-612505114/">LinkedIn</a>
<br>
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
I am a PhD student that joined the Robinson Lab as part of the Life
Science Zürich Graduate School. Through my past projects in academia and
industry I worked on various topics such as NGS, GWAS summary
statistics, epigenetic analyses (ChIP/ATAC-seq) and deep neural
networks. In my current project, I’m evaluating and developing tools
(mainly R packages) for single-cell RNA sequencing datasets. More
specifically, I’m interested in developing new methodologies and new
effective tools.
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
MSc in Molecular Life Sciences - Bioinformatics (2018), University of
Lausanne<br>BSc of Science in Biology (2015), University of Lausanne
</td>
</tr>
</tbody>
</table>
<table class="table table-condensed" style="width: auto !important; margin-left: auto; margin-right: auto;">
<tbody>
<tr>
<td style="border:none;text-align:left;width: 3cm; font-weight: bold;font-size: 18px;vertical-align: top !important;" rowspan="3">
<img src="img/thumbnail/Reto.jpg" />
</td>
<td style="border:none;text-align:left;width: 20cm; font-weight: bold;font-size: 18px;">
Reto Gerber
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
I joined the Robinson Lab as a PhD student after working as a Research
Assistant in the group. During this time I worked on scRNA-seq data from
Synovial tissue in collaboration with USZ. Furthermore I developped <a
href="https://pubassistant.ch">pubassistant.ch</a> (R shiny app to help
consolidate publication profiles of researchers) and <a
href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-021-04125-4">censcyt</a>
(method for differential abundance analysis in cytometry where a
covariate is subject to censoring).
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
MSc of Science in Computational Biology and Bioinformatics (2020), ETH,
University of Zürich<br>BSc of Science in Biology (2017), University of
Zürich
</td>
</tr>
</tbody>
</table>
<table class="table table-condensed" style="width: auto !important; margin-left: auto; margin-right: auto;">
<tbody>
<tr>
<td style="border:none;text-align:left;width: 3cm; font-weight: bold;font-size: 18px;vertical-align: top !important;" rowspan="4">
<img src="img/thumbnail/emanuel.jpg" />
</td>
<td style="border:none;text-align:left;width: 20cm; font-weight: bold;font-size: 18px;">
Emanuel Sonder
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
<i class="fa fa-twitter"></i> Twitter <a
href="https://twitter.com/EmanuelSonder">@EmanuelSonder</a><br><i class="fa fa-github"></i>
GitHub <a href="https://github.com/emsonder">emsonder</a><br>
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
After finishing my Master’s Thesis in the Robinson Group, I started a
shared PhD position with the <a
href="https://schrattlab.ethz.ch/the-group.html">Schratt lab</a>. I have
a general interest in the analysis of epigenetic data (ATAC-seq, WGBS).
My PhD project is focused on the reverse engineering of cell
type-specific transcriptional networks by integrating different data
modalities. The overarching goal is to provide insights in brain cell
type specific responses to different stressors when compared with
neuronal activity.
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
MSc in Computational Biology and Bioinformatics (2021), ETH Zürich,
University of Zürich<br>BSc in Biology (2018), ETH Zürich
</td>
</tr>
</tbody>
</table>
<table class="table table-condensed" style="width: auto !important; margin-left: auto; margin-right: auto;">
<tbody>
<tr>
<td style="border:none;text-align:left;width: 3cm; font-weight: bold;font-size: 18px;vertical-align: top !important;" rowspan="4">
<img src="img/thumbnail/vlad.png" />
</td>
<td style="border:none;text-align:left;width: 20cm; font-weight: bold;font-size: 18px;">
Vladyslav Korobeynyk
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
<i class="fa fa-github"></i> GitHub <a
href="https://github.com/vkorobeynyk">vkorobeynyk</a><br>
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
I did my master thesis in single cell analysis of the adult mouse
hippocampus in Jessberger Lab, HIFO, UZH. Shortly after, I started
working as an intern in Systems Biology group at Roche, where I gained
experience in ML and different type of sequencing technologies.
Currently I am doing a shared PhD with Jessberger Lab, where my project
is based on using single-cell and spatial transcriptomics data to
identify molecular changes, mainly in the neurogenic niche, that drive
neurogenesis to decline with age.
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
MSc in Biochemistry (2021), University of Zurich<br>BSc in Biochemistry
(2017), University Nova of Lisbon
</td>
</tr>
</tbody>
</table>
<table class="table table-condensed" style="width: auto !important; margin-left: auto; margin-right: auto;">
<tbody>
<tr>
<td style="border:none;text-align:left;width: 3cm; font-weight: bold;font-size: 18px;vertical-align: top !important;" rowspan="3">
<img src="img/thumbnail/david.jpg" />
</td>
<td style="border:none;text-align:left;width: 20cm; font-weight: bold;font-size: 18px;">
David Wissel
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
I am a shared PhD student between the Robinson Lab at UZH and the
BoevaLab at ETH. I am broadly interested in survival analysis,
single-cell and spatial transcriptomics as well as applications of
machine learning to all of these.
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
MSc Statistics (2021), ETH Zurich<br>BSc Business Administration (2019),
Free University of Berlin<br>
</td>
</tr>
</tbody>
</table>
<table class="table table-condensed" style="width: auto !important; margin-left: auto; margin-right: auto;">
<tbody>
<tr>
<td style="border:none;text-align:left;width: 3cm; font-weight: bold;font-size: 18px;vertical-align: top !important;" rowspan="3">
<img src="img/thumbnail/Siyuan.jpg" />
</td>
<td style="border:none;text-align:left;width: 20cm; font-weight: bold;font-size: 18px;">
Siyuan Luo
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
I’m an external PhD student from the <a
href="https://epigenetics.ethz.ch/">von Meyenn lab</a> with a background
in computational biology and bioinformatics. During my master study, I
worked on various topics including copy number variation and TCR/BCR
repertoire analysis. Now I’m mainly interested in analyzing single-cell
epigenetic data (ATAC-seq/CUT&amp;Tag). My PhD is about methods for
single-cell multi-omics data to reveal the interplay between the
epigenetic layers and transcriptomics and provide insights for chronic
diseases like obesity and ageing.
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
MSc in Computational Biology and Bioinformatics (2021), ETH Zürich,
University of Zürich<br>BSc in Biotechnology (2018), Harbin Institute of
Technology<br>
</td>
</tr>
</tbody>
</table>
<table class="table table-condensed" style="width: auto !important; margin-left: auto; margin-right: auto;">
<tbody>
<tr>
<td style="border:none;text-align:left;width: 3cm; font-weight: bold;font-size: 18px;vertical-align: top !important;" rowspan="4">
<img src="img/thumbnail/martin.jpg" />
</td>
<td style="border:none;text-align:left;width: 20cm; font-weight: bold;font-size: 18px;">
Martin Emons
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
<i class="fa fa-twitter"></i> Twitter <a
href="https://twitter.com/EmonsMartin">@EmonsMartin</a><br><i class="fa fa-github"></i>
GitHub <a href="https://github.com/mjemons">mjemons</a><br>
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
I am a PhD student in the Robinson Group since 2022 in the Epidemiology
and Biostatistics PhD program. My general interest lies in the
development of methods for biological and biomedical data science. In my
master studies I worked on various topics such as statistical analysis
of temporal trends or survival analysis at Roche Diagnostics. Currently,
I am interested in spatial transcriptomics and the challenges in this
new field.
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
MSc in Computational Biology and Bioinformatics (2022), ETH Zürich,
University of Zürich<br>BSc in Biology (2019), University of Bern
</td>
</tr>
</tbody>
</table>
<table class="table table-condensed" style="width: auto !important; margin-left: auto; margin-right: auto;">
<tbody>
<tr>
<td style="border:none;text-align:left;width: 3cm; font-weight: bold;font-size: 18px;vertical-align: top !important;" rowspan="4">
<img src="img/thumbnail/samuel.jpeg" />
</td>
<td style="border:none;text-align:left;width: 20cm; font-weight: bold;font-size: 18px;">
Samuel Gunz
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
<i class="fa fa-twitter"></i> Twitter <a
href="https://twitter.com/samuelgunz">@samuelgunz</a><br><i class="fa fa-github"></i>
GitHub <a href="https://github.com/sgunz">sgunz</a><br>
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
I am a PhD student in the Robinson lab as part of the Epidemiology and
Biostatistics program of the Life Science Graduate School Zurich. During
my past studies, I worked with all sorts of biological data, ranging
from microbial community analysis to characterizing epigenetic
modifications in cancer. During this time, I acknowledged the importance
of having robust and reproducible tools in research and that my interest
lies in developing such. My PhD project will focus on methods for
analyzing spatial omics data.
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
MSc in Computational Biology and Bioinformatics (2022), ETH Zurich,
University of Basel and University of Zurich<br>BSc in Biology (2018),
University of Zurich
</td>
</tr>
</tbody>
</table>
<table class="table table-condensed" style="width: auto !important; margin-left: auto; margin-right: auto;">
<tbody>
<tr>
<td style="border:none;text-align:left;width: 3cm; font-weight: bold;font-size: 18px;vertical-align: top !important;" rowspan="3">
<img src="img/thumbnail/peiying.jpg" />
</td>
<td style="border:none;text-align:left;width: 20cm; font-weight: bold;font-size: 18px;">
Peiying Cai
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
In February 2023, I started working as a PhD student in the Robinson
lab. Previously, I worked on developing analysis tools and pipelines to
identify spatially variable genes from spatial transcriptomics data. My
PhD research will focus on the development of statistical methods in
bioinformatics, mostly for single-cell RNA-seq and spatial
transcriptomics data.
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
MSc in Biostatistics (2022), University of Zürich<br>BSc of Science in
Statistics (2019), Xiamen University
</td>
</tr>
</tbody>
</table>
<table class="table table-condensed" style="width: auto !important; margin-left: auto; margin-right: auto;">
<tbody>
<tr>
<td style="border:none;text-align:left;width: 3cm; font-weight: bold;font-size: 18px;vertical-align: top !important;" rowspan="4">
<img src="img/thumbnail/danincicau.jpg" />
</td>
<td style="border:none;text-align:left;width: 20cm; font-weight: bold;font-size: 18px;">
Daniel Incicau
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
<i class="fa fa-linkedin-square"></i> <a
href="https://www.linkedin.com/in/daniel-incicau/">LinkedIn</a><br><i class="fa fa-github"></i>
<a href="https://github.com/DanInci">DanInci</a><br>
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
I am a Master’s student in Computational Biology and Bioinformatics. I’m
currently working on my thesis which involves evaluating cell
phenotyping methods in highly multiplexed imaging, with a focus on mass
cytometry images. I have a particular interest in immunology and I’m
your typical functional programming enthusiast.
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
MSc in Computational Biology and Bioinformatics (ongoing), ETH
Zürich<br>BSc in Computer Engineering (2020), Polytechnic University of
Timisoara
</td>
</tr>
</tbody>
</table>
<table class="table table-condensed" style="width: auto !important; margin-left: auto; margin-right: auto;">
<tbody>
<tr>
<td style="border:none;text-align:left;width: 3cm; font-weight: bold;font-size: 18px;vertical-align: top !important;" rowspan="4">
<img src="img/thumbnail/jiayi.jpg" />
</td>
<td style="border:none;text-align:left;width: 20cm; font-weight: bold;font-size: 18px;">
Jiayi Wang
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
<i class="fa fa-linkedin-square"></i> <a
href="https://www.linkedin.com/in/jiayi-wang-14a3261ba/">LinkedIn</a><br>
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
I am a Master’s student in Computational Biology and Bioinformatics. I
started as a part-time intern research student in the Robinson Group
since March 2023, working on disentangling type and state effects on
expression variability to improve analysis and interpretability of
scRNA-seq data. Since November 2023, I also started working on my
Master’s thesis on improving an R package for ATAC-seq analysis to
identify motif or annotation-related peak accessibility changes.
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
MSc in Computational Biology and Bioinformatics (ongoing), ETH
Zürich<br>BSc in Biomedical Sciences (2021), University College London
</td>
</tr>
</tbody>
</table>
<table class="table table-condensed" style="width: auto !important; margin-left: auto; margin-right: auto;">
<tbody>
<tr>
<td style="border:none;text-align:left;width: 3cm; font-weight: bold;font-size: 18px;vertical-align: top !important;" rowspan="3">
<img src="img/thumbnail/sarusan.jpg" />
</td>
<td style="border:none;text-align:left;width: 20cm; font-weight: bold;font-size: 18px;">
Sarusan Kathirchelvan
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
I am a Master’s student in Computational Biology and Bioinformatics. I
am currently working on my thesis involving the application of ensemble
clustering in spatial transcriptomics and evaluating the spatial
homogeneity, and the optimal cluster numbers of those detected clusters,
to improve the spatial domain detection problem as well as to transition
from a supervised to an unsupervised data-driven clustering.
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
MSc in Computational Biology and Bioinformatics (ongoing), ETH
Zürich<br>BSc in Computational Sciences (2022), University of Basel
</td>
</tr>
</tbody>
</table>
<div id="alumni" class="section level2">
<h2>Alumni</h2>
<ul>
<li>Yin Guo (visiting PhD student from Xi’an Jiaotong University)</li>
<li>Helena Crowell (PhD student; author of <a
href="https://bioconductor.org/packages/release/bioc/html/CATALYST.html">CATALYST</a>,
<a href="">muscat</a>, <a href="">SpatialExperiment</a>; now postdoc at
<a
href="https://www.cnag.eu/teams/genome-research-unit/single-cell-genomics-team">CNAG</a>,
Barcelona</li>
<li>Stefan Milosavljevic (PhD student shared with Shimizu group; author
of <a href="https://github.com/supermaxiste/ARPEGGIO">ARPEGGIO</a>; now
Biomedical Data Engineer at <a href="https://datascience.ch">Swiss Data
Science Center</a>)</li>
<li>Simone Tiberi (postdoc; author of <a
href="https://bioconductor.org/packages/release/bioc/html/BANDITS.html">BANDITS</a>
and <a
href="https://bioconductor.org/packages/release/bioc/html/distinct.html">distinct</a>;
now Assistant Professor at Università di Bologna)</li>
<li>Dominique Paul (research assistant)</li>
<li>Hao Lyu (visiting PhD student)</li>
<li>Elyas Heidari (Research assistant; author of <a
href="https://github.com/MarioniLab/SageNet">SageNet</a>; now PhD
student at EMBL Heidelberg)</li>
<li>Vladimir B. C. de Souza (PhD student; author of <a
href="https://github.com/vladimirsouza/lrRNAseqVariantCalling">lrRNAseqVariantCalling</a>)</li>
<li>Imran Fanaswala (Research assistant; founder of <a
href="https://yugacell.com/">YugaCell</a>)</li>
<li>Stephan Schmeing (PhD student; author of <a
href="https://github.com/schmeing/ReSeq">ReSeq</a>)</li>
<li>Ahmad Al Ajami (MSc student)</li>
<li>Kathi Hembach (PhD student; author of <a
href="https://github.com/khembach/DISCERNS">DISCERNS</a>; co-author of
<a href="https://github.com/csoneson/ARMOR">ARMOR</a>)</li>
<li>Stephany Orjuela (PhD student; author of <a
href="https://github.com/markrobinsonuzh/DAMEfinder">DAMEfinder</a>;
co-author of <a href="https://github.com/csoneson/ARMOR">ARMOR</a>; now
Scientific software developer at <a
href="https://oncobit.com/">Oncobit</a>)</li>
<li>Will Macnair (Postdoc; author of <a
href="https://github.com/wmacnair/SampleQC/">SampleQC</a>; now Data
Scientist at <a href="https://www.roche.ch/">Hoffman La Roche</a>)</li>
<li>Ruizhu (Fiona) Huang (PhD student; author of <a
href="https://github.com/fionarhuang/treeclimbR">treeclimbR</a>, <a
href="https://www.bioconductor.org/packages/release/bioc/html/TreeSummarizedExperiment.html">TreeSummarizeExperiment</a>,
co-author of <a href="https://github.com/csoneson/ARMOR">ARMOR</a>; now
Biostatistician at Cardio-CARE)</li>
<li><a
href="http://merlin.up.poznan.pl/kmmis2/index.php?id=158&amp;id2=105">Joanna
Zyprych-Walczak</a> (visiting professor)</li>
<li><a href="http://csoneson.github.io/">Charlotte Soneson</a> (postdoc,
investigated <a
href="https://f1000research.com/articles/4-1521/">various</a> <a
href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-015-0862-3">aspects</a>
of <a href="https://www.nature.com/articles/nmeth.4612">RNA-seq</a> <a
href="http://www.life-science-alliance.org/content/2/1/e201800175">data</a>
and <a href="https://www.nature.com/articles/nmeth.3805">method</a> <a
href="https://academic.oup.com/bioinformatics/article/34/4/691/4345646">benchmarking</a>;
now at <a href="https://www.fmi.ch">FMI</a>)</li>
<li>Helen Lindsay (postdoc, author of <a
href="https://bioconductor.org/packages/release/bioc/html/CrispRVariants.html">CrispRVariants</a>;
now at <a href="https://www.epfl.ch/">EPFL</a>)</li>
<li>Charity Law (postdoc, worked on differential splicing; now at <a
href="https://www.wehi.edu.au/">WEHI</a>)</li>
<li>Andrea Riebler (postdoc, author of <a
href="https://genomebiology.biomedcentral.com/articles/10.1186/gb-2014-15-2-r35">BayMeth</a>
in <a
href="http://bioconductor.org/packages/release/bioc/html/Repitools.html">Repitools</a>
package; now at <a href="https://www.ntnu.edu/imf">NTNU</a>)</li>
<li>Ian Morilla (postdoc, involved in differential expression analysis
using de novo RNA-seq assemblies; now at <a
href="https://www.inserm.fr/">INSERM</a>)</li>
<li><a href="https://lmweber.github.io/">Lukas Weber</a> (PhD student,
<a
href="https://onlinelibrary.wiley.com/doi/full/10.1002/cyto.a.23030">compared
CyTOF clustering algorithms</a>, author of <a
href="https://www.nature.com/articles/s42003-019-0415-5">diffcyt</a>,
reviewed <a
href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1738-8">benchmarking</a>;
now at <a
href="https://www.jhsph.edu/departments/biostatistics/index.html">JHU</a>)</li>
<li>Xiaobei Zhou (PhD student; author of <a
href="https://academic.oup.com/nar/article/42/11/e91/1427925">edgeR
robust</a>)</li>
<li>Malgorzata Nowicka (PhD student, author of <a
href="https://bioconductor.org/packages/DRIMSeq">DRIMSeq</a> and <a
href="https://bioconductor.org/packages/cytofWorkflow/">cytofWorkflow</a>;
now at <a href="https://www.roche.ch/">Hoffman La Roche</a>)</li>
<li>Hadi Gharibi (PhD student, analyzed BS-seq data; now at <a
href="http://ut.ac.ir/en">Uni. Tehran</a>)</li>
<li>Andrea Komljenovic (PhD student, analyzed DNA methylation data; now
at <a
href="https://www.unil.ch/dee/home/menuinst/people/graduate-students.html">UNIL</a>)</li>
<li>Olga Nikolayeva (PhD student, analyzed PacBio DNA-seq data)</li>
<li>Lourdes Rosano González (MSc student, analyzed transformations of
CyTOF data; now at <a href="http://www.nexus.ethz.ch/">ETH
NEXUS</a>)</li>
<li>Katarina Matthes (MSc student, compared <a
href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-015-0862-3">differential
splicing methods</a>)</li>
<li>Dania Machlab (MSc student, analyzed allele-specific methylation
from BS-seq data; now at <a href="https://www.fmi.ch/">FMI</a>)</li>
<li>Buildman Biyong (MSc student, established components of
CrispRVariantsLite)</li>
<li>Angelo Duò (MSc student, <a
href="https://f1000research.com/articles/7-1141/v2">compared scRNA-seq
clustering algorithms</a>; now at <a
href="https://www.scailyte.com/">Scailyte</a>)</li>
<li>Amr Elriedy (MSc student, analyzed isoform detection using PacBio
cDNA-seq; now at University of Oxford)</li>
<li>Romy Schleiss (MSc student, compared normalization for CyTOF
data)</li>
</ul>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
