<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />




<title></title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/flatly.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-4.5.0/css/font-awesome.min.css" rel="stylesheet" />

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 60px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 65px;
  margin-top: -65px;
}

.section h2 {
  padding-top: 65px;
  margin-top: -65px;
}
.section h3 {
  padding-top: 65px;
  margin-top: -65px;
}
.section h4 {
  padding-top: 65px;
  margin-top: -65px;
}
.section h5 {
  padding-top: 65px;
  margin-top: -65px;
}
.section h6 {
  padding-top: 65px;
  margin-top: -65px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->






<div class="navbar navbar-inverse  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Statistical Bioinformatics @SIB @UZH</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="research.html">
    <span class="fa fa-area-chart"></span>
     
    research
  </a>
</li>
<li>
  <a href="publications.html">
    <span class="fa fa-file-text-o"></span>
     
    publications
  </a>
</li>
<li>
  <a href="tools.html">
    <span class="fa fa-code"></span>
     
    tools
  </a>
</li>
<li>
  <a href="people.html">
    <span class="fa fa-user"></span>
     
    people
  </a>
</li>
<li>
  <a href="teaching.html">
    <span class="fa fa-graduation-cap"></span>
     
    teaching
  </a>
</li>
<li>
  <a href="news.html">
    <span class="fa fa-newspaper-o"></span>
     
    news
  </a>
</li>
<li>
  <a href="more_info.html">
    <span class="fa fa-info-circle"></span>
     
    more info
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">




</div>


<table class="table table-condensed" style="width: auto !important; margin-left: auto; margin-right: auto;">
<tbody>
<tr>
<td style="border:none;text-align:left;width: 3cm; font-weight: bold;font-size: 18px;vertical-align: top !important;" rowspan="3">
<img src="img/mark.jpg" />
</td>
<td style="border:none;text-align:left;width: 20cm; font-weight: bold;font-size: 18px;">
Mark Robinson
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
Text text text text text text text text text text text text text text text text text text text text text text text text text text text
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
More text (e.g. qualifications)<br>More text
</td>
</tr>
</tbody>
</table>
<table class="table table-condensed" style="width: auto !important; margin-left: auto; margin-right: auto;">
<tbody>
<tr>
<td style="border:none;text-align:left;width: 3cm; font-weight: bold;font-size: 18px;vertical-align: top !important;" rowspan="3">
<img src="img/helen.jpg" />
</td>
<td style="border:none;text-align:left;width: 20cm; font-weight: bold;font-size: 18px;">
Helen Lindsay
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
Text text text text text text text text text text text text text text text text text text text text text text text text text text text
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
More text (e.g. qualifications)<br>More text
</td>
</tr>
</tbody>
</table>
<table class="table table-condensed" style="width: auto !important; margin-left: auto; margin-right: auto;">
<tbody>
<tr>
<td style="border:none;text-align:left;width: 3cm; font-weight: bold;font-size: 18px;vertical-align: top !important;" rowspan="4">
<img src="img/charlotte.png" />
</td>
<td style="border:none;text-align:left;width: 20cm; font-weight: bold;font-size: 18px;">
Charlotte Soneson
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
<i class="fa fa-desktop"></i> <a href="http://csoneson.github.io/">Website</a><br><i class="fa fa-twitter"></i> Twitter <a href="https://twitter.com/CSoneson">@CSoneson</a><br><i class="fa fa-github"></i> GitHub <a href="https://github.com/csoneson">csoneson</a><br><i class="fa fa-newspaper-o"></i> <a href="https://scholar.google.ch/citations?user=c28GqkIAAAAJ">Google Scholar</a>
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
As a postdoctoral research associate, one of my main interests is the development and improvement of statistical methods and workflows, currently mainly in the field of RNA-seq. I am also interested in the process of evaluating and comparing methods in an objective, robust and reproducible manner.
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
PhD in Mathematics (2011), Lund University<br>MSc in Engineering Physics (2005), Lund University
</td>
</tr>
</tbody>
</table>
<table class="table table-condensed" style="width: auto !important; margin-left: auto; margin-right: auto;">
<tbody>
<tr>
<td style="border:none;text-align:left;width: 3cm; font-weight: bold;font-size: 18px;vertical-align: top !important;" rowspan="3">
<img src="img/simone.jpg" />
</td>
<td style="border:none;text-align:left;width: 20cm; font-weight: bold;font-size: 18px;">
Simone Tiberi
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
I am a postdoctoral research associate working on the development of cutting-edge statistical methods for RNA-seq data. In particular, I am currently focusing on a Bayesian hierarchical model to detect genes showing differential transcript usage (DTU); the method accounts for the subject-to-subject variability in the transcripts proportions and for the uncertainty in the allocation of each fragment to the corresponding transcript, which is treated as a latent variable. In general terms, my interests are quite broad and lie in the development of statistical methods for applications in the medical and biological fields.
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
PhD in Statistics (2017), The University of Warwick<br>MSc in Statistics (2012), The University of Padua<br>BSc in Statistics (2010), Sapienza University of Rome
</td>
</tr>
</tbody>
</table>
<table class="table table-condensed" style="width: auto !important; margin-left: auto; margin-right: auto;">
<tbody>
<tr>
<td style="border:none;text-align:left;width: 3cm; font-weight: bold;font-size: 18px;vertical-align: top !important;" rowspan="4">
<img src="img/izaskun.jpg" />
</td>
<td style="border:none;text-align:left;width: 20cm; font-weight: bold;font-size: 18px;">
Izaskun Mallona
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
<i class="fa fa-desktop"></i> <a href="http://imallona.bitbucket.io">Website</a><br><i class="fa fa-newspaper-o"></i> <a href="https://scholar.google.ch/citations?user=kMfxCIYAAAAJ">Google Scholar</a>
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
I am mainly interested in gene expression modulation, as it lays the ground for any cell fate. The questions which excite my curiosity the most are: How are gene expression programs coded? and maintained? how do they adapt during development? and disease? To mine the presumable genome-wide metastructure I’m fond of integrating gene expression and epigenomic data, and to develop methods to do so.
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
PhD Technical University of Cartagena 2012, MSc Biology University of Murcia 2007, BSc Computer Eng Open University of Catalonia 2014
</td>
</tr>
</tbody>
</table>
<table class="table table-condensed" style="width: auto !important; margin-left: auto; margin-right: auto;">
<tbody>
<tr>
<td style="border:none;text-align:left;width: 3cm; font-weight: bold;font-size: 18px;vertical-align: top !important;" rowspan="4">
<img src="img/lukas.jpg" />
</td>
<td style="border:none;text-align:left;width: 20cm; font-weight: bold;font-size: 18px;">
Lukas Weber
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
<i class="fa fa-desktop"></i> <a href="https://scholar.google.ch/citations?user=M8uPSzMAAAAJ">Website</a><br><i class="fa fa-twitter"></i> Twitter <a href="https://twitter.com/lmwebr">@lmwebr</a><br><i class="fa fa-github"></i> GitHub <a href="https://github.com/lmweber">lmweber</a><br><i class="fa fa-newspaper-o"></i> <a href="https://scholar.google.ch/citations?user=M8uPSzMAAAAJ">Google Scholar</a>
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
I am a PhD student in biostatistics, working on the development of statistical and computational methods for analysis of various types of high-throughput biological data. The main focus of my project is on methods for differential discovery in high-dimensional single-cell flow and mass cytometry data. These methods are now available in the <a href="http://bioconductor.org/packages/diffcyt">diffcyt</a> R package from Bioconductor. I also enjoy teaching, especially R programming and data analysis skills as an instructor for <a href="https://carpentries.org/">Software and Data Carpentry</a>.
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
MSc in Statistics (2014), ETH Zurich
</td>
</tr>
</tbody>
</table>
<table class="table table-condensed" style="width: auto !important; margin-left: auto; margin-right: auto;">
<tbody>
<tr>
<td style="border:none;text-align:left;width: 3cm; font-weight: bold;font-size: 18px;">
</td>
<td style="border:none;text-align:left;width: 20cm; font-weight: bold;font-size: 18px;">
Katharina Hembach
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 3cm; ">
</td>
<td style="border:none;text-align:left;width: 20cm; ">
I have a background in bioinformatics and I am a shared PhD student with the <a href="http://www.imls.uzh.ch/en/research/polymenidou.html">Polymenidou lab</a>. They focus on understanding the molecular mechanisms of neurodegenerative diseases such as Amyotrophic Lateral Sclerosis (ALS) and Frontotemporal Lobar Degeneration (FTLD). I help with the analysis and interpretation of different sequencing data, such as RNA-seq, CLIP-seq, and iCLIP. My main project is the development of a pipeline for the identification and quantification of alternative splicing events, specifically microexons and cryptic exons, from RNA-seq data.
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 3cm; ">
</td>
<td style="border:none;text-align:left;width: 20cm; ">
BSc and MSc in Bioinformatics, LMU and TU Munich
</td>
</tr>
</tbody>
</table>
<table class="table table-condensed" style="width: auto !important; margin-left: auto; margin-right: auto;">
<tbody>
<tr>
<td style="border:none;text-align:left;width: 3cm; font-weight: bold;font-size: 18px;vertical-align: top !important;" rowspan="3">
<img src="img/stephan.jpg" />
</td>
<td style="border:none;text-align:left;width: 20cm; font-weight: bold;font-size: 18px;">
Stephan Schmeing
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
As a PhD student coming from a particle physics background I am currently working on a more realistic Illumina sequence simulator called ReSequencer. It is able to create a profile from a bam file that is then used to simulated paired-read fastq files. Furthermore I will work on an assembly and population genomics of the Indo-Pacific bottlenose dolphin (tursiops aduncus).
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
MSc in Astro, Particle and Nuclear physis (2014), TU Munich<br>BSc in Physics (2011), TU Munich
</td>
</tr>
</tbody>
</table>
<table class="table table-condensed" style="width: auto !important; margin-left: auto; margin-right: auto;">
<tbody>
<tr>
<td style="border:none;text-align:left;width: 3cm; font-weight: bold;font-size: 18px;vertical-align: top !important;" rowspan="3">
<img src="img/fiona.jpg" />
</td>
<td style="border:none;text-align:left;width: 20cm; font-weight: bold;font-size: 18px;">
Ruizhu Huang
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
My major research interest is in the development and application of statistical methods in biological data. Currently, I am working on the project about tree signal aggregation.
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
MSc in Biostatistics (2015), University of Zurich<br>BSc in Applied Chemsitry (2010), Wuhan University
</td>
</tr>
</tbody>
</table>
<table class="table table-condensed" style="width: auto !important; margin-left: auto; margin-right: auto;">
<tbody>
<tr>
<td style="border:none;text-align:left;width: 3cm; font-weight: bold;font-size: 18px;">
</td>
<td style="border:none;text-align:left;width: 20cm; font-weight: bold;font-size: 18px;">
Vladimir Barbosa C. de Souza
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 3cm; ">
</td>
<td style="border:none;text-align:left;width: 20cm; ">
I am a PhD candidate in Bioinformatics, and I am working with PacBio data (third-generation sequencing). As there is just few documentation about the data, one of the focus of my research is to figure out their features, mainly about quality scores. At the moment I am working on a project to develop a new fast and accurate method to create CCS reads (circular consensus sequences).
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 3cm; ">
</td>
<td style="border:none;text-align:left;width: 20cm; ">
MSc in Applied Statistics (2015), University of Vicosa<br>BSc in Biology, specialization Genetics (2013), University of Vicosa
</td>
</tr>
</tbody>
</table>
<table class="table table-condensed" style="width: auto !important; margin-left: auto; margin-right: auto;">
<tbody>
<tr>
<td style="border:none;text-align:left;width: 3cm; font-weight: bold;font-size: 18px;vertical-align: top !important;" rowspan="3">
<img src="img/stephany.jpg" />
</td>
<td style="border:none;text-align:left;width: 20cm; font-weight: bold;font-size: 18px;">
Stephany Orjuela
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
I have a shared PhD position with the <a href="http://www.imcr.uzh.ch/en/research/Marra/Team.html">Marra lab</a>, which focuses on studying the molecular mechanisms underlying the development and progression of Colorectal Cancer. Most of my work focuses on analyzing and developing methods to process methylation data obtained from BS-seq, although I also work with RNA-seq. I am currently developing a method to identify allele specific methylation. I am also in the early stages of developing a method to integrate RNA-seq and BS-seq data.
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
MSc in Biology (2017), University of Zurich<br>BSc in Biology (2015), Universidad Nacional de Colombia
</td>
</tr>
</tbody>
</table>
<table class="table table-condensed" style="width: auto !important; margin-left: auto; margin-right: auto;">
<tbody>
<tr>
<td style="border:none;text-align:left;width: 3cm; font-weight: bold;font-size: 18px;vertical-align: top !important;" rowspan="3">
<img src="img/stefan.jpg" />
</td>
<td style="border:none;text-align:left;width: 20cm; font-weight: bold;font-size: 18px;">
Stefan Milosavljevic
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
I am a PhD student with a background in biology and bioinformatics. I am currently working on the rapid evolution of a plant species called Arabidopsis kamchatica. My project focuses on epigenetic modifications over short timescales and my general aim is to assess the impact and the importance of DNA methylation in evolution over short periods (rapid evolution).
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
MSc in Computational Biology and Bioinformatics (2015), ETH Zurich and University of Zurich<br>BSc in Biology (2013), Université de Lausanne
</td>
</tr>
</tbody>
</table>
<table class="table table-condensed" style="width: auto !important; margin-left: auto; margin-right: auto;">
<tbody>
<tr>
<td style="border:none;text-align:left;width: 3cm; font-weight: bold;font-size: 18px;vertical-align: top !important;" rowspan="3">
<img src="img/helena.png" />
</td>
<td style="border:none;text-align:left;width: 20cm; font-weight: bold;font-size: 18px;">
Helena Lucia Crowell
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
In collaboration with the <a href="http://www.bodenmillerlab.com">Bodenmiller lab</a> lab, I have worked on correcting for spillover in CyTOF data. The preprint to this project is available on <a href="https://doi.org/10.1101/185744">bioRxiv</a> and is awaiting publication. Alongside our compensation approach, I have made Finck et al.’s <a href="https://doi.org/10.1002/cyto.a.22271">bead-based normalization</a> and Zunder et al.’s <a href="https://doi.org/10.1038/nprot.2015.020">single-cell deconvolution</a> available in <a href="https://bioconductor.org/packages/release/bioc/html/CATALYST.html">CATALYST</a> (<strong>C</strong>ytometry d<strong>AT</strong>a an<strong>ALYS</strong>is <strong>T</strong>ools): a <a href="https://bioconductor.org">Bioconductor</a> package providing tools for preprocessing and analysis of cytometry data. Currently, I am working on integrating M. Nowicka’s <a href="https://f1000research.com/articles/6-748/v2">CyTOF workflow</a> for differential analysis into CATALYST. My Masters thesis project is predicated on the benchmarking of (non-)branching trajectory inference tools for scRNA-seq and cytometry data.
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 20cm; ">
MSc in Computational Biology &amp; Bioinformatics (present), ETH Zurich<br>BSc in Biochemistry (2015), University of Heidelberg
</td>
</tr>
</tbody>
</table>
<table class="table table-condensed" style="width: auto !important; margin-left: auto; margin-right: auto;">
<tbody>
<tr>
<td style="border:none;text-align:left;width: 3cm; font-weight: bold;font-size: 18px;">
</td>
<td style="border:none;text-align:left;width: 20cm; font-weight: bold;font-size: 18px;">
Angelo Duo
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 3cm; ">
</td>
<td style="border:none;text-align:left;width: 20cm; ">
Text here
</td>
</tr>
<tr>
<td style="border:none;text-align:left;width: 3cm; ">
</td>
<td style="border:none;text-align:left;width: 20cm; ">
More text here
</td>
</tr>
</tbody>
</table>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
