<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Publications</title>

<script src="site_libs/header-attrs-2.11/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/flatly.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>








<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-inverse  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Statistical Bioinformatics @SIB @UZH</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="research.html">
    <span class="fa fa-area-chart"></span>
     
    research
  </a>
</li>
<li>
  <a href="publications.html">
    <span class="fa fa-file-text-o"></span>
     
    publications
  </a>
</li>
<li>
  <a href="tools.html">
    <span class="fa fa-code"></span>
     
    resources
  </a>
</li>
<li>
  <a href="people.html">
    <span class="fa fa-user"></span>
     
    people
  </a>
</li>
<li>
  <a href="teaching.html">
    <span class="fa fa-graduation-cap"></span>
     
    teaching
  </a>
</li>
<li>
  <a href="more_info.html">
    <span class="fa fa-info-circle"></span>
     
    more info
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Publications</h1>
<h4 class="author"></h4>

</div>


<script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>
<div id="preprints" class="section level2">
<h2>Preprints</h2>
<ul>
<li><strong>Tiberi, S.</strong>; <strong>Crowell, H. L.</strong>; Samartsidis, P.; <strong>Weber, L. M.</strong>; <strong>Robinson, M. D.</strong>: distinct: a novel approach to differential distribution analyses. bioRxiv doi: <a href="https://doi.org/10.1101/2020.11.24.394213" target="_blank"> https://doi.org/10.1101/2020.11.24.394213</a>. Posted April 23, 2022. <a href="https://github.com/SimoneTiberi/distinct_manuscript" target="_blank"> GitHub repo</a>. <a href="https://github.com/SimoneTiberi/distinct" target="_blank"> Software</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="10.1101/2020.11.24.394213" data-hide-no-mentions="true">

</div></li>
<li>Weiler, P.; Van Den Berge, K.; Street, K.; <strong>Tiberi, S.</strong>: A guide to trajectory inference and RNA velocity. bioRxiv doi: <a href="https://doi.org/10.1101/2021.12.22.473434" target="_blank"> https://doi.org/10.1101/2021.12.22.473434</a>. Posted April 23, 2022.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="10.1101/2021.12.22.473434" data-hide-no-mentions="true">

</div></li>
<li><strong>Heidari, E.</strong>; Lohoff, T.; Tyser, R. C. V.; Marioni, J.; <strong>Robinson, M. D.</strong>; Ghazanfar, S.: Supervised spatial inference of dissociated single-cell data with SageNet. bioRxiv doi: <a href="https://doi.org/10.1101/2022.04.14.488419" target="_blank"> https://doi.org/10.1101/2022.04.14.488419</a>. Posted April 15, 2022. <a href="https://github.com/MarioniLab/sagenet_paper" target="_blank"> GitHub repo</a>. <a href="https://github.com/MarioniLab/SageNet" target="_blank"> Software</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="10.1101/2022.04.14.488419" data-hide-no-mentions="true">

</div></li>
<li><strong>Macnair, W.</strong>; Calini, D.; Agirre, E.; Bryois, J.; Jaekel, S.; Kukanja, P.; Stokar-Regenscheit, N.; Ott, V.; Foo, L. C.; Collin, L.; Schippling, S.; Urich, E.; Nutma, E.; Marzin, M.; Amor, S.; Magliozzi, R.; <strong>Heidari, E.</strong>; <strong>Robinson, M. D.</strong>; Ffrench-Constant, C.; Castelo-Branco, G.; Williams, A.; Malhotra, D.: Single nuclei RNAseq stratifies multiple sclerosis patients into three distinct white matter glia responses. bioRxiv doi: <a href="https://doi.org/10.1101/2022.04.06.487263" target="_blank"> https://doi.org/10.1101/2022.04.06.487263</a>. Posted April 09, 2022. <a href="https://wmacnair.gitlab.io/MS_lesions_snRNAseq/" target="_blank"> GitHub repo</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="10.1101/2022.04.06.487263" data-hide-no-mentions="true">

</div></li>
<li><strong>Macnair, W.</strong>; <strong>Robinson, M. D.</strong>: SampleQC: robust multivariate, multi-celltype, multi-sample quality control for single cell data. bioRxiv doi: <a href="https://doi.org/10.1101/2021.08.28.458012" target="_blank"> https://doi.org/10.1101/2021.08.28.458012</a>. Posted April 05, 2022. <a href="https://github.com/wmacnair/SampleQC_paper_analyses" target="_blank"> GitHub repo</a>. <a href="https://github.com/wmacnair/SampleQC" target="_blank"> Software</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="10.1101/2021.08.28.458012" data-hide-no-mentions="true">

</div></li>
<li>Schmeing, S.; <strong>Robinson, M. D.</strong>: Gapless provides combined scaffolding, gap filling and assembly correction with long reads. bioRxiv doi: <a href="https://doi.org/10.1101/2022.03.08.483466" target="_blank"> https://doi.org/10.1101/2022.03.08.483466</a>. Posted March 09, 2022. <a href="https://github.com/schmeing/gapless" target="_blank"> Software</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="10.1101/2022.03.08.483466" data-hide-no-mentions="true">

</div></li>
<li><strong>De Souza, V. B. C.</strong>; Jordan, B. T.; Tseng, E.; Nelson, E. A.; Hirschi, K. K.; Sheynkman, G. M.; <strong>Robinson, M. D.</strong>: Transformation of alignment files improves performance of variant callers for long-read RNA sequencing data. bioRxiv doi: <a href="https://doi.org/10.1101/2022.02.08.479579" target="_blank"> https://doi.org/10.1101/2022.02.08.479579</a>. Posted March 09, 2022. <a href="https://github.com/vladimirsouza/lrRNAseqBenchmark" target="_blank"> GitHub repo</a>. <a href="https://doi.org/10.5281/zenodo.5960321" target="_blank"> Data</a>. <a href="https://github.com/vladimirsouza/lrRNAseqVariantCalling" target="_blank"> Software</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="10.1101/2022.02.08.479579" data-hide-no-mentions="true">

</div></li>
<li><strong>Crowell, H. L.</strong>; <strong>Leonardo, S. X. M</strong>.; <strong>Soneson, C.</strong>; <strong>Robinson, M. D.</strong>: Built on sand: the shaky foundations of simulating single-cell RNA sequencing data. bioRxiv doi: <a href="https://doi.org/10.1101/2021.11.15.468676" target="_blank"> https://doi.org/10.1101/2021.11.15.468676</a>. Posted February 23, 2022. <a href="https://github.com/HelenaLC/simulation-comparison" target="_blank"> GitHub repo</a>. <a href="https://doi.org/10.5281/zenodo.5678872" target="_blank"> Data</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="10.1101/2021.11.15.468676" data-hide-no-mentions="true">

</div></li>
<li>Hruska-Plochan, M.; <strong>Hembach, K. M.</strong>; Ronchi, S.; Wiersma, V. I.; Maniecka, Z.; Hock, E.-M.; Laferriere, F.; Sahadevan, S.; Hoop, V.; Delvendahl, I.; Panatta, M.; Van Der Bourg, A.; Bohaciakova, D.; Frontzek, K.; Aguzzi, A.; Lashley, T.; <strong>Robinson, M. D.</strong>; Karayannis, T.; Mueller, M.; Hierlemann, A.; Polymenidou, M.: Human neural networks with sparse TDP-43 pathology reveal NPTX2 misregulation in ALS/FTLD. bioRxiv doi: <a href="https://doi.org/10.1101/2021.12.08.471089" target="_blank"> https://doi.org/10.1101/2021.12.08.471089</a>. Posted December 09, 2021.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="10.1101/2021.12.08.471089" data-hide-no-mentions="true">

</div></li>
<li>Giles, K. A.; Phipps, A. J.; Cashion, J. M.; Huskins, S. N.; Mercer, T. R.; <strong>Robinson, M. D.</strong>; Woodhouse, A.; Taberlay, P. C.: H3K4me3 enrichment defines neuronal age, while a youthful H3K27ac signature is recapitulated in aged neurons. bioRxiv doi: <a href="https://doi.org/10.1101/2021.11.11.467877" target="_blank"> https://doi.org/10.1101/2021.11.11.467877</a>. Posted November 12, 2021.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="10.1101/2021.11.11.467877" data-hide-no-mentions="true">

</div></li>
<li><strong>Pierre-Luc Germain</strong>, Aaron Lun, <strong>Will Macnair</strong>, <strong>Mark D. Robinson</strong>: Doublet identification in single-cell sequencing data using scDblFinder. F1000Research doi: <a href="https://doi.org/10.12688/f1000research.73600.1" target="_blank"> https://doi.org/10.12688/f1000research.73600.1</a>. Posted September 28, 2021. <a href="https://github.com/plger/scDblFinder" target="_blank"> GitHub repo</a>. <a href="https://bioconductor.org/packages/scDblFinder" target="_blank"> Software</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="10.12688/f1000research.73600.1" data-hide-no-mentions="true">

</div></li>
<li>Krieg, C.; <strong>Weber, L. M.</strong>; Fosso, B.; Hardiman, G.; Mileti, E.; El Aidy, S.; Marzano, M.; <strong>Robinson, M. D.</strong>; Guglietta, S.: Loss of C3aR induces immune infiltration and inflammatory microbiota in a novel spontaneous model of colon cancer. bioRxiv doi: <a href="https://doi.org/10.1101/2021.01.18.426963" target="_blank"> https://doi.org/10.1101/2021.01.18.426963</a>. Posted August 02, 2021.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="10.1101/2021.01.18.426963" data-hide-no-mentions="true">

</div></li>
<li><strong>Helena L. Crowell</strong>, Stéphane Chevrier, Andrea Jacobs, Sujana Sivapatham, Tumor Profiler Consortium, Bernd Bodenmiller, <strong>Mark D. Robinson</strong>: An R-based reproducible and user-friendly preprocessing pipeline for CyTOF data. F1000Research doi: <a href="https://doi.org/10.12688/f1000research.26073.1" target="_blank"> https://doi.org/10.12688/f1000research.26073.1</a>. Posted October 22, 2020.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="10.12688/f1000research.26073.1" data-hide-no-mentions="true">

</div></li>
</ul>
</div>
<div id="section" class="section level2">
<h2>2022</h2>
<ul>
<li><strong>Gerber R</strong>, <strong>Robinson MD</strong>: pubassistant.ch: consolidating publication profiles of researchers. F1000Res 10:989 (2022). <a href="https://doi.org/10.12688/f1000research.73493.3" target="_blank"> DOI: 10.12688/f1000research.73493.3</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/35475034" target="_blank"> PMID 35475034</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.12688/f1000research.73493.3" data-hide-no-mentions="true">

</div></li>
<li>Prummel KD, <strong>Crowell HL</strong>, Nieuwenhuize S, Brombacher EC, Daetwyler S, <strong>Soneson C</strong>, Kresoja-Rakic J, Kocere A, Ronner M, Ernst A, Labbaf Z, Clouthier DE, Firulli AB, Sánchez-Iranzo H, Naganathan SR, O’Rourke R, Raz E, Mercader N, Burger A, Felley-Bosco E, Huisken J, <strong>Robinson MD</strong>, Mosimann C: Hand2 delineates mesothelium progenitors and is reactivated in mesothelioma. Nat Commun 13(1):1677 (2022). <a href="https://doi.org/10.1038/s41467-022-29311-7" target="_blank"> DOI: 10.1038/s41467-022-29311-7</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/35354817" target="_blank"> PMID 35354817</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1038/s41467-022-29311-7" data-hide-no-mentions="true">

</div></li>
<li>Rubinstein MP, Williams C, Mart C, Beall J, MacPherson L, Azar J, Swiderska-Syn M, <strong>Manca P</strong>, Gibney BC, <strong>Robinson MD</strong>, Krieg C, Hill EG, Taha SA, Rock AD, Lee JH, Soon-Shiong P, Wrangle J: Phase I Trial Characterizing the Pharmacokinetic Profile of N-803, a Chimeric IL-15 Superagonist, in Healthy Volunteers. J Immunol 208(6):1362-1370 (2022). <a href="https://doi.org/10.4049/jimmunol.2100066" target="_blank"> DOI: 10.4049/jimmunol.2100066</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/35228263" target="_blank"> PMID 35228263</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.4049/jimmunol.2100066" data-hide-no-mentions="true">

</div></li>
<li>Wagner J, Masek M, Jacobs A, <strong>Soneson C</strong>, Sivapatham S, Damond N, de Souza N, <strong>Robinson MD</strong>, Bodenmiller B: Mass cytometric and transcriptomic profiling of epithelial-mesenchymal transitions in human mammary cell lines. Sci Data 9(1):44 (2022). <a href="https://doi.org/10.1038/s41597-022-01137-4" target="_blank"> DOI: 10.1038/s41597-022-01137-4</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/35140234" target="_blank"> PMID 35140234</a>. <a href="https://github.com/csoneson/WagnerEMT2020" target="_blank"> GitHub repo</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1038/s41597-022-01137-4" data-hide-no-mentions="true">

</div></li>
<li>Righelli D, <strong>Weber LM</strong>, <strong>Crowell HL</strong>, Pardo B, Collado-Torres L, Ghazanfar S, Lun ATL, Hicks SC, Risso D: SpatialExperiment: infrastructure for spatially resolved transcriptomics data in R using Bioconductor. Bioinformatics :btac299 (2022). <a href="https://doi.org/10.1093/bioinformatics/btac299" target="_blank"> DOI: 10.1093/bioinformatics/btac299</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/35482478" target="_blank"> PMID 35482478</a>. <a href="https://bioconductor.org/packages/SpatialExperiment" target="_blank"> Software</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1093/bioinformatics/btac299" data-hide-no-mentions="true">

</div></li>
<li>Burja B, <strong>Paul D</strong>, Tastanova A, Edalat SG, <strong>Gerber R</strong>, Houtman M, Elhai M, Bürki K, Staeger R, Restivo G, Lang R, Sodin-Semrl S, Lakota K, Tomšič M, Levesque MP, Distler O, Rotar Ž, <strong>Robinson MD</strong> and Frank-Bertoncelj M: An Optimized Tissue Dissociation Protocol for Single-Cell RNA Sequencing Analysis of Fresh and Cultured Human Skin Biopsies Front. Cell Dev. Biol. : (2022). <a href="https://doi.org/10.3389/fcell.2022.872688" target="_blank"> DOI: 10.3389/fcell.2022.872688</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.3389/fcell.2022.872688" data-hide-no-mentions="true">

</div></li>
</ul>
</div>
<div id="section-1" class="section level2">
<h2>2021</h2>
<ul>
<li>Bai Y, Caussinus E, Leo S, Bosshardt F, Myachina F, Rot G, <strong>Robinson MD</strong>, Lehner CF: A cis-regulatory element promoting increased transcription at low temperature in cultured ectothermic Drosophila cells. BMC Genomics 22(1):771 (2021). <a href="https://doi.org/10.1186/s12864-021-08057-4" target="_blank"> DOI: 10.1186/s12864-021-08057-4</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/34711176" target="_blank"> PMID 34711176</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1186/s12864-021-08057-4" data-hide-no-mentions="true">

</div></li>
<li><strong>Orjuela S</strong>, Parker HR, Sajibu S, Cereatti F, Sauter M, Buffoli F, <strong>Robinson MD</strong>, Marra G: Disentangling tumorigenesis-associated DNA methylation changes in colorectal tissues from those associated with ageing. Epigenetics :1-18 (2021). <a href="https://doi.org/10.1080/15592294.2021.1952375" target="_blank"> DOI: 10.1080/15592294.2021.1952375</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/34369258" target="_blank"> PMID 34369258</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1080/15592294.2021.1952375" data-hide-no-mentions="true">

</div></li>
<li><strong>Milosavljevic S</strong>, Kuo T, Decarli S, Mohn L, Sese J, Shimizu KK, Shimizu-Inatsugi R, <strong>Robinson MD</strong>: ARPEGGIO: Automated Reproducible Polyploid EpiGenetic GuIdance workflOw. BMC Genomics 22(1):547 (2021). <a href="https://doi.org/10.1186/s12864-021-07845-2" target="_blank"> DOI: 10.1186/s12864-021-07845-2</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/34273949" target="_blank"> PMID 34273949</a>. <a href="https://github.com/supermaxiste/ARPEGGIO_paperAnalyses" target="_blank"> GitHub repo</a>. <a href="https://github.com/supermaxiste/ARPEGGIO" target="_blank"> Software</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1186/s12864-021-07845-2" data-hide-no-mentions="true">

</div></li>
<li>Sahadevan S, <strong>Hembach KM</strong>, Tantardini E, Pérez-Berlanga M, Hruska-Plochan M, Megat S, Weber J, Schwarz P, Dupuis L, <strong>Robinson MD</strong>, De Rossi P, Polymenidou M: Synaptic FUS accumulation triggers early misregulation of synaptic RNAs in a mouse model of ALS. Nat Commun 12(1):3027 (2021). <a href="https://doi.org/10.1038/s41467-021-23188-8" target="_blank"> DOI: 10.1038/s41467-021-23188-8</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/34021139" target="_blank"> PMID 34021139</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1038/s41467-021-23188-8" data-hide-no-mentions="true">

</div></li>
<li><strong>Huang R</strong>, <strong>Soneson C</strong>, <strong>Germain PL</strong>, Schmidt TSB, Mering CV, <strong>Robinson MD</strong>: treeclimbR pinpoints the data-dependent resolution of hierarchical hypotheses. Genome Biol 22(1):157 (2021). <a href="https://doi.org/10.1186/s13059-021-02368-1" target="_blank"> DOI: 10.1186/s13059-021-02368-1</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/34001188" target="_blank"> PMID 34001188</a>. <a href="https://github.com/fionarhuang/treeclimbR_article" target="_blank"> GitHub repo</a>. <a href="https://github.com/fionarhuang/treeclimbR" target="_blank"> Software</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1186/s13059-021-02368-1" data-hide-no-mentions="true">

</div></li>
<li><strong>Gerber R</strong>, <strong>Robinson MD</strong>: Censcyt: censored covariates in differential abundance analysis in cytometry. BMC Bioinformatics 22(1):235 (2021). <a href="https://doi.org/10.1186/s12859-021-04125-4" target="_blank"> DOI: 10.1186/s12859-021-04125-4</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/33971812" target="_blank"> PMID 33971812</a>. <a href="https://github.com/retogerber/censcyt_paper_scripts" target="_blank"> GitHub repo</a>. <a href="https://bioconductor.org/packages/censcyt" target="_blank"> Software</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1186/s12859-021-04125-4" data-hide-no-mentions="true">

</div></li>
<li><strong>Lütge A</strong>, <strong>Zyprych-Walczak J</strong>, Brykczynska Kunzmann U, <strong>Crowell HL</strong>, Calini D, Malhotra D, <strong>Soneson C</strong>, <strong>Robinson MD</strong>: CellMixS: quantifying and visualizing batch effects in single-cell RNA-seq data. Life Sci Alliance 4(6):e202001004 (2021). <a href="https://doi.org/10.26508/lsa.202001004" target="_blank"> DOI: 10.26508/lsa.202001004</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/33758076" target="_blank"> PMID 33758076</a>. <a href="https://github.com/almutlue/mixing_benchmark" target="_blank"> GitHub repo</a>. <a href="https://doi.org/10.6084/m9.figshare.13341200" target="_blank"> Data</a>. <a href="https://www.bioconductor.org/packages/release/bioc/html/CellMixS.html" target="_blank"> Software</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.26508/lsa.202001004" data-hide-no-mentions="true">

</div></li>
<li><strong>Schmeing S</strong>, <strong>Robinson MD</strong>: ReSeq simulates realistic Illumina high-throughput sequencing data. Genome Biol 22(1):67 (2021). <a href="https://doi.org/10.1186/s13059-021-02265-7" target="_blank"> DOI: 10.1186/s13059-021-02265-7</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/33608040" target="_blank"> PMID 33608040</a>. <a href="https://github.com/schmeing/ReSeq-paper" target="_blank"> GitHub repo</a>. <a href="https://github.com/schmeing/ReSeq" target="_blank"> Software</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1186/s13059-021-02265-7" data-hide-no-mentions="true">

</div></li>
<li>Wang Y, <strong>Soneson C</strong>, Malinowska AL, Laski A, Ghosh S, Kanitz A, Gebert LFR, <strong>Robinson MD</strong>, Hall J: MiR-CLIP reveals iso-miR selective regulation in the miR-124 targetome. Nucleic Acids Res 49(1):25-37 (2021). <a href="https://doi.org/10.1093/nar/gkaa1117" target="_blank"> DOI: 10.1093/nar/gkaa1117</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/33300035" target="_blank"> PMID 33300035</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1093/nar/gkaa1117" data-hide-no-mentions="true">

</div></li>
<li><strong>Mallona I</strong>, Ilie IM, Karemaker ID, Butz S, Manzo M, Caflisch A, Baubec T: Flanking sequence preference modulates de novo DNA methylation in the mouse genome. Nucleic Acids Res 49(1):145-157 (2021). <a href="https://doi.org/10.1093/nar/gkaa1168" target="_blank"> DOI: 10.1093/nar/gkaa1168</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/33290556" target="_blank"> PMID 33290556</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1093/nar/gkaa1168" data-hide-no-mentions="true">

</div></li>
<li>Bottes S, Jaeger BN, Pilz GA, Jörg DJ, Cole JD, Kruse M, Harris L, Korobeynyk VI, <strong>Mallona I</strong>, Helmchen F, Guillemot F, Simons BD, Jessberger S: Long-term self-renewing stem cells in the adult mouse hippocampus identified by intravital imaging. Nat Neurosci 24(2):225-233 (2021). <a href="https://doi.org/10.1038/s41593-020-00759-4" target="_blank"> DOI: 10.1038/s41593-020-00759-4</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/33349709" target="_blank"> PMID 33349709</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1038/s41593-020-00759-4" data-hide-no-mentions="true">

</div></li>
</ul>
</div>
<div id="section-2" class="section level2">
<h2>2020</h2>
<ul>
<li><strong>Huang R</strong>, <strong>Soneson C</strong>, Ernst FGM, Rue-Albrecht KC, Yu G, Hicks SC, <strong>Robinson MD</strong>: TreeSummarizedExperiment: a S4 class for data with hierarchical structure. F1000Res 9:1246 (2020). <a href="https://doi.org/10.12688/f1000research.26669.2" target="_blank"> DOI: 10.12688/f1000research.26669.2</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/33274053" target="_blank"> PMID 33274053</a>. <a href="https://www.bioconductor.org/packages/release/bioc/html/TreeSummarizedExperiment.html" target="_blank"> Software</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.12688/f1000research.26669.2" data-hide-no-mentions="true">

</div></li>
<li>Primo P, Meccariello A, Inghilterra MG, Gravina A, Del Corsano G, Volpe G, Sollazzo G, Aceto S, <strong>Robinson MD</strong>, Salvemini M, Saccone G: Targeting the autosomal Ceratitis capitata transformer gene using Cas9 or dCas9 to masculinize XX individuals without inducing mutations. BMC Genet 21(Suppl 2):150 (2020). <a href="https://doi.org/10.1186/s12863-020-00941-4" target="_blank"> DOI: 10.1186/s12863-020-00941-4</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/33339496" target="_blank"> PMID 33339496</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1186/s12863-020-00941-4" data-hide-no-mentions="true">

</div></li>
<li><strong>Crowell HL</strong>, <strong>Soneson C</strong>, <strong>Germain PL</strong>, Calini D, Collin L, Raposo C, Malhotra D, <strong>Robinson MD</strong>: muscat detects subpopulation-specific state transitions from multi-sample multi-condition single-cell transcriptomics data. Nat Commun 11(1):6077 (2020). <a href="https://doi.org/10.1038/s41467-020-19894-4" target="_blank"> DOI: 10.1038/s41467-020-19894-4</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/33257685" target="_blank"> PMID 33257685</a>. <a href="https://github.com/HelenaLC/muscat-comparison" target="_blank"> GitHub repo</a>. <a href="https://doi.org/10.6084/m9.figshare.8986193.v3" target="_blank"> Data</a>. <a href="http://bioconductor.org/packages/release/bioc/html/muscat.html" target="_blank"> Software</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1038/s41467-020-19894-4" data-hide-no-mentions="true">

</div></li>
<li><strong>Germain PL</strong>, <strong>Sonrel A</strong>, <strong>Robinson MD</strong>: pipeComp, a general framework for the evaluation of computational pipelines, reveals performant single cell RNA-seq preprocessing tools. Genome Biol 21(1):227 (2020). <a href="https://doi.org/10.1186/s13059-020-02136-7" target="_blank"> DOI: 10.1186/s13059-020-02136-7</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/32873325" target="_blank"> PMID 32873325</a>. <a href="https://github.com/plger/pipeComp" target="_blank"> GitHub repo</a>. <a href="http://bioconductor.org/packages/release/bioc/html/pipeComp.html" target="_blank"> Software</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1186/s13059-020-02136-7" data-hide-no-mentions="true">

</div></li>
<li><strong>Orjuela S</strong>, Machlab D, Menigatti M, Marra G, <strong>Robinson MD</strong>: DAMEfinder: a method to detect differential allele-specific methylation. Epigenetics Chromatin 13(1):25 (2020). <a href="https://doi.org/10.1186/s13072-020-00346-8" target="_blank"> DOI: 10.1186/s13072-020-00346-8</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/32487212" target="_blank"> PMID 32487212</a>. <a href="http://bioconductor.org/packages/release/bioc/html/DAMEfinder.html" target="_blank"> Software</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1186/s13072-020-00346-8" data-hide-no-mentions="true">

</div></li>
<li>Malan-Müller S, <strong>de Souza VBC</strong>, Daniels WMU, Seedat S, <strong>Robinson MD</strong>, Hemmings SMJ: Shedding Light on the Transcriptomic Dark Matter in Biological Psychiatry: Role of Long Noncoding RNAs in D-cycloserine-Induced Fear Extinction in Posttraumatic Stress Disorder. OMICS 24(6):352-369 (2020). <a href="https://doi.org/10.1089/omi.2020.0031" target="_blank"> DOI: 10.1089/omi.2020.0031</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/32453623" target="_blank"> PMID 32453623</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1089/omi.2020.0031" data-hide-no-mentions="true">

</div></li>
<li><strong>Orjuela S</strong>, Menigatti M, Schraml P, Kambakamba P, <strong>Robinson MD</strong>, Marra G: The DNA hypermethylation phenotype of colorectal cancer liver metastases resembles that of the primary colorectal cancers. BMC Cancer 20(1):290 (2020). <a href="https://doi.org/10.1186/s12885-020-06777-6" target="_blank"> DOI: 10.1186/s12885-020-06777-6</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/32252665" target="_blank"> PMID 32252665</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1186/s12885-020-06777-6" data-hide-no-mentions="true">

</div></li>
<li><strong>Tiberi S</strong>, <strong>Robinson MD</strong>: BANDITS: Bayesian differential splicing accounting for sample-to-sample variability and mapping uncertainty. Genome Biol 21(1):69 (2020). <a href="https://doi.org/10.1186/s13059-020-01967-8" target="_blank"> DOI: 10.1186/s13059-020-01967-8</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/32178699" target="_blank"> PMID 32178699</a>. <a href="https://github.com/SimoneTiberi/BANDITS_manuscript" target="_blank"> GitHub repo</a>. <a href="http://bioconductor.org/packages/release/bioc/html/BANDITS.html" target="_blank"> Software</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1186/s13059-020-01967-8" data-hide-no-mentions="true">

</div></li>
<li>Lähnemann D, Köster J, Szczurek E, McCarthy DJ, Hicks SC, <strong>Robinson MD</strong>, Vallejos CA, Campbell KR, Beerenwinkel N, Mahfouz A, Pinello L, Skums P, Stamatakis A, Attolini CS, Aparicio S, Baaijens J, Balvert M, Barbanson B, Cappuccio A, Corleone G, Dutilh BE, Florescu M, Guryev V, Holmer R, Jahn K, Lobo TJ, Keizer EM, Khatri I, Kielbasa SM, Korbel JO, Kozlov AM, Kuo TH, Lelieveldt BPF, Mandoiu II, Marioni JC, Marschall T, Mölder F, Niknejad A, Raczkowski L, Reinders M, Ridder J, Saliba AE, Somarakis A, Stegle O, Theis FJ, Yang H, Zelikovsky A, McHardy AC, Raphael BJ, Shah SP, Schönhuth A: Eleven grand challenges in single-cell data science. Genome Biol 21(1):31 (2020). <a href="https://doi.org/10.1186/s13059-020-1926-6" target="_blank"> DOI: 10.1186/s13059-020-1926-6</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/32033589" target="_blank"> PMID 32033589</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1186/s13059-020-1926-6" data-hide-no-mentions="true">

</div></li>
<li>Shaul ME, Eyal O, Guglietta S, Aloni P, Zlotnik A, Forkosh E, Levy L, <strong>Weber LM</strong>, Levin Y, Pomerantz A, Nechushtan H, Eruslanov E, Singhal S, <strong>Robinson MD</strong>, Krieg C, Fridlender ZG: Circulating neutrophil subsets in advanced lung cancer patients exhibit unique immune signature and relate to prognosis. FASEB J 34(3):4204-4218 (2020). <a href="https://doi.org/10.1096/fj.201902467R" target="_blank"> DOI: 10.1096/fj.201902467R</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/31957112" target="_blank"> PMID 31957112</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1096/fj.201902467R" data-hide-no-mentions="true">

</div></li>
</ul>
</div>
<div id="section-3" class="section level2">
<h2>2019</h2>
<ul>
<li><strong>Robinson MD</strong>, Vitek O: Benchmarking comes of age. Genome Biol 20(1):205 (2019). <a href="https://doi.org/10.1186/s13059-019-1846-5" target="_blank"> DOI: 10.1186/s13059-019-1846-5</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/31597556" target="_blank"> PMID 31597556</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1186/s13059-019-1846-5" data-hide-no-mentions="true">

</div></li>
<li>Knop J, Spilgies LM, Rufli S, Reinhart R, Vasilikos L, Yabal M, Owsley E, Jost PJ, Marsh RA, Wajant H, <strong>Robinson MD</strong>, Kaufmann T, Wong WW: TNFR2 induced priming of the inflammasome leads to a RIPK1-dependent cell death in the absence of XIAP. Cell Death Dis 10(10):700 (2019). <a href="https://doi.org/10.1038/s41419-019-1938-x" target="_blank"> DOI: 10.1038/s41419-019-1938-x</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/31541082" target="_blank"> PMID 31541082</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1038/s41419-019-1938-x" data-hide-no-mentions="true">

</div></li>
<li>Meccariello A, Salvemini M, Primo P, Hall B, Koskinioti P, Dalíková M, Gravina A, Gucciardino MA, Forlenza F, Gregoriou ME, Ippolito D, Monti SM, Petrella V, Perrotta MM, <strong>Schmeing S</strong>, Ruggiero A, Scolari F, Giordano E, Tsoumani KT, Marec F, Windbichler N, Arunkumar KP, Bourtzis K, Mathiopoulos KD, Ragoussis J, Vitagliano L, Tu Z, Papathanos PA, <strong>Robinson MD</strong>, Saccone G: <i>Maleness-on-the-Y</i> (<i>MoY</i>) orchestrates male sex determination in major agricultural fruit fly pests. Science 365(6460):1457-1460 (2019). <a href="https://doi.org/10.1126/science.aax1318" target="_blank"> DOI: 10.1126/science.aax1318</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/31467189" target="_blank"> PMID 31467189</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1126/science.aax1318" data-hide-no-mentions="true">

</div></li>
<li><strong>Soneson C</strong>, <strong>Yao Y</strong>, Bratus-Neuenschwander A, Patrignani A, <strong>Robinson MD</strong>, Hussain S: A comprehensive examination of Nanopore native RNA sequencing for characterization of complex transcriptomes. Nat Commun 10(1):3359 (2019). <a href="https://doi.org/10.1038/s41467-019-11272-z" target="_blank"> DOI: 10.1038/s41467-019-11272-z</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/31366910" target="_blank"> PMID 31366910</a>. <a href="https://github.com/csoneson/NativeRNAseqComplexTranscriptome" target="_blank"> GitHub repo</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1038/s41467-019-11272-z" data-hide-no-mentions="true">

</div></li>
<li>Bick JT, Zeng S, <strong>Robinson MD</strong>, Ulbrich SE, Bauersachs S: Mammalian Annotation Database for improved annotation and functional classification of Omics datasets from less well-annotated organisms. Database (Oxford) 2019:baz086 (2019). <a href="https://doi.org/10.1093/database/baz086" target="_blank"> DOI: 10.1093/database/baz086</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/31353404" target="_blank"> PMID 31353404</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1093/database/baz086" data-hide-no-mentions="true">

</div></li>
<li>D’Agati G, Cabello EM, Frontzek K, Rushing EJ, Klemm R, <strong>Robinson MD</strong>, White RM, Mosimann C, Burger A: Active receptor tyrosine kinases, but not Brachyury, are sufficient to trigger chordoma in zebrafish. Dis Model Mech 12(7):dmm039545 (2019). <a href="https://doi.org/10.1242/dmm.039545" target="_blank"> DOI: 10.1242/dmm.039545</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/31221659" target="_blank"> PMID 31221659</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1242/dmm.039545" data-hide-no-mentions="true">

</div></li>
<li><strong>Weber LM</strong>, Saelens W, Cannoodt R, <strong>Soneson C</strong>, Hapfelmeier A, Gardner PP, Boulesteix AL, Saeys Y, <strong>Robinson MD</strong>: Essential guidelines for computational method benchmarking. Genome Biol 20(1):125 (2019). <a href="https://doi.org/10.1186/s13059-019-1738-8" target="_blank"> DOI: 10.1186/s13059-019-1738-8</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/31221194" target="_blank"> PMID 31221194</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1186/s13059-019-1738-8" data-hide-no-mentions="true">

</div></li>
<li>Chatterjee B, Deng Y, Holler A, Nunez N, Azzi T, Vanoaica LD, Müller A, Zdimerova H, Antsiferova O, Zbinden A, Capaul R, Dreyer JH, Nadal D, Becher B, <strong>Robinson MD</strong>, Stauss H, Münz C: CD8+ T cells retain protective functions despite sustained inhibitory receptor expression during Epstein-Barr virus infection in vivo. PLoS Pathog 15(5):e1007748 (2019). <a href="https://doi.org/10.1371/journal.ppat.1007748" target="_blank"> DOI: 10.1371/journal.ppat.1007748</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/31145756" target="_blank"> PMID 31145756</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1371/journal.ppat.1007748" data-hide-no-mentions="true">

</div></li>
<li><strong>Weber LM</strong>, <strong>Nowicka M</strong>, <strong>Soneson C</strong>, <strong>Robinson MD</strong>: diffcyt: Differential discovery in high-dimensional cytometry via high-resolution clustering. Commun Biol 2:183 (2019). <a href="https://doi.org/10.1038/s42003-019-0415-5" target="_blank"> DOI: 10.1038/s42003-019-0415-5</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/31098416" target="_blank"> PMID 31098416</a>. <a href="https://github.com/lmweber/diffcyt-evaluations" target="_blank"> GitHub repo</a>. <a href="http://flowrepository.org/id/FR-FCM-ZYL8" target="_blank"> Data</a>. <a href="https://bioconductor.org/packages/release/bioc/html/diffcyt.html" target="_blank"> Software</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1038/s42003-019-0415-5" data-hide-no-mentions="true">

</div></li>
<li><strong>Orjuela S</strong>, <strong>Huang R</strong>, <strong>Hembach KM</strong>, <strong>Robinson MD</strong>, <strong>Soneson C</strong>: ARMOR: An <u>A</u>utomated <u>R</u>eproducible <u>MO</u>dular Workflow for Preprocessing and Differential Analysis of <u>R</u>NA-seq Data. G3 (Bethesda) 9(7):2089-2096 (2019). <a href="https://doi.org/10.1534/g3.119.400185" target="_blank"> DOI: 10.1534/g3.119.400185</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/31088905" target="_blank"> PMID 31088905</a>. <a href="https://github.com/csoneson/ARMOR" target="_blank"> Software</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1534/g3.119.400185" data-hide-no-mentions="true">

</div></li>
<li>Cheng HW, Onder L, Novkovic M, <strong>Soneson C</strong>, Lütge M, Pikor N, Scandella E, <strong>Robinson MD</strong>, Miyazaki JI, Tersteegen A, Sorg U, Pfeffer K, Rülicke T, Hehlgans T, Ludewig B: Origin and differentiation trajectories of fibroblastic reticular cells in the splenic white pulp. Nat Commun 10(1):1739 (2019). <a href="https://doi.org/10.1038/s41467-019-09728-3" target="_blank"> DOI: 10.1038/s41467-019-09728-3</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/30988302" target="_blank"> PMID 30988302</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1038/s41467-019-09728-3" data-hide-no-mentions="true">

</div></li>
<li>Kaiser J, Maibach M, Salpeter I, Hagenbuch N, <strong>de Souza VBC</strong>, <strong>Robinson MD</strong>, Schwab ME: The Spinal Transcriptome after Cortical Stroke: In Search of Molecular Factors Regulating Spontaneous Recovery in the Spinal Cord. J Neurosci 39(24):4714-4726 (2019). <a href="https://doi.org/10.1523/JNEUROSCI.2571-18.2019" target="_blank"> DOI: 10.1523/JNEUROSCI.2571-18.2019</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/30962276" target="_blank"> PMID 30962276</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1523/JNEUROSCI.2571-18.2019" data-hide-no-mentions="true">

</div></li>
<li>Krattinger SG, Kang J, Bräunlich S, Boni R, Chauhan H, Selter LL, <strong>Robinson MD</strong>, Schmid MW, Wiederhold E, Hensel G, Kumlehn J, Sucher J, Martinoia E, Keller B: Abscisic acid is a substrate of the ABC transporter encoded by the durable wheat disease resistance gene Lr34. New Phytol 223(2):853-866 (2019). <a href="https://doi.org/10.1111/nph.15815" target="_blank"> DOI: 10.1111/nph.15815</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/30913300" target="_blank"> PMID 30913300</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1111/nph.15815" data-hide-no-mentions="true">

</div></li>
<li><strong>Soneson C</strong>, Love MI, Patro R, Hussain S, Malhotra D, <strong>Robinson MD</strong>: A junction coverage compatibility score to quantify the reliability of transcript abundance estimates and annotation catalogs. Life Sci Alliance 2(1):e201800175 (2019). <a href="https://doi.org/10.26508/lsa.201800175" target="_blank"> DOI: 10.26508/lsa.201800175</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/30655364" target="_blank"> PMID 30655364</a>. <a href="https://github.com/csoneson/annotation_problem_txabundance" target="_blank"> GitHub repo</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.26508/lsa.201800175" data-hide-no-mentions="true">

</div></li>
<li>Doumpas N, Lampart F, <strong>Robinson MD</strong>, Lentini A, Nestor CE, Cantù C, Basler K: TCF/LEF dependent and independent transcriptional regulation of Wnt/β-catenin target genes. EMBO J 38(2):e98873 (2019). <a href="https://doi.org/10.15252/embj.201798873" target="_blank"> DOI: 10.15252/embj.201798873</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/30425074" target="_blank"> PMID 30425074</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.15252/embj.201798873" data-hide-no-mentions="true">

</div></li>
<li>Mariani SA, Li Z, Rice S, Krieg C, Fragkogianni S, <strong>Robinson M</strong>, Vink CS, Pollard JW, Dzierzak E: Pro-inflammatory Aorta-Associated Macrophages Are Involved in Embryonic Development of Hematopoietic Stem Cells. Immunity 50(6):1439-1452.e5 (2019). <a href="https://doi.org/10.1016/j.immuni.2019.05.003" target="_blank"> DOI: 10.1016/j.immuni.2019.05.003</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/31178352" target="_blank"> PMID 31178352</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1016/j.immuni.2019.05.003" data-hide-no-mentions="true">

</div></li>
<li><strong>Weber LM</strong>, <strong>Soneson C</strong>: HDCytoData: Collection of high-dimensional cytometry benchmark datasets in Bioconductor object formats. F1000Res 8:1459 (2019). <a href="https://doi.org/10.12688/f1000research.20210.2" target="_blank"> DOI: 10.12688/f1000research.20210.2</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/31857895" target="_blank"> PMID 31857895</a>. <a href="https://github.com/lmweber/HDCytoData" target="_blank"> GitHub repo</a>. <a href="http://bioconductor.org/packages/HDCytoData/" target="_blank"> Software</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.12688/f1000research.20210.2" data-hide-no-mentions="true">

</div></li>
<li>Van den Berge K, <strong>Hembach KM</strong>, <strong>Soneson C</strong>, <strong>Tiberi S</strong>, Clement L, Love MI, Patro R, <strong>Robinson MD</strong>: RNA Sequencing Data: Hitchhiker’s Guide to Expression Analysis. Annual Review of Biomedical Data Science 2:139-173 (2019). <a href="https://doi.org/10.1146/annurev-biodatasci-072018-021255" target="_blank"> DOI: 10.1146/annurev-biodatasci-072018-021255</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1146/annurev-biodatasci-072018-021255" data-hide-no-mentions="true">

</div></li>
</ul>
</div>
<div id="section-4" class="section level2">
<h2>2018</h2>
<ul>
<li>Parker HR, <strong>Orjuela S</strong>, Martinho Oliveira A, Cereatti F, Sauter M, Heinrich H, Tanzi G, Weber A, Komminoth P, Vavricka S, Albanese L, Buffoli F, <strong>Robinson MD</strong>, Marra G: The proto CpG island methylator phenotype of sessile serrated adenomas/polyps. Epigenetics 13(10-11):1088-1105 (2018). <a href="https://doi.org/10.1080/15592294.2018.1543504" target="_blank"> DOI: 10.1080/15592294.2018.1543504</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/30398409" target="_blank"> PMID 30398409</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1080/15592294.2018.1543504" data-hide-no-mentions="true">

</div></li>
<li>Villiger L, Grisch-Chan HM, <strong>Lindsay H</strong>, Ringnalda F, Pogliano CB, Allegri G, Fingerhut R, Häberle J, Matos J, <strong>Robinson MD</strong>, Thöny B, Schwank G: Treatment of a metabolic liver disease by in vivo genome base editing in adult mice. Nat Med 24(10):1519-1525 (2018). <a href="https://doi.org/10.1038/s41591-018-0209-1" target="_blank"> DOI: 10.1038/s41591-018-0209-1</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/30297904" target="_blank"> PMID 30297904</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1038/s41591-018-0209-1" data-hide-no-mentions="true">

</div></li>
<li>Perez-Shibayama C, Gil-Cruz C, Cheng HW, Onder L, Printz A, Mörbe U, Novkovic M, Li C, Lopez-Macias C, Buechler MB, Turley SJ, Mack M, <strong>Soneson C</strong>, <strong>Robinson MD</strong>, Scandella E, Gommerman J, Ludewig B: Fibroblastic reticular cells initiate immune responses in visceral adipose tissues and secure peritoneal immunity. Sci Immunol 3(26):eaar4539 (2018). <a href="https://doi.org/10.1126/sciimmunol.aar4539" target="_blank"> DOI: 10.1126/sciimmunol.aar4539</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/30097537" target="_blank"> PMID 30097537</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1126/sciimmunol.aar4539" data-hide-no-mentions="true">

</div></li>
<li>Bick JT, Flöter VL, <strong>Robinson MD</strong>, Bauersachs S, Ulbrich SE: Small RNA-seq analysis of single porcine blastocysts revealed that maternal estradiol-17beta exposure does not affect miRNA isoform (isomiR) expression. BMC Genomics 19(1):590 (2018). <a href="https://doi.org/10.1186/s12864-018-4954-9" target="_blank"> DOI: 10.1186/s12864-018-4954-9</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/30081835" target="_blank"> PMID 30081835</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1186/s12864-018-4954-9" data-hide-no-mentions="true">

</div></li>
<li><strong>Duò A</strong>, <strong>Robinson MD</strong>, <strong>Soneson C</strong>: A systematic performance evaluation of clustering methods for single-cell RNA-seq data. F1000Res 7:1141 (2018). <a href="https://doi.org/10.12688/f1000research.15666.3" target="_blank"> DOI: 10.12688/f1000research.15666.3</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/30271584" target="_blank"> PMID 30271584</a>. <a href="https://github.com/markrobinsonuzh/scRNAseq_clustering_comparison" target="_blank"> GitHub repo</a>. <a href="https://bioconductor.org/packages/DuoClustering2018/" target="_blank"> Data</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.12688/f1000research.15666.3" data-hide-no-mentions="true">

</div></li>
<li>Littringer K, Moresi C, Rakebrandt N, <strong>Zhou X</strong>, Schorer M, Dolowschiak T, Kirchner F, Rost F, Keller CW, McHugh D, LeibundGut-Landmann S, <strong>Robinson MD</strong>, Joller N: Common Features of Regulatory T Cell Specialization During Th1 Responses. Front Immunol 9:1344 (2018). <a href="https://doi.org/10.3389/fimmu.2018.01344" target="_blank"> DOI: 10.3389/fimmu.2018.01344</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/29951069" target="_blank"> PMID 29951069</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.3389/fimmu.2018.01344" data-hide-no-mentions="true">

</div></li>
<li>Savic N, Ringnalda FC, <strong>Lindsay H</strong>, Berk C, Bargsten K, Li Y, Neri D, <strong>Robinson MD</strong>, Ciaudo C, Hall J, Jinek M, Schwank G: Covalent linkage of the DNA repair template to the CRISPR-Cas9 nuclease enhances homology-directed repair. Elife 7:e33761 (2018). <a href="https://doi.org/10.7554/eLife.33761" target="_blank"> DOI: 10.7554/eLife.33761</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/29809142" target="_blank"> PMID 29809142</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.7554/eLife.33761" data-hide-no-mentions="true">

</div></li>
<li>Wrangle JM, Velcheti V, Patel MR, Garrett-Mayer E, Hill EG, Ravenel JG, Miller JS, Farhad M, Anderton K, Lindsey K, Taffaro-Neskey M, Sherman C, Suriano S, Swiderska-Syn M, Sion A, Harris J, Edwards AR, Rytlewski JA, Sanders CM, Yusko EC, <strong>Robinson MD</strong>, Krieg C, Redmond WL, Egan JO, Rhode PR, Jeng EK, Rock AD, Wong HC, Rubinstein MP: ALT-803, an IL-15 superagonist, in combination with nivolumab in patients with metastatic non-small cell lung cancer: a non-randomised, open-label, phase 1b trial. Lancet Oncol 19(5):694-704 (2018). <a href="https://doi.org/10.1016/S1470-2045(18)30148-7" target="_blank"> DOI: 10.1016/S1470-2045(18)30148-7</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/29628312" target="_blank"> PMID 29628312</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1016/S1470-2045(18)30148-7" data-hide-no-mentions="true">

</div></li>
<li>Chevrier S, <strong>Crowell HL</strong>, Zanotelli VRT, Engler S, <strong>Robinson MD</strong>, Bodenmiller B: Compensation of Signal Spillover in Suspension and Imaging Mass Cytometry. Cell Syst 6(5):612-620.e5 (2018). <a href="https://doi.org/10.1016/j.cels.2018.02.010" target="_blank"> DOI: 10.1016/j.cels.2018.02.010</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/29605184" target="_blank"> PMID 29605184</a>. <a href="http://bioconductor.org/packages/release/bioc/html/CATALYST.html" target="_blank"> Software</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1016/j.cels.2018.02.010" data-hide-no-mentions="true">

</div></li>
<li><strong>Soneson C</strong>, <strong>Robinson MD</strong>: Bias, robustness and scalability in single-cell differential expression analysis. Nat Methods 15(4):255-261 (2018). <a href="https://doi.org/10.1038/nmeth.4612" target="_blank"> DOI: 10.1038/nmeth.4612</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/29481549" target="_blank"> PMID 29481549</a>. <a href="https://github.com/csoneson/conquer_comparison" target="_blank"> GitHub repo</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1038/nmeth.4612" data-hide-no-mentions="true">

</div></li>
<li>Van den Berge K, Perraudeau F, <strong>Soneson C</strong>, Love MI, Risso D, Vert JP, <strong>Robinson MD</strong>, Dudoit S, Clement L: Observation weights unlock bulk RNA-seq tools for zero inflation and single-cell applications. Genome Biol 19(1):24 (2018). <a href="https://doi.org/10.1186/s13059-018-1406-4" target="_blank"> DOI: 10.1186/s13059-018-1406-4</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/29478411" target="_blank"> PMID 29478411</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1186/s13059-018-1406-4" data-hide-no-mentions="true">

</div></li>
<li>Praz CR, Menardo F, <strong>Robinson MD</strong>, Müller MC, Wicker T, Bourras S, Keller B: Non-parent of Origin Expression of Numerous Effector Genes Indicates a Role of Gene Regulation in Host Adaption of the Hybrid Triticale Powdery Mildew Pathogen. Front Plant Sci 9:49 (2018). <a href="https://doi.org/10.3389/fpls.2018.00049" target="_blank"> DOI: 10.3389/fpls.2018.00049</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/29441081" target="_blank"> PMID 29441081</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.3389/fpls.2018.00049" data-hide-no-mentions="true">

</div></li>
<li>Krieg C, <strong>Nowicka M</strong>, Guglietta S, Schindler S, Hartmann FJ, <strong>Weber LM</strong>, Dummer R, <strong>Robinson MD</strong>, Levesque MP, Becher B: High-dimensional single-cell analysis predicts response to anti-PD-1 immunotherapy. Nat Med 24(2):144-153 (2018). <a href="https://doi.org/10.1038/nm.4466" target="_blank"> DOI: 10.1038/nm.4466</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/29309059" target="_blank"> PMID 29309059</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1038/nm.4466" data-hide-no-mentions="true">

</div></li>
<li><strong>Soneson C</strong>, <strong>Robinson MD</strong>: Towards unified quality verification of synthetic count data with countsimQC. Bioinformatics 34(4):691-692 (2018). <a href="https://doi.org/10.1093/bioinformatics/btx631" target="_blank"> DOI: 10.1093/bioinformatics/btx631</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/29028961" target="_blank"> PMID 29028961</a>. <a href="https://www.bioconductor.org/packages/countsimQC/" target="_blank"> Software</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1093/bioinformatics/btx631" data-hide-no-mentions="true">

</div></li>
<li>Rue-Albrecht K, Marini F, <strong>Soneson C</strong>, Lun ATL: iSEE: Interactive SummarizedExperiment Explorer. F1000Res 7:741 (2018). <a href="https://doi.org/10.12688/f1000research.14966.1" target="_blank"> DOI: 10.12688/f1000research.14966.1</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/30002819" target="_blank"> PMID 30002819</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.12688/f1000research.14966.1" data-hide-no-mentions="true">

</div></li>
<li>Love MI, <strong>Soneson C</strong>, Patro R: Swimming downstream: statistical analysis of differential transcript usage following Salmon quantification. F1000Res 7:952 (2018). <a href="https://doi.org/10.12688/f1000research.15398.3" target="_blank"> DOI: 10.12688/f1000research.15398.3</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/30356428" target="_blank"> PMID 30356428</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.12688/f1000research.15398.3" data-hide-no-mentions="true">

</div></li>
</ul>
</div>
<div id="section-5" class="section level2">
<h2>2017</h2>
<ul>
<li>Omasits U, Varadarajan AR, Schmid M, Goetze S, Melidis D, Bourqui M, <strong>Nikolayeva O</strong>, Québatte M, Patrignani A, Dehio C, Frey JE, <strong>Robinson MD</strong>, Wollscheid B, Ahrens CH: An integrative strategy to identify the entire protein coding potential of prokaryotic genomes by proteogenomics. Genome Res 27(12):2083-2095 (2017). <a href="https://doi.org/10.1101/gr.218255.116" target="_blank"> DOI: 10.1101/gr.218255.116</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/29141959" target="_blank"> PMID 29141959</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1101/gr.218255.116" data-hide-no-mentions="true">

</div></li>
<li>Meccariello A, Monti SM, Romanelli A, Colonna R, Primo P, Inghilterra MG, Del Corsano G, Ramaglia A, Iazzetti G, Chiarore A, Patti F, Heinze SD, Salvemini M, <strong>Lindsay H</strong>, Chiavacci E, Burger A, <strong>Robinson MD</strong>, Mosimann C, Bopp D, Saccone G: Highly efficient DNA-free gene disruption in the agricultural pest Ceratitis capitata by CRISPR-Cas9 ribonucleoprotein complexes. Sci Rep 7(1):10061 (2017). <a href="https://doi.org/10.1038/s41598-017-10347-5" target="_blank"> DOI: 10.1038/s41598-017-10347-5</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/28855635" target="_blank"> PMID 28855635</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1038/s41598-017-10347-5" data-hide-no-mentions="true">

</div></li>
<li>Van den Berge K, <strong>Soneson C</strong>, <strong>Robinson MD</strong>, Clement L: stageR: a general stage-wise method for controlling the gene-level false discovery rate in differential expression and differential transcript usage. Genome Biol 18(1):151 (2017). <a href="https://doi.org/10.1186/s13059-017-1277-0" target="_blank"> DOI: 10.1186/s13059-017-1277-0</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/28784146" target="_blank"> PMID 28784146</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1186/s13059-017-1277-0" data-hide-no-mentions="true">

</div></li>
<li>Herdman C, Mars JC, Stefanovsky VY, Tremblay MG, Sabourin-Felix M, <strong>Lindsay H</strong>, <strong>Robinson MD</strong>, Moss T: A unique enhancer boundary complex on the mouse ribosomal RNA genes persists after loss of Rrn3 or UBF and the inactivation of RNA polymerase I transcription. PLoS Genet 13(7):e1006899 (2017). <a href="https://doi.org/10.1371/journal.pgen.1006899" target="_blank"> DOI: 10.1371/journal.pgen.1006899</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/28715449" target="_blank"> PMID 28715449</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1371/journal.pgen.1006899" data-hide-no-mentions="true">

</div></li>
<li><strong>Nowicka M</strong>, Krieg C, <strong>Crowell HL</strong>, <strong>Weber LM</strong>, Hartmann FJ, Guglietta S, Becher B, Levesque MP, <strong>Robinson MD</strong>: CyTOF workflow: differential discovery in high-throughput high-dimensional cytometry datasets. F1000Res 6:748 (2017). <a href="https://doi.org/10.12688/f1000research.11622.3" target="_blank"> DOI: 10.12688/f1000research.11622.3</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/28663787" target="_blank"> PMID 28663787</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.12688/f1000research.11622.3" data-hide-no-mentions="true">

</div></li>
<li>Sharma A, Heinze SD, Wu Y, Kohlbrenner T, <strong>Morilla I</strong>, Brunner C, Wimmer EA, van de Zande L, <strong>Robinson MD</strong>, Beukeboom LW, Bopp D: Male sex in houseflies is determined by <i>Mdmd</i>, a paralog of the generic splice factor gene <i>CWC22</i>. Science 356(6338):642-645 (2017). <a href="https://doi.org/10.1126/science.aam5498" target="_blank"> DOI: 10.1126/science.aam5498</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/28495751" target="_blank"> PMID 28495751</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1126/science.aam5498" data-hide-no-mentions="true">

</div></li>
<li>Marinova Z, Maercker A, Küffer A, <strong>Robinson MD</strong>, Wojdacz TK, Walitza S, Grünblatt E, Burri A: DNA methylation profiles of elderly individuals subjected to indentured childhood labor and trauma. BMC Med Genet 18(1):21 (2017). <a href="https://doi.org/10.1186/s12881-017-0370-2" target="_blank"> DOI: 10.1186/s12881-017-0370-2</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/28241754" target="_blank"> PMID 28241754</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1186/s12881-017-0370-2" data-hide-no-mentions="true">

</div></li>
<li>Sauerwald J, <strong>Soneson C</strong>, <strong>Robinson MD</strong>, Luschnig S: Faithful mRNA splicing depends on the Prp19 complex subunit <i>faint sausage</i> and is required for tracheal branching morphogenesis in <i>Drosophila</i>. Development 144(4):657-663 (2017). <a href="https://doi.org/10.1242/dev.144535" target="_blank"> DOI: 10.1242/dev.144535</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/28087625" target="_blank"> PMID 28087625</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1242/dev.144535" data-hide-no-mentions="true">

</div></li>
</ul>
</div>
<div id="section-6" class="section level2">
<h2>2016</h2>
<ul>
<li><strong>Nowicka M</strong>, <strong>Robinson MD</strong>: DRIMSeq: a Dirichlet-multinomial framework for multivariate count outcomes in genomics. F1000Res 5:1356 (2016). <a href="https://doi.org/10.12688/f1000research.8900.2" target="_blank"> DOI: 10.12688/f1000research.8900.2</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/28105305" target="_blank"> PMID 28105305</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.12688/f1000research.8900.2" data-hide-no-mentions="true">

</div></li>
<li><strong>Weber LM</strong>, <strong>Robinson MD</strong>: Comparison of clustering methods for high-dimensional single-cell flow and mass cytometry data. Cytometry A 89(12):1084-1096 (2016). <a href="https://doi.org/10.1002/cyto.a.23030" target="_blank"> DOI: 10.1002/cyto.a.23030</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/27992111" target="_blank"> PMID 27992111</a>. <a href="https://github.com/lmweber/cytometry-clustering-comparison" target="_blank"> GitHub repo</a>. <a href="http://flowrepository.org/id/FR-FCM-ZZPH" target="_blank"> Data</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1002/cyto.a.23030" data-hide-no-mentions="true">

</div></li>
<li>Burri A, Marinova Z, <strong>Robinson MD</strong>, Kühnel B, Waldenberger M, Wahl S, Kunze S, Gieger C, Livshits G, Williams F: Are Epigenetic Factors Implicated in Chronic Widespread Pain? PLoS One 11(11):e0165548 (2016). <a href="https://doi.org/10.1371/journal.pone.0165548" target="_blank"> DOI: 10.1371/journal.pone.0165548</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/27832094" target="_blank"> PMID 27832094</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1371/journal.pone.0165548" data-hide-no-mentions="true">

</div></li>
<li>Hartmann FJ, Bernard-Valnet R, Quériault C, Mrdjen D, <strong>Weber LM</strong>, Galli E, Krieg C, <strong>Robinson MD</strong>, Nguyen XH, Dauvilliers Y, Liblau RS, Becher B: High-dimensional single-cell analysis reveals the immune signature of narcolepsy. J Exp Med 213(12):2621-2633 (2016). <a href="https://doi.org/10.1084/jem.20160897" target="_blank"> DOI: 10.1084/jem.20160897</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/27821550" target="_blank"> PMID 27821550</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1084/jem.20160897" data-hide-no-mentions="true">

</div></li>
<li>Hawkins ED, Duarte D, Akinduro O, Khorshed RA, Passaro D, <strong>Nowicka M</strong>, Straszkowski L, Scott MK, Rothery S, Ruivo N, Foster K, Waibel M, Johnstone RW, Harrison SJ, Westerman DA, Quach H, Gribben J, <strong>Robinson MD</strong>, Purton LE, Bonnet D, Lo Celso C: T-cell acute leukaemia exhibits dynamic interactions with bone marrow microenvironments. Nature 538(7626):518-522 (2016). <a href="https://doi.org/10.1038/nature19801" target="_blank"> DOI: 10.1038/nature19801</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/27750279" target="_blank"> PMID 27750279</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1038/nature19801" data-hide-no-mentions="true">

</div></li>
<li><strong>Lindsay H</strong>, Burger A, <strong>Biyong B</strong>, Felker A, Hess C, Zaugg J, Chiavacci E, Anders C, Jinek M, Mosimann C, <strong>Robinson MD</strong>: CrispRVariants charts the mutation spectrum of genome engineering experiments. Nat Biotechnol 34(7):701-2 (2016). <a href="https://doi.org/10.1038/nbt.3628" target="_blank"> DOI: 10.1038/nbt.3628</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/27404876" target="_blank"> PMID 27404876</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1038/nbt.3628" data-hide-no-mentions="true">

</div></li>
<li>Burger A, <strong>Lindsay H</strong>, Felker A, Hess C, Anders C, Chiavacci E, Zaugg J, <strong>Weber LM</strong>, Catena R, Jinek M, <strong>Robinson MD</strong>, Mosimann C: Maximizing mutagenesis with solubilized CRISPR-Cas9 ribonucleoprotein complexes. Development 143(11):2025-37 (2016). <a href="https://doi.org/10.1242/dev.134809" target="_blank"> DOI: 10.1242/dev.134809</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/27130213" target="_blank"> PMID 27130213</a>. <a href="http://lmweber.github.io/CrispantCal/" target="_blank"> Software</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1242/dev.134809" data-hide-no-mentions="true">

</div></li>
<li><strong>Soneson C</strong>, <strong>Robinson MD</strong>: iCOBRA: open, reproducible, standardized and live method benchmarking. Nat Methods 13(4):283 (2016). <a href="https://doi.org/10.1038/nmeth.3805" target="_blank"> DOI: 10.1038/nmeth.3805</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/27027585" target="_blank"> PMID 27027585</a>. <a href="http://bioconductor.org/packages/release/bioc/html/iCOBRA.html" target="_blank"> Software</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1038/nmeth.3805" data-hide-no-mentions="true">

</div></li>
<li><strong>Soneson C</strong>, <strong>Matthes KL</strong>, <strong>Nowicka M</strong>, <strong>Law CW</strong>, <strong>Robinson MD</strong>: Isoform prefiltering improves performance of count-based methods for analysis of differential transcript usage. Genome Biol 17:12 (2016). <a href="https://doi.org/10.1186/s13059-015-0862-3" target="_blank"> DOI: 10.1186/s13059-015-0862-3</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/26813113" target="_blank"> PMID 26813113</a>. <a href="https://github.com/markrobinsonuzh/diff_splice_paper" target="_blank"> GitHub repo</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1186/s13059-015-0862-3" data-hide-no-mentions="true">

</div></li>
<li>Flori M, Schmid CA, Sumrall ET, Tzankov A, <strong>Law CW</strong>, <strong>Robinson MD</strong>, Müller A: The hematopoietic oncoprotein FOXP1 promotes tumor cell survival in diffuse large B-cell lymphoma by repressing S1PR2 signaling. Blood 127(11):1438-48 (2016). <a href="https://doi.org/10.1182/blood-2015-08-662635" target="_blank"> DOI: 10.1182/blood-2015-08-662635</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/26729899" target="_blank"> PMID 26729899</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1182/blood-2015-08-662635" data-hide-no-mentions="true">

</div></li>
</ul>
</div>
<div id="section-7" class="section level2">
<h2>2015</h2>
<ul>
<li><strong>Soneson C</strong>, Love MI, <strong>Robinson MD</strong>: Differential analyses for RNA-seq: transcript-level estimates improve gene-level inferences. F1000Res 4:1521 (2015). <a href="https://doi.org/10.12688/f1000research.7563.2" target="_blank"> DOI: 10.12688/f1000research.7563.2</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/26925227" target="_blank"> PMID 26925227</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.12688/f1000research.7563.2" data-hide-no-mentions="true">

</div></li>
<li><strong>Zhou X</strong>, <strong>Robinson MD</strong>: Do count-based differential expression methods perform poorly when genes are expressed in only one condition? Genome Biol 16:222 (2015). <a href="https://doi.org/10.1186/s13059-015-0781-3" target="_blank"> DOI: 10.1186/s13059-015-0781-3</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/26450178" target="_blank"> PMID 26450178</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1186/s13059-015-0781-3" data-hide-no-mentions="true">

</div></li>
<li>Bilgin Sonay T, Carvalho T, <strong>Robinson MD</strong>, Greminger MP, Krützen M, Comas D, Highnam G, Mittelman D, Sharp A, Marques-Bonet T, Wagner A: Tandem repeat variation in human and great ape populations and its impact on gene expression divergence. Genome Res 25(11):1591-9 (2015). <a href="https://doi.org/10.1101/gr.190868.115" target="_blank"> DOI: 10.1101/gr.190868.115</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/26290536" target="_blank"> PMID 26290536</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1101/gr.190868.115" data-hide-no-mentions="true">

</div></li>
<li>Locke WJ, Zotenko E, Stirzaker C, <strong>Robinson MD</strong>, Hinshelwood RA, Stone A, Reddel RR, Huschtscha LI, Clark SJ: Coordinated epigenetic remodelling of transcriptional networks occurs during early breast carcinogenesis. Clin Epigenetics 7:52 (2015). <a href="https://doi.org/10.1186/s13148-015-0086-0" target="_blank"> DOI: 10.1186/s13148-015-0086-0</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/25960784" target="_blank"> PMID 25960784</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1186/s13148-015-0086-0" data-hide-no-mentions="true">

</div></li>
<li>Schmid CA, <strong>Robinson MD</strong>, Scheifinger NA, Müller S, Cogliatti S, Tzankov A, Müller A: DUSP4 deficiency caused by promoter hypermethylation drives JNK signaling and tumor cell survival in diffuse large B cell lymphoma. J Exp Med 212(5):775-92 (2015). <a href="https://doi.org/10.1084/jem.20141957" target="_blank"> DOI: 10.1084/jem.20141957</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/25847947" target="_blank"> PMID 25847947</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1084/jem.20141957" data-hide-no-mentions="true">

</div></li>
<li><strong>Robinson MD</strong>, Pelizzola M: Computational epigenomics: challenges and opportunities. Front Genet 6:88 (2015). <a href="https://doi.org/10.3389/fgene.2015.00088" target="_blank"> DOI: 10.3389/fgene.2015.00088</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/25798147" target="_blank"> PMID 25798147</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.3389/fgene.2015.00088" data-hide-no-mentions="true">

</div></li>
<li>Stirzaker C, Zotenko E, Song JZ, Qu W, Nair SS, Locke WJ, Stone A, Armstong NJ, <strong>Robinson MD</strong>, Dobrovic A, Avery-Kiejda KA, Peters KM, French JD, Stein S, Korbie DJ, Trau M, Forbes JF, Scott RJ, Brown MA, Francis GD, Clark SJ: Methylome sequencing in triple-negative breast cancer reveals distinct methylation clusters with prognostic value. Nat Commun 6:5899 (2015). <a href="https://doi.org/10.1038/ncomms6899" target="_blank"> DOI: 10.1038/ncomms6899</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/25641231" target="_blank"> PMID 25641231</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1038/ncomms6899" data-hide-no-mentions="true">

</div></li>
<li>Baker EK, Taylor S, Gupte A, Chalk AM, Bhattacharya S, Green AC, Martin TJ, Strbenac D, <strong>Robinson MD</strong>, Purton LE, Walkley CR: Wnt inhibitory factor 1 (WIF1) is a marker of osteoblastic differentiation stage and is not silenced by DNA methylation in osteosarcoma. Bone 73:223-32 (2015). <a href="https://doi.org/10.1016/j.bone.2014.12.063" target="_blank"> DOI: 10.1016/j.bone.2014.12.063</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/25571841" target="_blank"> PMID 25571841</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1016/j.bone.2014.12.063" data-hide-no-mentions="true">

</div></li>
<li>Kulic I, Robertson G, Chang L, Baker JH, Lockwood WW, Mok W, Fuller M, Fournier M, Wong N, Chou V, <strong>Robinson MD</strong>, Chun HJ, Gilks B, Kempkes B, Thomson TA, Hirst M, Minchinton AI, Lam WL, Jones S, Marra M, Karsan A: Loss of the Notch effector RBPJ promotes tumorigenesis. J Exp Med 212(1):37-52 (2015). <a href="https://doi.org/10.1084/jem.20121192" target="_blank"> DOI: 10.1084/jem.20121192</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/25512468" target="_blank"> PMID 25512468</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1084/jem.20121192" data-hide-no-mentions="true">

</div></li>
<li>Gu L, Frommel SC, Oakes CC, Simon R, Grupp K, Gerig CY, Bär D, <strong>Robinson MD</strong>, Baer C, Weiss M, Gu Z, Schapira M, Kuner R, Sültmann H, Provenzano M, ICGC Project on Early Onset Prostate Cancer., Yaspo ML, Brors B, Korbel J, Schlomm T, Sauter G, Eils R, Plass C, Santoro R: BAZ2A (TIP5) is involved in epigenetic alterations in prostate cancer and its overexpression predicts disease recurrence. Nat Genet 47(1):22-30 (2015). <a href="https://doi.org/10.1038/ng.3165" target="_blank"> DOI: 10.1038/ng.3165</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/25485837" target="_blank"> PMID 25485837</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1038/ng.3165" data-hide-no-mentions="true">

</div></li>
<li>Petrella V, Aceto S, Musacchia F, Colonna V, <strong>Robinson M</strong>, Benes V, Cicotti G, Bongiorno G, Gradoni L, Volf P, Salvemini M: De novo assembly and sex-specific transcriptome profiling in the sand fly Phlebotomus perniciosus (Diptera, Phlebotominae), a major Old World vector of Leishmania infantum. BMC Genomics 16:847 (2015). <a href="https://doi.org/10.1186/s12864-015-2088-x" target="_blank"> DOI: 10.1186/s12864-015-2088-x</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/26493315" target="_blank"> PMID 26493315</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1186/s12864-015-2088-x" data-hide-no-mentions="true">

</div></li>
</ul>
</div>
<div id="section-8" class="section level2">
<h2>2014</h2>
<ul>
<li><strong>Robinson MD</strong>, Kahraman A, <strong>Law CW</strong>, <strong>Lindsay H</strong>, <strong>Nowicka M</strong>, <strong>Weber LM</strong>, <strong>Zhou X</strong>: Statistical methods for detecting differentially methylated loci and regions. Front Genet 5:324 (2014). <a href="https://doi.org/10.3389/fgene.2014.00324" target="_blank"> DOI: 10.3389/fgene.2014.00324</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/25278959" target="_blank"> PMID 25278959</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.3389/fgene.2014.00324" data-hide-no-mentions="true">

</div></li>
<li>Hickey PF, <strong>Robinson MD</strong>: Genomics by the beach. Genome Biol 15(4):304 (2014). <a href="https://doi.org/10.1186/gb4171" target="_blank"> DOI: 10.1186/gb4171</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/25001045" target="_blank"> PMID 25001045</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1186/gb4171" data-hide-no-mentions="true">

</div></li>
<li>Almouzni G, Altucci L, Amati B, Ashley N, Baulcombe D, Beaujean N, Bock C, Bongcam-Rudloff E, Bousquet J, Braun S, Bressac-de Paillerets B, Bussemakers M, Clarke L, Conesa A, Estivill X, Fazeli A, Grgurević N, Gut I, Heijmans BT, Hermouet S, Houwing-Duistermaat J, Iacobucci I, Ilaš J, Kandimalla R, Krauss-Etschmann S, Lasko P, Lehmann S, Lindroth A, Majdič G, Marcotte E, Martinelli G, Martinet N, Meyer E, Miceli C, Mills K, Moreno-Villanueva M, Morvan G, Nickel D, Niesler B, Nowacki M, Nowak J, Ossowski S, Pelizzola M, Pochet R, Potočnik U, Radwanska M, Raes J, Rattray M, <strong>Robinson MD</strong>, Roelen B, Sauer S, Schinzer D, Slagboom E, Spector T, Stunnenberg HG, Tiligada E, Torres-Padilla ME, Tsonaka R, Van Soom A, Vidaković M, Widschwendter M: Relationship between genome and epigenome–challenges and requirements for future research. BMC Genomics 15:487 (2014). <a href="https://doi.org/10.1186/1471-2164-15-487" target="_blank"> DOI: 10.1186/1471-2164-15-487</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/24942464" target="_blank"> PMID 24942464</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1186/1471-2164-15-487" data-hide-no-mentions="true">

</div></li>
<li><strong>Zhou X</strong>, <strong>Lindsay H</strong>, <strong>Robinson MD</strong>: Robustly detecting differential expression in RNA sequencing data using observation weights. Nucleic Acids Res 42(11):e91 (2014). <a href="https://doi.org/10.1093/nar/gku310" target="_blank"> DOI: 10.1093/nar/gku310</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/24753412" target="_blank"> PMID 24753412</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1093/nar/gku310" data-hide-no-mentions="true">

</div></li>
<li><strong>Nikolayeva O</strong>, <strong>Robinson MD</strong>: edgeR for differential RNA-seq and ChIP-seq analysis: an application to stem cell biology. Methods Mol Biol 1150:45-79 (2014). <a href="https://doi.org/10.1007/978-1-4939-0512-6_3" target="_blank"> DOI: 10.1007/978-1-4939-0512-6_3</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/24743990" target="_blank"> PMID 24743990</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1007/978-1-4939-0512-6_3" data-hide-no-mentions="true">

</div></li>
<li><strong>Riebler A</strong>, Menigatti M, Song JZ, Statham AL, Stirzaker C, Mahmud N, Mein CA, Clark SJ, <strong>Robinson MD</strong>: BayMeth: improved DNA methylation quantification for affinity capture sequencing data using a flexible Bayesian approach. Genome Biol 15(2):R35 (2014). <a href="https://doi.org/10.1186/gb-2014-15-2-r35" target="_blank"> DOI: 10.1186/gb-2014-15-2-r35</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/24517713" target="_blank"> PMID 24517713</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1186/gb-2014-15-2-r35" data-hide-no-mentions="true">

</div></li>
<li>Cai H, Kumar N, Bagheri HC, von Mering C, <strong>Robinson MD</strong>, Baudis M: Chromothripsis-like patterns are recurring but heterogeneously distributed features in a survey of 22,347 cancer genome screens. BMC Genomics 15:82 (2014). <a href="https://doi.org/10.1186/1471-2164-15-82" target="_blank"> DOI: 10.1186/1471-2164-15-82</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/24476156" target="_blank"> PMID 24476156</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1186/1471-2164-15-82" data-hide-no-mentions="true">

</div></li>
</ul>
</div>
<div id="section-9" class="section level2">
<h2>2013</h2>
<ul>
<li>Maier L, Vyas R, Cordova CD, <strong>Lindsay H</strong>, Schmidt TS, Brugiroux S, Periaswamy B, Bauer R, Sturm A, Schreiber F, von Mering C, <strong>Robinson MD</strong>, Stecher B, Hardt WD: Microbiota-derived hydrogen fuels Salmonella typhimurium invasion of the gut ecosystem. Cell Host Microbe 14(6):641-51 (2013). <a href="https://doi.org/10.1016/j.chom.2013.11.002" target="_blank"> DOI: 10.1016/j.chom.2013.11.002</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/24331462" target="_blank"> PMID 24331462</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1016/j.chom.2013.11.002" data-hide-no-mentions="true">

</div></li>
<li>Anders S, McCarthy DJ, Chen Y, Okoniewski M, Smyth GK, Huber W, <strong>Robinson MD</strong>: Count-based differential expression analysis of RNA sequencing data using R and Bioconductor. Nat Protoc 8(9):1765-86 (2013). <a href="https://doi.org/10.1038/nprot.2013.099" target="_blank"> DOI: 10.1038/nprot.2013.099</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/23975260" target="_blank"> PMID 23975260</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1038/nprot.2013.099" data-hide-no-mentions="true">

</div></li>
<li>Omasits U, Quebatte M, Stekhoven DJ, Fortes C, Roschitzki B, <strong>Robinson MD</strong>, Dehio C, Ahrens CH: Directed shotgun proteomics guided by saturated RNA-seq identifies a complete expressed prokaryotic proteome. Genome Res 23(11):1916-27 (2013). <a href="https://doi.org/10.1101/gr.151035.112" target="_blank"> DOI: 10.1101/gr.151035.112</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/23878158" target="_blank"> PMID 23878158</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1101/gr.151035.112" data-hide-no-mentions="true">

</div></li>
<li><strong>Zhou X</strong>, Oshlack A, <strong>Robinson MD</strong>: miRNA-Seq normalization comparisons need improvement. RNA 19(6):733-4 (2013). <a href="https://doi.org/10.1261/rna.037895.112" target="_blank"> DOI: 10.1261/rna.037895.112</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/23616640" target="_blank"> PMID 23616640</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1261/rna.037895.112" data-hide-no-mentions="true">

</div></li>
<li>Bert SA, <strong>Robinson MD</strong>, Strbenac D, Statham AL, Song JZ, Hulf T, Sutherland RL, Coolen MW, Stirzaker C, Clark SJ: Regional activation of the cancer genome by long-range epigenetic remodeling. Cancer Cell 23(1):9-22 (2013). <a href="https://doi.org/10.1016/j.ccr.2012.11.006" target="_blank"> DOI: 10.1016/j.ccr.2012.11.006</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/23245995" target="_blank"> PMID 23245995</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1016/j.ccr.2012.11.006" data-hide-no-mentions="true">

</div></li>
<li>Menigatti M, Staiano T, Manser CN, Bauerfeind P, <strong>Komljenovic A</strong>, <strong>Robinson M</strong>, Jiricny J, Buffoli F, Marra G: Epigenetic silencing of monoallelically methylated miRNA loci in precancerous colorectal lesions. Oncogenesis 2:e56 (2013). <a href="https://doi.org/10.1038/oncsis.2013.21" target="_blank"> DOI: 10.1038/oncsis.2013.21</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/23857251" target="_blank"> PMID 23857251</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1038/oncsis.2013.21" data-hide-no-mentions="true">

</div></li>
</ul>
</div>
<div id="section-10" class="section level2">
<h2>2012</h2>
<ul>
<li><strong>Robinson MD</strong>, Strbenac D, Stirzaker C, Statham AL, Song J, Speed TP, Clark SJ: Copy-number-aware differential analysis of quantitative DNA sequencing data. Genome Res 22(12):2489-96 (2012). <a href="https://doi.org/10.1101/gr.139055.112" target="_blank"> DOI: 10.1101/gr.139055.112</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/22879430" target="_blank"> PMID 22879430</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1101/gr.139055.112" data-hide-no-mentions="true">

</div></li>
<li>Fiume M, Smith EJ, Brook A, Strbenac D, Turner B, Mezlini AM, <strong>Robinson MD</strong>, Wodak SJ, Brudno M: Savant Genome Browser 2: visualization and analysis for population-scale genomics. Nucleic Acids Res 40(Web Server issue):W615-21 (2012). <a href="https://doi.org/10.1093/nar/gks427" target="_blank"> DOI: 10.1093/nar/gks427</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/22638571" target="_blank"> PMID 22638571</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1093/nar/gks427" data-hide-no-mentions="true">

</div></li>
<li>Statham AL, <strong>Robinson MD</strong>, Song JZ, Coolen MW, Stirzaker C, Clark SJ: Bisulfite sequencing of chromatin immunoprecipitated DNA (BisChIP-seq) directly informs methylation status of histone-modified DNA. Genome Res 22(6):1120-7 (2012). <a href="https://doi.org/10.1101/gr.132076.111" target="_blank"> DOI: 10.1101/gr.132076.111</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/22466171" target="_blank"> PMID 22466171</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1101/gr.132076.111" data-hide-no-mentions="true">

</div></li>
<li>Valdés-Mora F, Song JZ, Statham AL, Strbenac D, <strong>Robinson MD</strong>, Nair SS, Patterson KI, Tremethick DJ, Stirzaker C, Clark SJ: Acetylation of H2A.Z is a key epigenetic modification associated with gene deregulation and epigenetic remodeling in cancer. Genome Res 22(2):307-21 (2012). <a href="https://doi.org/10.1101/gr.118919.110" target="_blank"> DOI: 10.1101/gr.118919.110</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/21788347" target="_blank"> PMID 21788347</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1101/gr.118919.110" data-hide-no-mentions="true">

</div></li>
</ul>
</div>
<div id="section-11" class="section level2">
<h2>2011</h2>
<ul>
<li>Siebert S, <strong>Robinson MD</strong>, Tintori SC, Goetz F, Helm RR, Smith SA, Shaner N, Haddock SH, Dunn CW: Differential gene expression in the siphonophore Nanomia bijuga (Cnidaria) assessed with multiple next-generation sequencing workflows. PLoS One 6(7):e22953 (2011). <a href="https://doi.org/10.1371/journal.pone.0022953" target="_blank"> DOI: 10.1371/journal.pone.0022953</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/21829563" target="_blank"> PMID 21829563</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1371/journal.pone.0022953" data-hide-no-mentions="true">

</div></li>
<li>Hulf T, Sibbritt T, Wiklund ED, Bert S, Strbenac D, Statham AL, <strong>Robinson MD</strong>, Clark SJ: Discovery pipeline for epigenetically deregulated miRNAs in cancer: integration of primary miRNA transcription. BMC Genomics 12:54 (2011). <a href="https://doi.org/10.1186/1471-2164-12-54" target="_blank"> DOI: 10.1186/1471-2164-12-54</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/21255435" target="_blank"> PMID 21255435</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1186/1471-2164-12-54" data-hide-no-mentions="true">

</div></li>
<li>Nair SS, Coolen MW, Stirzaker C, Song JZ, Statham AL, Strbenac D, <strong>Robinson MD</strong>, Clark SJ: Comparison of methyl-DNA immunoprecipitation (MeDIP) and methyl-CpG binding domain (MBD) protein capture for genome-wide DNA methylation analysis reveal CpG sequence coverage bias. Epigenetics 6(1):34-44 (2011). <a href="https://doi.org/10.4161/epi.6.1.13313" target="_blank"> DOI: 10.4161/epi.6.1.13313</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/20818161" target="_blank"> PMID 20818161</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.4161/epi.6.1.13313" data-hide-no-mentions="true">

</div></li>
</ul>
</div>
<div id="section-12" class="section level2">
<h2>2010</h2>
<ul>
<li><strong>Robinson MD</strong>, Statham AL, Speed TP, Clark SJ: Protocol matters: which methylome are you actually studying? Epigenomics 2(4):587-98 (2010). <a href="https://doi.org/10.2217/epi.10.36" target="_blank"> DOI: 10.2217/epi.10.36</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/21566704" target="_blank"> PMID 21566704</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.2217/epi.10.36" data-hide-no-mentions="true">

</div></li>
<li>Oshlack A, <strong>Robinson MD</strong>, Young MD: From RNA-seq reads to differential expression results. Genome Biol 11(12):220 (2010). <a href="https://doi.org/10.1186/gb-2010-11-12-220" target="_blank"> DOI: 10.1186/gb-2010-11-12-220</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/21176179" target="_blank"> PMID 21176179</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1186/gb-2010-11-12-220" data-hide-no-mentions="true">

</div></li>
<li><strong>Robinson MD</strong>, Stirzaker C, Statham AL, Coolen MW, Song JZ, Nair SS, Strbenac D, Speed TP, Clark SJ: Evaluation of affinity-based genome-wide DNA methylation data: effects of CpG density, amplification bias, and copy number variation. Genome Res 20(12):1719-29 (2010). <a href="https://doi.org/10.1101/gr.110601.110" target="_blank"> DOI: 10.1101/gr.110601.110</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/21045081" target="_blank"> PMID 21045081</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1101/gr.110601.110" data-hide-no-mentions="true">

</div></li>
<li>Statham AL, Strbenac D, Coolen MW, Stirzaker C, Clark SJ, <strong>Robinson MD</strong>: Repitools: an R package for the analysis of enrichment-based epigenomic data. Bioinformatics 26(13):1662-3 (2010). <a href="https://doi.org/10.1093/bioinformatics/btq247" target="_blank"> DOI: 10.1093/bioinformatics/btq247</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/20457667" target="_blank"> PMID 20457667</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1093/bioinformatics/btq247" data-hide-no-mentions="true">

</div></li>
<li><strong>Robinson MD</strong>, Oshlack A: A scaling normalization method for differential expression analysis of RNA-seq data. Genome Biol 11(3):R25 (2010). <a href="https://doi.org/10.1186/gb-2010-11-3-r25" target="_blank"> DOI: 10.1186/gb-2010-11-3-r25</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/20196867" target="_blank"> PMID 20196867</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1186/gb-2010-11-3-r25" data-hide-no-mentions="true">

</div></li>
<li>Coolen MW, Stirzaker C, Song JZ, Statham AL, Kassir Z, Moreno CS, Young AN, Varma V, Speed TP, Cowley M, Lacaze P, Kaplan W, <strong>Robinson MD</strong>, Clark SJ: Consolidation of the cancer genome into domains of repressive chromatin by long-range epigenetic silencing (LRES) reduces transcriptional plasticity. Nat Cell Biol 12(3):235-46 (2010). <a href="https://doi.org/10.1038/ncb2023" target="_blank"> DOI: 10.1038/ncb2023</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/20173741" target="_blank"> PMID 20173741</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1038/ncb2023" data-hide-no-mentions="true">

</div></li>
<li><strong>Robinson MD</strong>, McCarthy DJ, Smyth GK: edgeR: a Bioconductor package for differential expression analysis of digital gene expression data. Bioinformatics 26(1):139-40 (2010). <a href="https://doi.org/10.1093/bioinformatics/btp616" target="_blank"> DOI: 10.1093/bioinformatics/btp616</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/19910308" target="_blank"> PMID 19910308</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1093/bioinformatics/btp616" data-hide-no-mentions="true">

</div></li>
</ul>
</div>
<div id="section-13" class="section level2">
<h2>2009</h2>
<ul>
<li><strong>Robinson MD</strong>, Speed TP: Differential splicing using whole-transcript microarrays. BMC Bioinformatics 10:156 (2009). <a href="https://doi.org/10.1186/1471-2105-10-156" target="_blank"> DOI: 10.1186/1471-2105-10-156</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/19463149" target="_blank"> PMID 19463149</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1186/1471-2105-10-156" data-hide-no-mentions="true">

</div></li>
</ul>
</div>
<div id="section-14" class="section level2">
<h2>2008</h2>
<ul>
<li><strong>Robinson MD</strong>, Smyth GK: Small-sample estimation of negative binomial dispersion, with applications to SAGE data. Biostatistics 9(2):321-32 (2008). <a href="https://doi.org/10.1093/biostatistics/kxm030" target="_blank"> DOI: 10.1093/biostatistics/kxm030</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/17728317" target="_blank"> PMID 17728317</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1093/biostatistics/kxm030" data-hide-no-mentions="true">

</div></li>
<li>Cancer Genome Atlas Research Network.: Comprehensive genomic characterization defines human glioblastoma genes and core pathways. Nature 455(7216):1061-8 (2008). <a href="https://doi.org/10.1038/nature07385" target="_blank"> DOI: 10.1038/nature07385</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/18772890" target="_blank"> PMID 18772890</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1038/nature07385" data-hide-no-mentions="true">

</div></li>
<li>Purdom E, Simpson KM, <strong>Robinson MD</strong>, Conboy JG, Lapuk AV, Speed TP: FIRMA: a method for detection of alternative splicing from exon array data. Bioinformatics 24(15):1707-14 (2008). <a href="https://doi.org/10.1093/bioinformatics/btn284" target="_blank"> DOI: 10.1093/bioinformatics/btn284</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/18573797" target="_blank"> PMID 18573797</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1093/bioinformatics/btn284" data-hide-no-mentions="true">

</div></li>
</ul>
</div>
<div id="section-15" class="section level2">
<h2>2007</h2>
<ul>
<li><strong>Robinson MD</strong>, Speed TP: A comparison of Affymetrix gene expression arrays. BMC Bioinformatics 8:449 (2007). <a href="https://doi.org/10.1186/1471-2105-8-449" target="_blank"> DOI: 10.1186/1471-2105-8-449</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/18005448" target="_blank"> PMID 18005448</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1186/1471-2105-8-449" data-hide-no-mentions="true">

</div></li>
<li><strong>Robinson MD</strong>, De Souza DP, Keen WW, Saunders EC, McConville MJ, Speed TP, Likić VA: A dynamic programming approach for the alignment of signal peaks in multiple gas chromatography-mass spectrometry experiments. BMC Bioinformatics 8:419 (2007). <a href="https://doi.org/10.1186/1471-2105-8-419" target="_blank"> DOI: 10.1186/1471-2105-8-419</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/17963529" target="_blank"> PMID 17963529</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1186/1471-2105-8-419" data-hide-no-mentions="true">

</div></li>
<li><strong>Robinson MD</strong>, Smyth GK: Moderated statistical tests for assessing differences in tag abundance. Bioinformatics 23(21):2881-7 (2007). <a href="https://doi.org/10.1093/bioinformatics/btm453" target="_blank"> DOI: 10.1093/bioinformatics/btm453</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/17881408" target="_blank"> PMID 17881408</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1093/bioinformatics/btm453" data-hide-no-mentions="true">

</div></li>
<li>Ewing RM, Chu P, Elisma F, Li H, Taylor P, Climie S, McBroom-Cerajewski L, <strong>Robinson MD</strong>, O’Connor L, Li M, Taylor R, Dharsee M, Ho Y, Heilbut A, Moore L, Zhang S, Ornatsky O, Bukhman YV, Ethier M, Sheng Y, Vasilescu J, Abu-Farha M, Lambert JP, Duewel HS, Stewart II, Kuehl B, Hogue K, Colwill K, Gladwish K, Muskat B, Kinach R, Adams SL, Moran MF, Morin GB, Topaloglou T, Figeys D: Large-scale mapping of human protein-protein interactions by mass spectrometry. Mol Syst Biol 3:89 (2007). <a href="https://doi.org/10.1038/msb4100134" target="_blank"> DOI: 10.1038/msb4100134</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/17353931" target="_blank"> PMID 17353931</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1038/msb4100134" data-hide-no-mentions="true">

</div></li>
</ul>
</div>
<div id="section-16" class="section level2">
<h2>2006</h2>
<ul>
<li>Chua G, Morris QD, Sopko R, <strong>Robinson MD</strong>, Ryan O, Chan ET, Frey BJ, Andrews BJ, Boone C, Hughes TR: Identifying transcription factor functions and targets by phenotypic activation. Proc Natl Acad Sci U S A 103(32):12045-50 (2006). <a href="https://doi.org/10.1073/pnas.0605140103" target="_blank"> DOI: 10.1073/pnas.0605140103</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/16880382" target="_blank"> PMID 16880382</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1073/pnas.0605140103" data-hide-no-mentions="true">

</div></li>
<li>Krogan NJ, Cagney G, Yu H, Zhong G, Guo X, Ignatchenko A, Li J, Pu S, Datta N, Tikuisis AP, Punna T, Peregrín-Alvarez JM, Shales M, Zhang X, Davey M, <strong>Robinson MD</strong>, Paccanaro A, Bray JE, Sheung A, Beattie B, Richards DP, Canadien V, Lalev A, Mena F, Wong P, Starostine A, Canete MM, Vlasblom J, Wu S, Orsi C, Collins SR, Chandran S, Haw R, Rilstone JJ, Gandi K, Thompson NJ, Musso G, St Onge P, Ghanny S, Lam MH, Butland G, Altaf-Ul AM, Kanaya S, Shilatifard A, O’Shea E, Weissman JS, Ingles CJ, Hughes TR, Parkinson J, Gerstein M, Wodak SJ, Emili A, Greenblatt JF: Global landscape of protein complexes in the yeast Saccharomyces cerevisiae. Nature 440(7084):637-43 (2006). <a href="https://doi.org/10.1038/nature04670" target="_blank"> DOI: 10.1038/nature04670</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/16554755" target="_blank"> PMID 16554755</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1038/nature04670" data-hide-no-mentions="true">

</div></li>
<li>Belov K, Deakin JE, Papenfuss AT, Baker ML, Melman SD, Siddle HV, Gouin N, Goode DL, Sargeant TJ, <strong>Robinson MD</strong>, Wakefield MJ, Mahony S, Cross JG, Benos PV, Samollow PB, Speed TP, Graves JA, Miller RD: Reconstructing an ancestral mammalian immune supercomplex from a marsupial major histocompatibility complex. PLoS Biol 4(3):e46 (2006). <a href="https://doi.org/10.1371/journal.pbio.0040046" target="_blank"> DOI: 10.1371/journal.pbio.0040046</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/16435885" target="_blank"> PMID 16435885</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1371/journal.pbio.0040046" data-hide-no-mentions="true">

</div></li>
</ul>
</div>
<div id="section-17" class="section level2">
<h2>2005</h2>
<ul>
<li>Davierwala AP, Haynes J, Li Z, Brost RL, <strong>Robinson MD</strong>, Yu L, Mnaimneh S, Ding H, Zhu H, Chen Y, Cheng X, Brown GW, Boone C, Andrews BJ, Hughes TR: The synthetic genetic interaction spectrum of essential genes. Nat Genet 37(10):1147-52 (2005). <a href="https://doi.org/10.1038/ng1640" target="_blank"> DOI: 10.1038/ng1640</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/16155567" target="_blank"> PMID 16155567</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1038/ng1640" data-hide-no-mentions="true">

</div></li>
<li>Frey BJ, Mohammad N, Morris QD, Zhang W, <strong>Robinson MD</strong>, Mnaimneh S, Chang R, Pan Q, Sat E, Rossant J, Bruneau BG, Aubin JE, Blencowe BJ, Hughes TR: Genome-wide analysis of mouse transcripts using exon microarrays and factor graphs. Nat Genet 37(9):991-6 (2005). <a href="https://doi.org/10.1038/ng1630" target="_blank"> DOI: 10.1038/ng1630</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/16127451" target="_blank"> PMID 16127451</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1038/ng1630" data-hide-no-mentions="true">

</div></li>
<li>Barrios-Rodiles M, Brown KR, Ozdamar B, Bose R, Liu Z, Donovan RS, Shinjo F, Liu Y, Dembowy J, Taylor IW, Luga V, Przulj N, <strong>Robinson M</strong>, Suzuki H, Hayashizaki Y, Jurisica I, Wrana JL: High-throughput mapping of a dynamic signaling network in mammalian cells. Science 307(5715):1621-5 (2005). <a href="https://doi.org/10.1126/science.1105776" target="_blank"> DOI: 10.1126/science.1105776</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/15761153" target="_blank"> PMID 15761153</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1126/science.1105776" data-hide-no-mentions="true">

</div></li>
</ul>
</div>
<div id="section-18" class="section level2">
<h2>2004</h2>
<ul>
<li>Le Bihan T, <strong>Robinson MD</strong>, Stewart II, Figeys D: Definition and characterization of a “trypsinosome” from specific peptide characteristics by nano-HPLC-MS/MS and in silico analysis of complex protein mixtures. J Proteome Res 3(6):1138-48 (2004). <a href="https://doi.org/10.1021/pr049909x" target="_blank"> DOI: 10.1021/pr049909x</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/15595722" target="_blank"> PMID 15595722</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1021/pr049909x" data-hide-no-mentions="true">

</div></li>
<li>Zhang W, Morris QD, Chang R, Shai O, Bakowski MA, Mitsakakis N, Mohammad N, <strong>Robinson MD</strong>, Zirngibl R, Somogyi E, Laurin N, Eftekharpour E, Sat E, Grigull J, Pan Q, Peng WT, Krogan N, Greenblatt J, Fehlings M, van der Kooy D, Aubin J, Bruneau BG, Rossant J, Blencowe BJ, Frey BJ, Hughes TR: The functional landscape of mouse gene expression. J Biol 3(5):21 (2004). <a href="https://doi.org/10.1186/jbiol16" target="_blank"> DOI: 10.1186/jbiol16</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/15588312" target="_blank"> PMID 15588312</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1186/jbiol16" data-hide-no-mentions="true">

</div></li>
<li>Chua G, <strong>Robinson MD</strong>, Morris Q, Hughes TR: Transcriptional networks: reverse-engineering gene regulation on a global scale. Curr Opin Microbiol 7(6):638-46 (2004). <a href="https://doi.org/10.1016/j.mib.2004.10.009" target="_blank"> DOI: 10.1016/j.mib.2004.10.009</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/15556037" target="_blank"> PMID 15556037</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1016/j.mib.2004.10.009" data-hide-no-mentions="true">

</div></li>
<li>Hughes TR, <strong>Robinson MD</strong>, Mitsakakis N, Johnston M: The promise of functional genomics: completing the encyclopedia of a cell. Curr Opin Microbiol 7(5):546-54 (2004). <a href="https://doi.org/10.1016/j.mib.2004.08.015" target="_blank"> DOI: 10.1016/j.mib.2004.08.015</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/15451511" target="_blank"> PMID 15451511</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1016/j.mib.2004.08.015" data-hide-no-mentions="true">

</div></li>
<li>Grigull J, Mnaimneh S, Pootoolal J, <strong>Robinson MD</strong>, Hughes TR: Genome-wide analysis of mRNA stability using transcription inhibitors and microarrays reveals posttranscriptional control of ribosome biogenesis factors. Mol Cell Biol 24(12):5534-47 (2004). <a href="https://doi.org/10.1128/MCB.24.12.5534-5547.2004" target="_blank"> DOI: 10.1128/MCB.24.12.5534-5547.2004</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/15169913" target="_blank"> PMID 15169913</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1128/MCB.24.12.5534-5547.2004" data-hide-no-mentions="true">

</div></li>
<li>Krogan NJ, Peng WT, Cagney G, <strong>Robinson MD</strong>, Haw R, Zhong G, Guo X, Zhang X, Canadien V, Richards DP, Beattie BK, Lalev A, Zhang W, Davierwala AP, Mnaimneh S, Starostine A, Tikuisis AP, Grigull J, Datta N, Bray JE, Hughes TR, Emili A, Greenblatt JF: High-definition macromolecular composition of yeast RNA-processing complexes. Mol Cell 13(2):225-39 (2004). <a href="https://doi.org/10.1016/s1097-2765(04)00003-6" target="_blank"> DOI: 10.1016/s1097-2765(04)00003-6</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/14759368" target="_blank"> PMID 14759368</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1016/s1097-2765(04)00003-6" data-hide-no-mentions="true">

</div></li>
</ul>
</div>
<div id="section-19" class="section level2">
<h2>2003</h2>
<ul>
<li>Peng WT, <strong>Robinson MD</strong>, Mnaimneh S, Krogan NJ, Cagney G, Morris Q, Davierwala AP, Grigull J, Yang X, Zhang W, Mitsakakis N, Ryan OW, Datta N, Jojic V, Pal C, Canadien V, Richards D, Beattie B, Wu LF, Altschuler SJ, Roweis S, Frey BJ, Emili A, Greenblatt JF, Hughes TR: A panoramic view of yeast noncoding RNA processing. Cell 113(7):919-33 (2003). <a href="https://doi.org/10.1016/s0092-8674(03)00466-5" target="_blank"> DOI: 10.1016/s0092-8674(03)00466-5</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/12837249" target="_blank"> PMID 12837249</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1016/s0092-8674(03)00466-5" data-hide-no-mentions="true">

</div></li>
</ul>
</div>
<div id="section-20" class="section level2">
<h2>2002</h2>
<ul>
<li>Jorgensen P, Nelson B, <strong>Robinson MD</strong>, Chen Y, Andrews B, Tyers M, Boone C: High-resolution genetic mapping with ordered arrays of Saccharomyces cerevisiae deletion mutants. Genetics 162(3):1091-9 (2002). <a href="https://doi.org/10.1093/genetics/162.3.1091" target="_blank"> DOI: 10.1093/genetics/162.3.1091</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/12454058" target="_blank"> PMID 12454058</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1093/genetics/162.3.1091" data-hide-no-mentions="true">

</div></li>
<li><strong>Robinson MD</strong>, Grigull J, Mohammad N, Hughes TR: FunSpec: a web-based cluster interpreter for yeast. BMC Bioinformatics 3:35 (2002). <a href="https://doi.org/10.1186/1471-2105-3-35" target="_blank"> DOI: 10.1186/1471-2105-3-35</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/12431279" target="_blank"> PMID 12431279</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1186/1471-2105-3-35" data-hide-no-mentions="true">

</div></li>
<li>Wu LF, Hughes TR, Davierwala AP, <strong>Robinson MD</strong>, Stoughton R, Altschuler SJ: Large-scale prediction of Saccharomyces cerevisiae gene function using overlapping transcriptional clusters. Nat Genet 31(3):255-65 (2002). <a href="https://doi.org/10.1038/ng906" target="_blank"> DOI: 10.1038/ng906</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/12089522" target="_blank"> PMID 12089522</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1038/ng906" data-hide-no-mentions="true">

</div></li>
</ul>
</div>
<div id="section-21" class="section level2">
<h2>2001</h2>
<ul>
<li>Tong AH, Evangelista M, Parsons AB, Xu H, Bader GD, Pagé N, <strong>Robinson M</strong>, Raghibizadeh S, Hogue CW, Bussey H, Andrews B, Tyers M, Boone C: Systematic genetic analysis with ordered arrays of yeast deletion mutants. Science 294(5550):2364-8 (2001). <a href="https://doi.org/10.1126/science.1065810" target="_blank"> DOI: 10.1126/science.1065810</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/11743205" target="_blank"> PMID 11743205</a>.
<div class="altmetric-embed" data-badge-popover="right" data-no-score="true" data-badge-type="bar" data-doi="https://doi.org/10.1126/science.1065810" data-hide-no-mentions="true">

</div></li>
</ul>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
