<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />




<title>Publications</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/flatly.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.0.13/css/fa-svg-with-js.css" rel="stylesheet" />
<script src="site_libs/font-awesome-5.0.13/js/fontawesome-all.min.js"></script>
<script src="site_libs/font-awesome-5.0.13/js/fa-v4-shims.min.js"></script>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 60px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 65px;
  margin-top: -65px;
}

.section h2 {
  padding-top: 65px;
  margin-top: -65px;
}
.section h3 {
  padding-top: 65px;
  margin-top: -65px;
}
.section h4 {
  padding-top: 65px;
  margin-top: -65px;
}
.section h5 {
  padding-top: 65px;
  margin-top: -65px;
}
.section h6 {
  padding-top: 65px;
  margin-top: -65px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->






<div class="navbar navbar-inverse  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Statistical Bioinformatics @SIB @UZH</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="research.html">
    <span class="fa fa-area-chart"></span>
     
    research
  </a>
</li>
<li>
  <a href="publications.html">
    <span class="fa fa-file-text-o"></span>
     
    publications
  </a>
</li>
<li>
  <a href="tools.html">
    <span class="fa fa-code"></span>
     
    resources
  </a>
</li>
<li>
  <a href="people.html">
    <span class="fa fa-user"></span>
     
    people
  </a>
</li>
<li>
  <a href="teaching.html">
    <span class="fa fa-graduation-cap"></span>
     
    teaching
  </a>
</li>
<li>
  <a href="news.html">
    <span class="fa fa-newspaper-o"></span>
     
    news
  </a>
</li>
<li>
  <a href="more_info.html">
    <span class="fa fa-info-circle"></span>
     
    more info
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Publications</h1>

</div>


<script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>
<div id="preprints" class="section level2">
<h2>Preprints</h2>
<ul>
<li><strong>Lukas M Weber</strong>, <strong>Malgorzata Nowicka</strong>, <strong>Charlotte Soneson</strong>, <strong>Mark D Robinson</strong>: diffcyt: Differential discovery in high-dimensional cytometry via high-resolution clustering. bioRxiv doi: <a href="https://doi.org/10.1101/349738" target="_blank"> https://doi.org/10.1101/349738</a>. Posted June 18, 2018. <a href="https://github.com/lmweber/diffcyt-evaluations" target="_blank"> GitHub repo</a>. <a href="http://flowrepository.org/id/FR-FCM-ZYL8" target="_blank"> Data</a>. <a href="https://bioconductor.org/packages/release/bioc/html/diffcyt.html" target="_blank"> Software</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1101/349738" data-hide-no-mentions="true">

</div></li>
</ul>
</div>
<div id="section" class="section level2">
<h2>2018</h2>
<ul>
<li>Savic N, Ringnalda FC, <strong>Lindsay H</strong>, Berk C, Bargsten K, Li Y, Neri D, <strong>Robinson MD</strong>, Ciaudo C, Hall J, Jinek M, Schwank G: Covalent linkage of the DNA repair template to the CRISPR-Cas9 nuclease enhances homology-directed repair. Elife 7: (2018). DOI: 10.7554/eLife.33761. <a href="http://www.ncbi.nlm.nih.gov/pubmed/29809142" target="_blank"> PMID 29809142</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.7554/eLife.33761" data-hide-no-mentions="true">

</div></li>
<li>Wrangle JM, Velcheti V, Patel MR, Garrett-Mayer E, Hill EG, Ravenel JG, Miller JS, Farhad M, Anderton K, Lindsey K, Taffaro-Neskey M, Sherman C, Suriano S, Swiderska-Syn M, Sion A, Harris J, Edwards AR, Rytlewski JA, Sanders CM, Yusko EC, <strong>Robinson MD</strong>, Krieg C, Redmond WL, Egan JO, Rhode PR, Jeng EK, Rock AD, Wong HC, Rubinstein MP: ALT-803, an IL-15 superagonist, in combination with nivolumab in patients with metastatic non-small cell lung cancer: a non-randomised, open-label, phase 1b trial. Lancet Oncol 19(5):694-704 (2018). DOI: 10.1016/S1470-2045(18)30148-7. <a href="http://www.ncbi.nlm.nih.gov/pubmed/29628312" target="_blank"> PMID 29628312</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1016/S1470-2045(18)30148-7" data-hide-no-mentions="true">

</div></li>
<li>Chevrier S, <strong>Crowell HL</strong>, Zanotelli VRT, Engler S, <strong>Robinson MD</strong>, Bodenmiller B: Compensation of Signal Spillover in Suspension and Imaging Mass Cytometry. Cell Syst 6(5):612-620.e5 (2018). DOI: 10.1016/j.cels.2018.02.010. <a href="http://www.ncbi.nlm.nih.gov/pubmed/29605184" target="_blank"> PMID 29605184</a>. <a href="http://bioconductor.org/packages/release/bioc/html/CATALYST.html" target="_blank"> Software</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1016/j.cels.2018.02.010" data-hide-no-mentions="true">

</div></li>
<li><strong>Soneson C</strong>, <strong>Robinson MD</strong>: Bias, robustness and scalability in single-cell differential expression analysis. Nat Methods 15(4):255-261 (2018). DOI: 10.1038/nmeth.4612. <a href="http://www.ncbi.nlm.nih.gov/pubmed/29481549" target="_blank"> PMID 29481549</a>. <a href="https://github.com/csoneson/conquer_comparison" target="_blank"> GitHub repo</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1038/nmeth.4612" data-hide-no-mentions="true">

</div></li>
<li>Van den Berge K, Perraudeau F, <strong>Soneson C</strong>, Love MI, Risso D, Vert JP, <strong>Robinson MD</strong>, Dudoit S, Clement L: Observation weights unlock bulk RNA-seq tools for zero inflation and single-cell applications. Genome Biol 19(1):24 (2018). DOI: 10.1186/s13059-018-1406-4. <a href="http://www.ncbi.nlm.nih.gov/pubmed/29478411" target="_blank"> PMID 29478411</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1186/s13059-018-1406-4" data-hide-no-mentions="true">

</div></li>
<li>Praz CR, Menardo F, <strong>Robinson MD</strong>, Müller MC, Wicker T, Bourras S, Keller B: Non-parent of Origin Expression of Numerous Effector Genes Indicates a Role of Gene Regulation in Host Adaption of the Hybrid Triticale Powdery Mildew Pathogen. Front Plant Sci 9:49 (2018). DOI: 10.3389/fpls.2018.00049. <a href="http://www.ncbi.nlm.nih.gov/pubmed/29441081" target="_blank"> PMID 29441081</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.3389/fpls.2018.00049" data-hide-no-mentions="true">

</div></li>
<li>Krieg C, <strong>Nowicka M</strong>, Guglietta S, Schindler S, Hartmann FJ, <strong>Weber LM</strong>, Dummer R, <strong>Robinson MD</strong>, Levesque MP, Becher B: High-dimensional single-cell analysis predicts response to anti-PD-1 immunotherapy. Nat Med 24(2):144-153 (2018). DOI: 10.1038/nm.4466. <a href="http://www.ncbi.nlm.nih.gov/pubmed/29309059" target="_blank"> PMID 29309059</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1038/nm.4466" data-hide-no-mentions="true">

</div></li>
<li><strong>Soneson C</strong>, <strong>Robinson MD</strong>: Towards unified quality verification of synthetic count data with countsimQC. Bioinformatics 34(4):691-692 (2018). DOI: 10.1093/bioinformatics/btx631. <a href="http://www.ncbi.nlm.nih.gov/pubmed/29028961" target="_blank"> PMID 29028961</a>. <a href="https://github.com/csoneson/countsimQC" target="_blank"> Software</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1093/bioinformatics/btx631" data-hide-no-mentions="true">

</div></li>
</ul>
</div>
<div id="section-1" class="section level2">
<h2>2017</h2>
<ul>
<li><strong>Nowicka M</strong>, Krieg C, <strong>Weber LM</strong>, Hartmann FJ, Guglietta S, Becher B, Levesque MP, <strong>Robinson MD</strong>: CyTOF workflow: differential discovery in high-throughput high-dimensional cytometry datasets. F1000Res 6:748 (2017). DOI: 10.12688/f1000research.11622.2. <a href="http://www.ncbi.nlm.nih.gov/pubmed/28663787" target="_blank"> PMID 28663787</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.12688/f1000research.11622.2" data-hide-no-mentions="true">

</div></li>
<li>Omasits U, Varadarajan AR, Schmid M, Goetze S, Melidis D, Bourqui M, <strong>Nikolayeva O</strong>, Québatte M, Patrignani A, Dehio C, Frey JE, <strong>Robinson MD</strong>, Wollscheid B, Ahrens CH: An integrative strategy to identify the entire protein coding potential of prokaryotic genomes by proteogenomics. Genome Res 27(12):2083-2095 (2017). DOI: 10.1101/gr.218255.116. <a href="http://www.ncbi.nlm.nih.gov/pubmed/29141959" target="_blank"> PMID 29141959</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1101/gr.218255.116" data-hide-no-mentions="true">

</div></li>
<li>Meccariello A, Monti SM, Romanelli A, Colonna R, Primo P, Inghilterra MG, Del Corsano G, Ramaglia A, Iazzetti G, Chiarore A, Patti F, Heinze SD, Salvemini M, <strong>Lindsay H</strong>, Chiavacci E, Burger A, <strong>Robinson MD</strong>, Mosimann C, Bopp D, Saccone G: Highly efficient DNA-free gene disruption in the agricultural pest Ceratitis capitata by CRISPR-Cas9 ribonucleoprotein complexes. Sci Rep 7(1):10061 (2017). DOI: 10.1038/s41598-017-10347-5. <a href="http://www.ncbi.nlm.nih.gov/pubmed/28855635" target="_blank"> PMID 28855635</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1038/s41598-017-10347-5" data-hide-no-mentions="true">

</div></li>
<li>Van den Berge K, <strong>Soneson C</strong>, <strong>Robinson MD</strong>, Clement L: stageR: a general stage-wise method for controlling the gene-level false discovery rate in differential expression and differential transcript usage. Genome Biol 18(1):151 (2017). DOI: 10.1186/s13059-017-1277-0. <a href="http://www.ncbi.nlm.nih.gov/pubmed/28784146" target="_blank"> PMID 28784146</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1186/s13059-017-1277-0" data-hide-no-mentions="true">

</div></li>
<li>Herdman C, Mars JC, Stefanovsky VY, Tremblay MG, Sabourin-Felix M, <strong>Lindsay H</strong>, <strong>Robinson MD</strong>, Moss T: A unique enhancer boundary complex on the mouse ribosomal RNA genes persists after loss of Rrn3 or UBF and the inactivation of RNA polymerase I transcription. PLoS Genet 13(7):e1006899 (2017). DOI: 10.1371/journal.pgen.1006899. <a href="http://www.ncbi.nlm.nih.gov/pubmed/28715449" target="_blank"> PMID 28715449</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1371/journal.pgen.1006899" data-hide-no-mentions="true">

</div></li>
<li>Sharma A, Heinze SD, Wu Y, Kohlbrenner T, <strong>Morilla I</strong>, Brunner C, Wimmer EA, van de Zande L, <strong>Robinson MD</strong>, Beukeboom LW, Bopp D: Male sex in houseflies is determined by <em>Mdmd</em>, a paralog of the generic splice factor gene <em>CWC22</em>. Science 356(6338):642-645 (2017). DOI: 10.1126/science.aam5498. <a href="http://www.ncbi.nlm.nih.gov/pubmed/28495751" target="_blank"> PMID 28495751</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1126/science.aam5498" data-hide-no-mentions="true">

</div></li>
<li>Marinova Z, Maercker A, Küffer A, <strong>Robinson MD</strong>, Wojdacz TK, Walitza S, Grünblatt E, Burri A: DNA methylation profiles of elderly individuals subjected to indentured childhood labor and trauma. BMC Med Genet 18(1):21 (2017). DOI: 10.1186/s12881-017-0370-2. <a href="http://www.ncbi.nlm.nih.gov/pubmed/28241754" target="_blank"> PMID 28241754</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1186/s12881-017-0370-2" data-hide-no-mentions="true">

</div></li>
<li>Sauerwald J, <strong>Soneson C</strong>, <strong>Robinson MD</strong>, Luschnig S: Faithful mRNA splicing depends on the Prp19 complex subunit <em>faint sausage</em> and is required for tracheal branching morphogenesis in <em>Drosophila</em>. Development 144(4):657-663 (2017). DOI: 10.1242/dev.144535. <a href="http://www.ncbi.nlm.nih.gov/pubmed/28087625" target="_blank"> PMID 28087625</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1242/dev.144535" data-hide-no-mentions="true">

</div></li>
</ul>
</div>
<div id="section-2" class="section level2">
<h2>2016</h2>
<ul>
<li><strong>Nowicka M</strong>, <strong>Robinson MD</strong>: DRIMSeq: a Dirichlet-multinomial framework for multivariate count outcomes in genomics. F1000Res 5:1356 (2016). DOI: 10.12688/f1000research.8900.2. <a href="http://www.ncbi.nlm.nih.gov/pubmed/28105305" target="_blank"> PMID 28105305</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.12688/f1000research.8900.2" data-hide-no-mentions="true">

</div></li>
<li><strong>Weber LM</strong>, <strong>Robinson MD</strong>: Comparison of clustering methods for high-dimensional single-cell flow and mass cytometry data. Cytometry A 89(12):1084-1096 (2016). DOI: 10.1002/cyto.a.23030. <a href="http://www.ncbi.nlm.nih.gov/pubmed/27992111" target="_blank"> PMID 27992111</a>. <a href="https://github.com/lmweber/cytometry-clustering-comparison" target="_blank"> GitHub repo</a>. <a href="http://flowrepository.org/id/FR-FCM-ZZPH" target="_blank"> Data</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1002/cyto.a.23030" data-hide-no-mentions="true">

</div></li>
<li>Burri A, Marinova Z, <strong>Robinson MD</strong>, Kühnel B, Waldenberger M, Wahl S, Kunze S, Gieger C, Livshits G, Williams F: Are Epigenetic Factors Implicated in Chronic Widespread Pain? PLoS One 11(11):e0165548 (2016). DOI: 10.1371/journal.pone.0165548. <a href="http://www.ncbi.nlm.nih.gov/pubmed/27832094" target="_blank"> PMID 27832094</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1371/journal.pone.0165548" data-hide-no-mentions="true">

</div></li>
<li>Hartmann FJ, Bernard-Valnet R, Quériault C, Mrdjen D, <strong>Weber LM</strong>, Galli E, Krieg C, <strong>Robinson MD</strong>, Nguyen XH, Dauvilliers Y, Liblau RS, Becher B: High-dimensional single-cell analysis reveals the immune signature of narcolepsy. J Exp Med 213(12):2621-2633 (2016). DOI: 10.1084/jem.20160897. <a href="http://www.ncbi.nlm.nih.gov/pubmed/27821550" target="_blank"> PMID 27821550</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1084/jem.20160897" data-hide-no-mentions="true">

</div></li>
<li>Hawkins ED, Duarte D, Akinduro O, Khorshed RA, Passaro D, <strong>Nowicka M</strong>, Straszkowski L, Scott MK, Rothery S, Ruivo N, Foster K, Waibel M, Johnstone RW, Harrison SJ, Westerman DA, Quach H, Gribben J, <strong>Robinson MD</strong>, Purton LE, Bonnet D, Lo Celso C: T-cell acute leukaemia exhibits dynamic interactions with bone marrow microenvironments. Nature 538(7626):518-522 (2016). DOI: 10.1038/nature19801. <a href="http://www.ncbi.nlm.nih.gov/pubmed/27750279" target="_blank"> PMID 27750279</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1038/nature19801" data-hide-no-mentions="true">

</div></li>
<li><strong>Lindsay H</strong>, Burger A, <strong>Biyong B</strong>, Felker A, Hess C, Zaugg J, Chiavacci E, Anders C, Jinek M, Mosimann C, <strong>Robinson MD</strong>: CrispRVariants charts the mutation spectrum of genome engineering experiments. Nat Biotechnol 34(7):701-2 (2016). DOI: 10.1038/nbt.3628. <a href="http://www.ncbi.nlm.nih.gov/pubmed/27404876" target="_blank"> PMID 27404876</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1038/nbt.3628" data-hide-no-mentions="true">

</div></li>
<li>Burger A, <strong>Lindsay H</strong>, Felker A, Hess C, Anders C, Chiavacci E, Zaugg J, <strong>Weber LM</strong>, Catena R, Jinek M, <strong>Robinson MD</strong>, Mosimann C: Maximizing mutagenesis with solubilized CRISPR-Cas9 ribonucleoprotein complexes. Development 143(11):2025-37 (2016). DOI: 10.1242/dev.134809. <a href="http://www.ncbi.nlm.nih.gov/pubmed/27130213" target="_blank"> PMID 27130213</a>. <a href="http://lmweber.github.io/CrispantCal/" target="_blank"> Software</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1242/dev.134809" data-hide-no-mentions="true">

</div></li>
<li><strong>Soneson C</strong>, <strong>Robinson MD</strong>: iCOBRA: open, reproducible, standardized and live method benchmarking. Nat Methods 13(4):283 (2016). DOI: 10.1038/nmeth.3805. <a href="http://www.ncbi.nlm.nih.gov/pubmed/27027585" target="_blank"> PMID 27027585</a>. <a href="http://bioconductor.org/packages/release/bioc/html/iCOBRA.html" target="_blank"> Software</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1038/nmeth.3805" data-hide-no-mentions="true">

</div></li>
<li><strong>Soneson C</strong>, <strong>Matthes KL</strong>, <strong>Nowicka M</strong>, <strong>Law CW</strong>, <strong>Robinson MD</strong>: Isoform prefiltering improves performance of count-based methods for analysis of differential transcript usage. Genome Biol 17:12 (2016). DOI: 10.1186/s13059-015-0862-3. <a href="http://www.ncbi.nlm.nih.gov/pubmed/26813113" target="_blank"> PMID 26813113</a>. <a href="https://github.com/markrobinsonuzh/diff_splice_paper" target="_blank"> GitHub repo</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1186/s13059-015-0862-3" data-hide-no-mentions="true">

</div></li>
<li>Flori M, Schmid CA, Sumrall ET, Tzankov A, <strong>Law CW</strong>, <strong>Robinson MD</strong>, Müller A: The hematopoietic oncoprotein FOXP1 promotes tumor cell survival in diffuse large B-cell lymphoma by repressing S1PR2 signaling. Blood 127(11):1438-48 (2016). DOI: 10.1182/blood-2015-08-662635. <a href="http://www.ncbi.nlm.nih.gov/pubmed/26729899" target="_blank"> PMID 26729899</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1182/blood-2015-08-662635" data-hide-no-mentions="true">

</div></li>
</ul>
</div>
<div id="section-3" class="section level2">
<h2>2015</h2>
<ul>
<li><strong>Soneson C</strong>, Love MI, <strong>Robinson MD</strong>: Differential analyses for RNA-seq: transcript-level estimates improve gene-level inferences. F1000Res 4:1521 (2015). DOI: 10.12688/f1000research.7563.2. <a href="http://www.ncbi.nlm.nih.gov/pubmed/26925227" target="_blank"> PMID 26925227</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.12688/f1000research.7563.2" data-hide-no-mentions="true">

</div></li>
<li><strong>Zhou X</strong>, <strong>Robinson MD</strong>: Do count-based differential expression methods perform poorly when genes are expressed in only one condition? Genome Biol 16:222 (2015). DOI: 10.1186/s13059-015-0781-3. <a href="http://www.ncbi.nlm.nih.gov/pubmed/26450178" target="_blank"> PMID 26450178</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1186/s13059-015-0781-3" data-hide-no-mentions="true">

</div></li>
<li>Bilgin Sonay T, Carvalho T, <strong>Robinson MD</strong>, Greminger MP, Krützen M, Comas D, Highnam G, Mittelman D, Sharp A, Marques-Bonet T, Wagner A: Tandem repeat variation in human and great ape populations and its impact on gene expression divergence. Genome Res 25(11):1591-9 (2015). DOI: 10.1101/gr.190868.115. <a href="http://www.ncbi.nlm.nih.gov/pubmed/26290536" target="_blank"> PMID 26290536</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1101/gr.190868.115" data-hide-no-mentions="true">

</div></li>
<li>Locke WJ, Zotenko E, Stirzaker C, <strong>Robinson MD</strong>, Hinshelwood RA, Stone A, Reddel RR, Huschtscha LI, Clark SJ: Coordinated epigenetic remodelling of transcriptional networks occurs during early breast carcinogenesis. Clin Epigenetics 7:52 (2015). DOI: 10.1186/s13148-015-0086-0. <a href="http://www.ncbi.nlm.nih.gov/pubmed/25960784" target="_blank"> PMID 25960784</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1186/s13148-015-0086-0" data-hide-no-mentions="true">

</div></li>
<li>Schmid CA, <strong>Robinson MD</strong>, Scheifinger NA, Müller S, Cogliatti S, Tzankov A, Müller A: DUSP4 deficiency caused by promoter hypermethylation drives JNK signaling and tumor cell survival in diffuse large B cell lymphoma. J Exp Med 212(5):775-92 (2015). DOI: 10.1084/jem.20141957. <a href="http://www.ncbi.nlm.nih.gov/pubmed/25847947" target="_blank"> PMID 25847947</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1084/jem.20141957" data-hide-no-mentions="true">

</div></li>
<li><strong>Robinson MD</strong>, Pelizzola M: Computational epigenomics: challenges and opportunities. Front Genet 6:88 (2015). DOI: 10.3389/fgene.2015.00088. <a href="http://www.ncbi.nlm.nih.gov/pubmed/25798147" target="_blank"> PMID 25798147</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.3389/fgene.2015.00088" data-hide-no-mentions="true">

</div></li>
<li>Stirzaker C, Zotenko E, Song JZ, Qu W, Nair SS, Locke WJ, Stone A, Armstong NJ, <strong>Robinson MD</strong>, Dobrovic A, Avery-Kiejda KA, Peters KM, French JD, Stein S, Korbie DJ, Trau M, Forbes JF, Scott RJ, Brown MA, Francis GD, Clark SJ: Methylome sequencing in triple-negative breast cancer reveals distinct methylation clusters with prognostic value. Nat Commun 6:5899 (2015). DOI: 10.1038/ncomms6899. <a href="http://www.ncbi.nlm.nih.gov/pubmed/25641231" target="_blank"> PMID 25641231</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1038/ncomms6899" data-hide-no-mentions="true">

</div></li>
<li>Baker EK, Taylor S, Gupte A, Chalk AM, Bhattacharya S, Green AC, Martin TJ, Strbenac D, <strong>Robinson MD</strong>, Purton LE, Walkley CR: Wnt inhibitory factor 1 (WIF1) is a marker of osteoblastic differentiation stage and is not silenced by DNA methylation in osteosarcoma. Bone 73:223-32 (2015). DOI: 10.1016/j.bone.2014.12.063. <a href="http://www.ncbi.nlm.nih.gov/pubmed/25571841" target="_blank"> PMID 25571841</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1016/j.bone.2014.12.063" data-hide-no-mentions="true">

</div></li>
<li>Kulic I, Robertson G, Chang L, Baker JH, Lockwood WW, Mok W, Fuller M, Fournier M, Wong N, Chou V, <strong>Robinson MD</strong>, Chun HJ, Gilks B, Kempkes B, Thomson TA, Hirst M, Minchinton AI, Lam WL, Jones S, Marra M, Karsan A: Loss of the Notch effector RBPJ promotes tumorigenesis. J Exp Med 212(1):37-52 (2015). DOI: 10.1084/jem.20121192. <a href="http://www.ncbi.nlm.nih.gov/pubmed/25512468" target="_blank"> PMID 25512468</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1084/jem.20121192" data-hide-no-mentions="true">

</div></li>
<li>Gu L, Frommel SC, Oakes CC, Simon R, Grupp K, Gerig CY, Bär D, <strong>Robinson MD</strong>, Baer C, Weiss M, Gu Z, Schapira M, Kuner R, Sültmann H, Provenzano M, ICGC Project on Early Onset Prostate Cancer., Yaspo ML, Brors B, Korbel J, Schlomm T, Sauter G, Eils R, Plass C, Santoro R: BAZ2A (TIP5) is involved in epigenetic alterations in prostate cancer and its overexpression predicts disease recurrence. Nat Genet 47(1):22-30 (2015). DOI: 10.1038/ng.3165. <a href="http://www.ncbi.nlm.nih.gov/pubmed/25485837" target="_blank"> PMID 25485837</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1038/ng.3165" data-hide-no-mentions="true">

</div></li>
<li>Petrella V, Aceto S, Musacchia F, Colonna V, <strong>Robinson M</strong>, Benes V, Cicotti G, Bongiorno G, Gradoni L, Volf P, Salvemini M: De novo assembly and sex-specific transcriptome profiling in the sand fly Phlebotomus perniciosus (Diptera, Phlebotominae), a major Old World vector of Leishmania infantum. BMC Genomics 16:847 (2015). DOI: 10.1186/s12864-015-2088-x. <a href="http://www.ncbi.nlm.nih.gov/pubmed/26493315" target="_blank"> PMID 26493315</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1186/s12864-015-2088-x" data-hide-no-mentions="true">

</div></li>
</ul>
</div>
<div id="section-4" class="section level2">
<h2>2014</h2>
<ul>
<li><strong>Robinson MD</strong>, Kahraman A, <strong>Law CW</strong>, <strong>Lindsay H</strong>, <strong>Nowicka M</strong>, <strong>Weber LM</strong>, <strong>Zhou X</strong>: Statistical methods for detecting differentially methylated loci and regions. Front Genet 5:324 (2014). DOI: 10.3389/fgene.2014.00324. <a href="http://www.ncbi.nlm.nih.gov/pubmed/25278959" target="_blank"> PMID 25278959</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.3389/fgene.2014.00324" data-hide-no-mentions="true">

</div></li>
<li>Hickey PF, <strong>Robinson MD</strong>: Genomics by the beach. Genome Biol 15(4):304 (2014). DOI: 10.1186/gb4171. <a href="http://www.ncbi.nlm.nih.gov/pubmed/25001045" target="_blank"> PMID 25001045</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1186/gb4171" data-hide-no-mentions="true">

</div></li>
<li>Almouzni G, Altucci L, Amati B, Ashley N, Baulcombe D, Beaujean N, Bock C, Bongcam-Rudloff E, Bousquet J, Braun S, Bressac-de Paillerets B, Bussemakers M, Clarke L, Conesa A, Estivill X, Fazeli A, Grgurević N, Gut I, Heijmans BT, Hermouet S, Houwing-Duistermaat J, Iacobucci I, Ilaš J, Kandimalla R, Krauss-Etschmann S, Lasko P, Lehmann S, Lindroth A, Majdič G, Marcotte E, Martinelli G, Martinet N, Meyer E, Miceli C, Mills K, Moreno-Villanueva M, Morvan G, Nickel D, Niesler B, Nowacki M, Nowak J, Ossowski S, Pelizzola M, Pochet R, Potočnik U, Radwanska M, Raes J, Rattray M, <strong>Robinson MD</strong>, Roelen B, Sauer S, Schinzer D, Slagboom E, Spector T, Stunnenberg HG, Tiligada E, Torres-Padilla ME, Tsonaka R, Van Soom A, Vidaković M, Widschwendter M: Relationship between genome and epigenome–challenges and requirements for future research. BMC Genomics 15:487 (2014). DOI: 10.1186/1471-2164-15-487. <a href="http://www.ncbi.nlm.nih.gov/pubmed/24942464" target="_blank"> PMID 24942464</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1186/1471-2164-15-487" data-hide-no-mentions="true">

</div></li>
<li><strong>Zhou X</strong>, <strong>Lindsay H</strong>, <strong>Robinson MD</strong>: Robustly detecting differential expression in RNA sequencing data using observation weights. Nucleic Acids Res 42(11):e91 (2014). DOI: 10.1093/nar/gku310. <a href="http://www.ncbi.nlm.nih.gov/pubmed/24753412" target="_blank"> PMID 24753412</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1093/nar/gku310" data-hide-no-mentions="true">

</div></li>
<li><strong>Nikolayeva O</strong>, <strong>Robinson MD</strong>: edgeR for differential RNA-seq and ChIP-seq analysis: an application to stem cell biology. Methods Mol Biol 1150:45-79 (2014). DOI: 10.1007/978-1-4939-0512-6_3. <a href="http://www.ncbi.nlm.nih.gov/pubmed/24743990" target="_blank"> PMID 24743990</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1007/978-1-4939-0512-6_3" data-hide-no-mentions="true">

</div></li>
<li><strong>Riebler A</strong>, Menigatti M, Song JZ, Statham AL, Stirzaker C, Mahmud N, Mein CA, Clark SJ, <strong>Robinson MD</strong>: BayMeth: improved DNA methylation quantification for affinity capture sequencing data using a flexible Bayesian approach. Genome Biol 15(2):R35 (2014). DOI: 10.1186/gb-2014-15-2-r35. <a href="http://www.ncbi.nlm.nih.gov/pubmed/24517713" target="_blank"> PMID 24517713</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1186/gb-2014-15-2-r35" data-hide-no-mentions="true">

</div></li>
<li>Cai H, Kumar N, Bagheri HC, von Mering C, <strong>Robinson MD</strong>, Baudis M: Chromothripsis-like patterns are recurring but heterogeneously distributed features in a survey of 22,347 cancer genome screens. BMC Genomics 15:82 (2014). DOI: 10.1186/1471-2164-15-82. <a href="http://www.ncbi.nlm.nih.gov/pubmed/24476156" target="_blank"> PMID 24476156</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1186/1471-2164-15-82" data-hide-no-mentions="true">

</div></li>
</ul>
</div>
<div id="section-5" class="section level2">
<h2>2013</h2>
<ul>
<li>Maier L, Vyas R, Cordova CD, <strong>Lindsay H</strong>, Schmidt TS, Brugiroux S, Periaswamy B, Bauer R, Sturm A, Schreiber F, von Mering C, <strong>Robinson MD</strong>, Stecher B, Hardt WD: Microbiota-derived hydrogen fuels Salmonella typhimurium invasion of the gut ecosystem. Cell Host Microbe 14(6):641-51 (2013). DOI: 10.1016/j.chom.2013.11.002. <a href="http://www.ncbi.nlm.nih.gov/pubmed/24331462" target="_blank"> PMID 24331462</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1016/j.chom.2013.11.002" data-hide-no-mentions="true">

</div></li>
<li>Anders S, McCarthy DJ, Chen Y, Okoniewski M, Smyth GK, Huber W, <strong>Robinson MD</strong>: Count-based differential expression analysis of RNA sequencing data using R and Bioconductor. Nat Protoc 8(9):1765-86 (2013). DOI: 10.1038/nprot.2013.099. <a href="http://www.ncbi.nlm.nih.gov/pubmed/23975260" target="_blank"> PMID 23975260</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1038/nprot.2013.099" data-hide-no-mentions="true">

</div></li>
<li>Omasits U, Quebatte M, Stekhoven DJ, Fortes C, Roschitzki B, <strong>Robinson MD</strong>, Dehio C, Ahrens CH: Directed shotgun proteomics guided by saturated RNA-seq identifies a complete expressed prokaryotic proteome. Genome Res 23(11):1916-27 (2013). DOI: 10.1101/gr.151035.112. <a href="http://www.ncbi.nlm.nih.gov/pubmed/23878158" target="_blank"> PMID 23878158</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1101/gr.151035.112" data-hide-no-mentions="true">

</div></li>
<li><strong>Zhou X</strong>, Oshlack A, <strong>Robinson MD</strong>: miRNA-Seq normalization comparisons need improvement. RNA 19(6):733-4 (2013). DOI: 10.1261/rna.037895.112. <a href="http://www.ncbi.nlm.nih.gov/pubmed/23616640" target="_blank"> PMID 23616640</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1261/rna.037895.112" data-hide-no-mentions="true">

</div></li>
<li>Bert SA, <strong>Robinson MD</strong>, Strbenac D, Statham AL, Song JZ, Hulf T, Sutherland RL, Coolen MW, Stirzaker C, Clark SJ: Regional activation of the cancer genome by long-range epigenetic remodeling. Cancer Cell 23(1):9-22 (2013). DOI: 10.1016/j.ccr.2012.11.006. <a href="http://www.ncbi.nlm.nih.gov/pubmed/23245995" target="_blank"> PMID 23245995</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1016/j.ccr.2012.11.006" data-hide-no-mentions="true">

</div></li>
<li>Menigatti M, Staiano T, Manser CN, Bauerfeind P, Komljenovic A, <strong>Robinson M</strong>, Jiricny J, Buffoli F, Marra G: Epigenetic silencing of monoallelically methylated miRNA loci in precancerous colorectal lesions. Oncogenesis 2:e56 (2013). DOI: 10.1038/oncsis.2013.21. <a href="http://www.ncbi.nlm.nih.gov/pubmed/23857251" target="_blank"> PMID 23857251</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1038/oncsis.2013.21" data-hide-no-mentions="true">

</div></li>
</ul>
</div>
<div id="section-6" class="section level2">
<h2>2012</h2>
<ul>
<li><strong>Robinson MD</strong>, Strbenac D, Stirzaker C, Statham AL, Song J, Speed TP, Clark SJ: Copy-number-aware differential analysis of quantitative DNA sequencing data. Genome Res 22(12):2489-96 (2012). DOI: 10.1101/gr.139055.112. <a href="http://www.ncbi.nlm.nih.gov/pubmed/22879430" target="_blank"> PMID 22879430</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1101/gr.139055.112" data-hide-no-mentions="true">

</div></li>
<li>Fiume M, Smith EJ, Brook A, Strbenac D, Turner B, Mezlini AM, <strong>Robinson MD</strong>, Wodak SJ, Brudno M: Savant Genome Browser 2: visualization and analysis for population-scale genomics. Nucleic Acids Res 40(Web Server issue):W615-21 (2012). DOI: 10.1093/nar/gks427. <a href="http://www.ncbi.nlm.nih.gov/pubmed/22638571" target="_blank"> PMID 22638571</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1093/nar/gks427" data-hide-no-mentions="true">

</div></li>
<li>Statham AL, <strong>Robinson MD</strong>, Song JZ, Coolen MW, Stirzaker C, Clark SJ: Bisulfite sequencing of chromatin immunoprecipitated DNA (BisChIP-seq) directly informs methylation status of histone-modified DNA. Genome Res 22(6):1120-7 (2012). DOI: 10.1101/gr.132076.111. <a href="http://www.ncbi.nlm.nih.gov/pubmed/22466171" target="_blank"> PMID 22466171</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1101/gr.132076.111" data-hide-no-mentions="true">

</div></li>
<li>Valdés-Mora F, Song JZ, Statham AL, Strbenac D, <strong>Robinson MD</strong>, Nair SS, Patterson KI, Tremethick DJ, Stirzaker C, Clark SJ: Acetylation of H2A.Z is a key epigenetic modification associated with gene deregulation and epigenetic remodeling in cancer. Genome Res 22(2):307-21 (2012). DOI: 10.1101/gr.118919.110. <a href="http://www.ncbi.nlm.nih.gov/pubmed/21788347" target="_blank"> PMID 21788347</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1101/gr.118919.110" data-hide-no-mentions="true">

</div></li>
</ul>
</div>
<div id="section-7" class="section level2">
<h2>2011</h2>
<ul>
<li>Siebert S, <strong>Robinson MD</strong>, Tintori SC, Goetz F, Helm RR, Smith SA, Shaner N, Haddock SH, Dunn CW: Differential gene expression in the siphonophore Nanomia bijuga (Cnidaria) assessed with multiple next-generation sequencing workflows. PLoS One 6(7):e22953 (2011). DOI: 10.1371/journal.pone.0022953. <a href="http://www.ncbi.nlm.nih.gov/pubmed/21829563" target="_blank"> PMID 21829563</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1371/journal.pone.0022953" data-hide-no-mentions="true">

</div></li>
<li>Hulf T, Sibbritt T, Wiklund ED, Bert S, Strbenac D, Statham AL, <strong>Robinson MD</strong>, Clark SJ: Discovery pipeline for epigenetically deregulated miRNAs in cancer: integration of primary miRNA transcription. BMC Genomics 12:54 (2011). DOI: 10.1186/1471-2164-12-54. <a href="http://www.ncbi.nlm.nih.gov/pubmed/21255435" target="_blank"> PMID 21255435</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1186/1471-2164-12-54" data-hide-no-mentions="true">

</div></li>
<li>Nair SS, Coolen MW, Stirzaker C, Song JZ, Statham AL, Strbenac D, <strong>Robinson MD</strong>, Clark SJ: Comparison of methyl-DNA immunoprecipitation (MeDIP) and methyl-CpG binding domain (MBD) protein capture for genome-wide DNA methylation analysis reveal CpG sequence coverage bias. Epigenetics 6(1):34-44 (2011). DOI: 10.4161/epi.6.1.13313. <a href="http://www.ncbi.nlm.nih.gov/pubmed/20818161" target="_blank"> PMID 20818161</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.4161/epi.6.1.13313" data-hide-no-mentions="true">

</div></li>
</ul>
</div>
<div id="section-8" class="section level2">
<h2>2010</h2>
<ul>
<li><strong>Robinson MD</strong>, Statham AL, Speed TP, Clark SJ: Protocol matters: which methylome are you actually studying? Epigenomics 2(4):587-98 (2010). DOI: 10.2217/epi.10.36. <a href="http://www.ncbi.nlm.nih.gov/pubmed/21566704" target="_blank"> PMID 21566704</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.2217/epi.10.36" data-hide-no-mentions="true">

</div></li>
<li>Oshlack A, <strong>Robinson MD</strong>, Young MD: From RNA-seq reads to differential expression results. Genome Biol 11(12):220 (2010). DOI: 10.1186/gb-2010-11-12-220. <a href="http://www.ncbi.nlm.nih.gov/pubmed/21176179" target="_blank"> PMID 21176179</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1186/gb-2010-11-12-220" data-hide-no-mentions="true">

</div></li>
<li><strong>Robinson MD</strong>, Stirzaker C, Statham AL, Coolen MW, Song JZ, Nair SS, Strbenac D, Speed TP, Clark SJ: Evaluation of affinity-based genome-wide DNA methylation data: effects of CpG density, amplification bias, and copy number variation. Genome Res 20(12):1719-29 (2010). DOI: 10.1101/gr.110601.110. <a href="http://www.ncbi.nlm.nih.gov/pubmed/21045081" target="_blank"> PMID 21045081</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1101/gr.110601.110" data-hide-no-mentions="true">

</div></li>
<li>Statham AL, Strbenac D, Coolen MW, Stirzaker C, Clark SJ, <strong>Robinson MD</strong>: Repitools: an R package for the analysis of enrichment-based epigenomic data. Bioinformatics 26(13):1662-3 (2010). DOI: 10.1093/bioinformatics/btq247. <a href="http://www.ncbi.nlm.nih.gov/pubmed/20457667" target="_blank"> PMID 20457667</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1093/bioinformatics/btq247" data-hide-no-mentions="true">

</div></li>
<li><strong>Robinson MD</strong>, Oshlack A: A scaling normalization method for differential expression analysis of RNA-seq data. Genome Biol 11(3):R25 (2010). DOI: 10.1186/gb-2010-11-3-r25. <a href="http://www.ncbi.nlm.nih.gov/pubmed/20196867" target="_blank"> PMID 20196867</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1186/gb-2010-11-3-r25" data-hide-no-mentions="true">

</div></li>
<li>Coolen MW, Stirzaker C, Song JZ, Statham AL, Kassir Z, Moreno CS, Young AN, Varma V, Speed TP, Cowley M, Lacaze P, Kaplan W, <strong>Robinson MD</strong>, Clark SJ: Consolidation of the cancer genome into domains of repressive chromatin by long-range epigenetic silencing (LRES) reduces transcriptional plasticity. Nat Cell Biol 12(3):235-46 (2010). DOI: 10.1038/ncb2023. <a href="http://www.ncbi.nlm.nih.gov/pubmed/20173741" target="_blank"> PMID 20173741</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1038/ncb2023" data-hide-no-mentions="true">

</div></li>
<li><strong>Robinson MD</strong>, McCarthy DJ, Smyth GK: edgeR: a Bioconductor package for differential expression analysis of digital gene expression data. Bioinformatics 26(1):139-40 (2010). DOI: 10.1093/bioinformatics/btp616. <a href="http://www.ncbi.nlm.nih.gov/pubmed/19910308" target="_blank"> PMID 19910308</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1093/bioinformatics/btp616" data-hide-no-mentions="true">

</div></li>
</ul>
</div>
<div id="section-9" class="section level2">
<h2>2009</h2>
<ul>
<li><strong>Robinson MD</strong>, Speed TP: Differential splicing using whole-transcript microarrays. BMC Bioinformatics 10:156 (2009). DOI: 10.1186/1471-2105-10-156. <a href="http://www.ncbi.nlm.nih.gov/pubmed/19463149" target="_blank"> PMID 19463149</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1186/1471-2105-10-156" data-hide-no-mentions="true">

</div></li>
</ul>
</div>
<div id="section-10" class="section level2">
<h2>2008</h2>
<ul>
<li><strong>Robinson MD</strong>, Smyth GK: Small-sample estimation of negative binomial dispersion, with applications to SAGE data. Biostatistics 9(2):321-32 (2008). DOI: 10.1093/biostatistics/kxm030. <a href="http://www.ncbi.nlm.nih.gov/pubmed/17728317" target="_blank"> PMID 17728317</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1093/biostatistics/kxm030" data-hide-no-mentions="true">

</div></li>
</ul>
</div>
<div id="section-11" class="section level2">
<h2>2007</h2>
<ul>
<li><strong>Robinson MD</strong>, Speed TP: A comparison of Affymetrix gene expression arrays. BMC Bioinformatics 8:449 (2007). DOI: 10.1186/1471-2105-8-449. <a href="http://www.ncbi.nlm.nih.gov/pubmed/18005448" target="_blank"> PMID 18005448</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1186/1471-2105-8-449" data-hide-no-mentions="true">

</div></li>
<li><strong>Robinson MD</strong>, De Souza DP, Keen WW, Saunders EC, McConville MJ, Speed TP, Likić VA: A dynamic programming approach for the alignment of signal peaks in multiple gas chromatography-mass spectrometry experiments. BMC Bioinformatics 8:419 (2007). DOI: 10.1186/1471-2105-8-419. <a href="http://www.ncbi.nlm.nih.gov/pubmed/17963529" target="_blank"> PMID 17963529</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1186/1471-2105-8-419" data-hide-no-mentions="true">

</div></li>
<li><strong>Robinson MD</strong>, Smyth GK: Moderated statistical tests for assessing differences in tag abundance. Bioinformatics 23(21):2881-7 (2007). DOI: 10.1093/bioinformatics/btm453. <a href="http://www.ncbi.nlm.nih.gov/pubmed/17881408" target="_blank"> PMID 17881408</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1093/bioinformatics/btm453" data-hide-no-mentions="true">

</div></li>
<li>Ewing RM, Chu P, Elisma F, Li H, Taylor P, Climie S, McBroom-Cerajewski L, <strong>Robinson MD</strong>, O’Connor L, Li M, Taylor R, Dharsee M, Ho Y, Heilbut A, Moore L, Zhang S, Ornatsky O, Bukhman YV, Ethier M, Sheng Y, Vasilescu J, Abu-Farha M, Lambert JP, Duewel HS, Stewart II, Kuehl B, Hogue K, Colwill K, Gladwish K, Muskat B, Kinach R, Adams SL, Moran MF, Morin GB, Topaloglou T, Figeys D: Large-scale mapping of human protein-protein interactions by mass spectrometry. Mol Syst Biol 3:89 (2007). DOI: 10.1038/msb4100134. <a href="http://www.ncbi.nlm.nih.gov/pubmed/17353931" target="_blank"> PMID 17353931</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1038/msb4100134" data-hide-no-mentions="true">

</div></li>
</ul>
</div>
<div id="section-12" class="section level2">
<h2>2006</h2>
<ul>
<li>Chua G, Morris QD, Sopko R, <strong>Robinson MD</strong>, Ryan O, Chan ET, Frey BJ, Andrews BJ, Boone C, Hughes TR: Identifying transcription factor functions and targets by phenotypic activation. Proc Natl Acad Sci U S A 103(32):12045-50 (2006). DOI: 10.1073/pnas.0605140103. <a href="http://www.ncbi.nlm.nih.gov/pubmed/16880382" target="_blank"> PMID 16880382</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1073/pnas.0605140103" data-hide-no-mentions="true">

</div></li>
<li>Krogan NJ, Cagney G, Yu H, Zhong G, Guo X, Ignatchenko A, Li J, Pu S, Datta N, Tikuisis AP, Punna T, Peregrín-Alvarez JM, Shales M, Zhang X, Davey M, <strong>Robinson MD</strong>, Paccanaro A, Bray JE, Sheung A, Beattie B, Richards DP, Canadien V, Lalev A, Mena F, Wong P, Starostine A, Canete MM, Vlasblom J, Wu S, Orsi C, Collins SR, Chandran S, Haw R, Rilstone JJ, Gandi K, Thompson NJ, Musso G, St Onge P, Ghanny S, Lam MH, Butland G, Altaf-Ul AM, Kanaya S, Shilatifard A, O’Shea E, Weissman JS, Ingles CJ, Hughes TR, Parkinson J, Gerstein M, Wodak SJ, Emili A, Greenblatt JF: Global landscape of protein complexes in the yeast Saccharomyces cerevisiae. Nature 440(7084):637-43 (2006). DOI: 10.1038/nature04670. <a href="http://www.ncbi.nlm.nih.gov/pubmed/16554755" target="_blank"> PMID 16554755</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1038/nature04670" data-hide-no-mentions="true">

</div></li>
<li>Belov K, Deakin JE, Papenfuss AT, Baker ML, Melman SD, Siddle HV, Gouin N, Goode DL, Sargeant TJ, <strong>Robinson MD</strong>, Wakefield MJ, Mahony S, Cross JG, Benos PV, Samollow PB, Speed TP, Graves JA, Miller RD: Reconstructing an ancestral mammalian immune supercomplex from a marsupial major histocompatibility complex. PLoS Biol 4(3):e46 (2006). DOI: 10.1371/journal.pbio.0040046. <a href="http://www.ncbi.nlm.nih.gov/pubmed/16435885" target="_blank"> PMID 16435885</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1371/journal.pbio.0040046" data-hide-no-mentions="true">

</div></li>
</ul>
</div>
<div id="section-13" class="section level2">
<h2>2005</h2>
<ul>
<li>Davierwala AP, Haynes J, Li Z, Brost RL, <strong>Robinson MD</strong>, Yu L, Mnaimneh S, Ding H, Zhu H, Chen Y, Cheng X, Brown GW, Boone C, Andrews BJ, Hughes TR: The synthetic genetic interaction spectrum of essential genes. Nat Genet 37(10):1147-52 (2005). DOI: 10.1038/ng1640. <a href="http://www.ncbi.nlm.nih.gov/pubmed/16155567" target="_blank"> PMID 16155567</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1038/ng1640" data-hide-no-mentions="true">

</div></li>
<li>Frey BJ, Mohammad N, Morris QD, Zhang W, <strong>Robinson MD</strong>, Mnaimneh S, Chang R, Pan Q, Sat E, Rossant J, Bruneau BG, Aubin JE, Blencowe BJ, Hughes TR: Genome-wide analysis of mouse transcripts using exon microarrays and factor graphs. Nat Genet 37(9):991-6 (2005). DOI: 10.1038/ng1630. <a href="http://www.ncbi.nlm.nih.gov/pubmed/16127451" target="_blank"> PMID 16127451</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1038/ng1630" data-hide-no-mentions="true">

</div></li>
<li>Barrios-Rodiles M, Brown KR, Ozdamar B, Bose R, Liu Z, Donovan RS, Shinjo F, Liu Y, Dembowy J, Taylor IW, Luga V, Przulj N, <strong>Robinson M</strong>, Suzuki H, Hayashizaki Y, Jurisica I, Wrana JL: High-throughput mapping of a dynamic signaling network in mammalian cells. Science 307(5715):1621-5 (2005). DOI: 10.1126/science.1105776. <a href="http://www.ncbi.nlm.nih.gov/pubmed/15761153" target="_blank"> PMID 15761153</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1126/science.1105776" data-hide-no-mentions="true">

</div></li>
</ul>
</div>
<div id="section-14" class="section level2">
<h2>2004</h2>
<ul>
<li>Le Bihan T, <strong>Robinson MD</strong>, Stewart II, Figeys D: Definition and characterization of a “trypsinosome” from specific peptide characteristics by nano-HPLC-MS/MS and in silico analysis of complex protein mixtures. J Proteome Res 3(6):1138-48 (2004). DOI: 10.1021/pr049909x. <a href="http://www.ncbi.nlm.nih.gov/pubmed/15595722" target="_blank"> PMID 15595722</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1021/pr049909x" data-hide-no-mentions="true">

</div></li>
<li>Zhang W, Morris QD, Chang R, Shai O, Bakowski MA, Mitsakakis N, Mohammad N, <strong>Robinson MD</strong>, Zirngibl R, Somogyi E, Laurin N, Eftekharpour E, Sat E, Grigull J, Pan Q, Peng WT, Krogan N, Greenblatt J, Fehlings M, van der Kooy D, Aubin J, Bruneau BG, Rossant J, Blencowe BJ, Frey BJ, Hughes TR: The functional landscape of mouse gene expression. J Biol 3(5):21 (2004). DOI: 10.1186/jbiol16. <a href="http://www.ncbi.nlm.nih.gov/pubmed/15588312" target="_blank"> PMID 15588312</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1186/jbiol16" data-hide-no-mentions="true">

</div></li>
<li>Chua G, <strong>Robinson MD</strong>, Morris Q, Hughes TR: Transcriptional networks: reverse-engineering gene regulation on a global scale. Curr Opin Microbiol 7(6):638-46 (2004). DOI: 10.1016/j.mib.2004.10.009. <a href="http://www.ncbi.nlm.nih.gov/pubmed/15556037" target="_blank"> PMID 15556037</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1016/j.mib.2004.10.009" data-hide-no-mentions="true">

</div></li>
<li>Hughes TR, <strong>Robinson MD</strong>, Mitsakakis N, Johnston M: The promise of functional genomics: completing the encyclopedia of a cell. Curr Opin Microbiol 7(5):546-54 (2004). DOI: 10.1016/j.mib.2004.08.015. <a href="http://www.ncbi.nlm.nih.gov/pubmed/15451511" target="_blank"> PMID 15451511</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1016/j.mib.2004.08.015" data-hide-no-mentions="true">

</div></li>
<li>Grigull J, Mnaimneh S, Pootoolal J, <strong>Robinson MD</strong>, Hughes TR: Genome-wide analysis of mRNA stability using transcription inhibitors and microarrays reveals posttranscriptional control of ribosome biogenesis factors. Mol Cell Biol 24(12):5534-47 (2004). DOI: 10.1128/MCB.24.12.5534-5547.2004. <a href="http://www.ncbi.nlm.nih.gov/pubmed/15169913" target="_blank"> PMID 15169913</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1128/MCB.24.12.5534-5547.2004" data-hide-no-mentions="true">

</div></li>
<li>Krogan NJ, Peng WT, Cagney G, <strong>Robinson MD</strong>, Haw R, Zhong G, Guo X, Zhang X, Canadien V, Richards DP, Beattie BK, Lalev A, Zhang W, Davierwala AP, Mnaimneh S, Starostine A, Tikuisis AP, Grigull J, Datta N, Bray JE, Hughes TR, Emili A, Greenblatt JF: High-definition macromolecular composition of yeast RNA-processing complexes. Mol Cell 13(2):225-39 (2004). DOI: NA. <a href="http://www.ncbi.nlm.nih.gov/pubmed/14759368" target="_blank"> PMID 14759368</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/NA" data-hide-no-mentions="true">

</div></li>
</ul>
</div>
<div id="section-15" class="section level2">
<h2>2003</h2>
<ul>
<li>Peng WT, <strong>Robinson MD</strong>, Mnaimneh S, Krogan NJ, Cagney G, Morris Q, Davierwala AP, Grigull J, Yang X, Zhang W, Mitsakakis N, Ryan OW, Datta N, Jojic V, Pal C, Canadien V, Richards D, Beattie B, Wu LF, Altschuler SJ, Roweis S, Frey BJ, Emili A, Greenblatt JF, Hughes TR: A panoramic view of yeast noncoding RNA processing. Cell 113(7):919-33 (2003). DOI: NA. <a href="http://www.ncbi.nlm.nih.gov/pubmed/12837249" target="_blank"> PMID 12837249</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/NA" data-hide-no-mentions="true">

</div></li>
</ul>
</div>
<div id="section-16" class="section level2">
<h2>2002</h2>
<ul>
<li>Jorgensen P, Nelson B, <strong>Robinson MD</strong>, Chen Y, Andrews B, Tyers M, Boone C: High-resolution genetic mapping with ordered arrays of Saccharomyces cerevisiae deletion mutants. Genetics 162(3):1091-9 (2002). DOI: NA. <a href="http://www.ncbi.nlm.nih.gov/pubmed/12454058" target="_blank"> PMID 12454058</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/NA" data-hide-no-mentions="true">

</div></li>
<li><strong>Robinson MD</strong>, Grigull J, Mohammad N, Hughes TR: FunSpec: a web-based cluster interpreter for yeast. BMC Bioinformatics 3:35 (2002). DOI: NA. <a href="http://www.ncbi.nlm.nih.gov/pubmed/12431279" target="_blank"> PMID 12431279</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/NA" data-hide-no-mentions="true">

</div></li>
<li>Wu LF, Hughes TR, Davierwala AP, <strong>Robinson MD</strong>, Stoughton R, Altschuler SJ: Large-scale prediction of Saccharomyces cerevisiae gene function using overlapping transcriptional clusters. Nat Genet 31(3):255-65 (2002). DOI: 10.1038/ng906. <a href="http://www.ncbi.nlm.nih.gov/pubmed/12089522" target="_blank"> PMID 12089522</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1038/ng906" data-hide-no-mentions="true">

</div></li>
</ul>
</div>
<div id="section-17" class="section level2">
<h2>2001</h2>
<ul>
<li>Tong AH, Evangelista M, Parsons AB, Xu H, Bader GD, Pagé N, <strong>Robinson M</strong>, Raghibizadeh S, Hogue CW, Bussey H, Andrews B, Tyers M, Boone C: Systematic genetic analysis with ordered arrays of yeast deletion mutants. Science 294(5550):2364-8 (2001). DOI: 10.1126/science.1065810. <a href="http://www.ncbi.nlm.nih.gov/pubmed/11743205" target="_blank"> PMID 11743205</a>.
<div class="altmetric-embed" data-badge-popover="right" data-badge-type="4" data-doi="https://doi.org/10.1126/science.1065810" data-hide-no-mentions="true">

</div></li>
</ul>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
