<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />




<title>Research</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/flatly.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-4.5.0/css/font-awesome.min.css" rel="stylesheet" />

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 60px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 65px;
  margin-top: -65px;
}

.section h2 {
  padding-top: 65px;
  margin-top: -65px;
}
.section h3 {
  padding-top: 65px;
  margin-top: -65px;
}
.section h4 {
  padding-top: 65px;
  margin-top: -65px;
}
.section h5 {
  padding-top: 65px;
  margin-top: -65px;
}
.section h6 {
  padding-top: 65px;
  margin-top: -65px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->






<div class="navbar navbar-inverse  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Statistical Bioinformatics @SIB @UZH</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="research.html">
    <span class="fa fa-area-chart"></span>
     
    research
  </a>
</li>
<li>
  <a href="publications.html">
    <span class="fa fa-file-text-o"></span>
     
    publications
  </a>
</li>
<li>
  <a href="tools.html">
    <span class="fa fa-code"></span>
     
    resources
  </a>
</li>
<li>
  <a href="people.html">
    <span class="fa fa-user"></span>
     
    people
  </a>
</li>
<li>
  <a href="teaching.html">
    <span class="fa fa-graduation-cap"></span>
     
    teaching
  </a>
</li>
<li>
  <a href="news.html">
    <span class="fa fa-newspaper-o"></span>
     
    news
  </a>
</li>
<li>
  <a href="more_info.html">
    <span class="fa fa-info-circle"></span>
     
    more info
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Research</h1>

</div>


<p><br></p>
<p><img src="research_files/figure-html/unnamed-chunk-2-1.png" width="672" /></p>
<div id="overall-philosophy-of-interdisciplinary-methods-research" class="section level4">
<h4><strong>Overall philosophy of interdisciplinary methods research</strong></h4>
<p>High-throughput sequencing technologies are revolutionizing the way genomes, transcriptomes and epigenomes are studied. Generating, processing and interpreting this unprecedented magnitude of complex data and integrating multiple dimensions requires considerable research effort, for biologists, statisticians and computer scientists alike. Our group develops statistical methods for interpreting high-throughput sequencing and other genomics technologies in areas such as genome sequencing, gene expression and regulation and analysis of epigenomes. Research projects encompass three interlaced themes:</p>
<ol style="list-style-type: decimal">
<li><p>contribute to relevant genome-scale experimental design and data analysis challenges in close collaboration with biologists;</p></li>
<li><p>develop statistical methods and software tools for the interpretation and integration of multiple types of genome-scale data;</p></li>
<li><p>perform technical investigations to understand the performance of the technologies used to generate and of the algorithms used to process the data.</p></li>
</ol>
<p><br></p>
<!-- ```{r, echo= FALSE, results = "asis"} -->
<!-- # This is to add two blank lines -->
<!-- cat("<br><br>") -->
<!-- ``` -->
<!-- --- -->
<!-- #### **Research area 1** [ [**_Publications on this area_**](file://localhost/Users/ruizhu/Documents/work/robinsonlabuzh.github.io/publications.html)] [ [**_Tools on this area_**](file://localhost/Users/ruizhu/Documents/work/robinsonlabuzh.github.io/tools.html)] -->
<!-- ```{r echo = FALSE} -->
<!-- plot(rpois(100, 50), rpois(100, 25),  -->
<!--      cex = runif(100, min = 0, max = 5),  -->
<!--      col = rgb(0.2, runif(100), 1, 0.5), -->
<!--      xlab = "", ylab = "", pch = 19, -->
<!--      axes = FALSE) -->
<!-- ``` -->
<!-- Description of the project. Description of the project. Description of the project. Description of the project. Description of the project. Description of the project. Description of the project. Description of the project. Description of the project. Description of the project. Description of the project. Description of the project. Description of the project. Description of the project. Description of the project. Description of the project. Description of the project. Description of the project. Description of the project. Description of the project. Description of the project. Description of the project. Description of the project. Description of the project. Description of the project. Description of the project.   -->
<!-- ```{r, echo= FALSE, results = "asis"} -->
<!-- # add one blank line -->
<!-- cat("<br>") -->
<!-- ``` -->
</div>
<div id="our-formula-for-embedded-methodological-research" class="section level4">
<h4><strong>Our formula for embedded methodological research</strong></h4>
<p>Our team of computational scientists is fully immersed in and motivated by biological investigations. Many of us are statisticians, thus it is critical to be involved at the experimental design of these studies. Overall, our focus is on methodology development, but a large amount of our research time is ultimately spent with our collaborators investigating questions, either technical or biological, in a specific dataset. We do not necessarily drive the biological questions, but our expertise drives the method development that is fundamental to answering them.</p>
<p><br></p>
</div>
<div id="open-science-and-reproducibility" class="section level4">
<h4><strong>Open science and reproducibility</strong></h4>
<p>Our group maintains a firm stance on open and reproducible science. The default policy is that we: i) produce documented open source software for our developed statistical and computational frameworks (typically through the Bioconductor project); ii) create code repositories for manuscript analyses; and, iii) post manuscripts as preprints. In many subfields, modern biologists are transitioning to faster, more transparent modes of scientific dissemination and we fully support these initiatives (e.g., ASAPBio).</p>
<p><br></p>
<!-- #### **Research area 3** [ [**_Publications on this area_**](file://localhost/Users/ruizhu/Documents/work/robinsonlabuzh.github.io/publications.html)] [ [**_Tools on this area_**](file://localhost/Users/ruizhu/Documents/work/robinsonlabuzh.github.io/tools.html)] -->
<!-- ```{r, echo= FALSE, results = "asis"} -->
<!-- # create a two column style -->
<!-- l1 <- paste0("<style> \n") -->
<!-- l2 <- paste0("\t", ".col2 {\n") -->
<!-- l3 <- paste0("\t", "\t","columns: 2 200px; \n") -->
<!-- l4 <- paste0("\t", "\t","-webkit-columns: 2 200px; \n") -->
<!-- l5 <- paste0("\t", "\t","-moz-columns: 2 200px;") -->
<!-- l6 <- paste0("\t", "} \n") -->
<!-- l7 <- paste0("</style> \n") -->
<!-- style <- do.call(paste, mget(paste0("l",1:7))) -->
<!-- cat(style) -->
<!-- ``` -->
<!-- ```{r, echo= FALSE, results = "asis"} -->
<!-- # start a division -->
<!-- cat("<div class='col2'>") -->
<!-- ``` -->
<!-- ```{r, echo=FALSE} -->
<!-- # one figure in the left column -->
<!-- plot(rpois(100, 50), rpois(100, 25),  -->
<!--      cex = runif(100, min = 0, max = 5),  -->
<!--      col = rgb(0.2, runif(100), 1, 0.5), -->
<!--      xlab = "", ylab = "", pch = 19, -->
<!--      axes = FALSE) -->
<!-- # text in the right column -->
<!-- ``` -->
<!-- ```{r, echo= FALSE, results = "asis"} -->
<!-- cat("<br>") -->
<!-- ``` -->
<!-- Description of the project. Description of the project. Description of the project. Description of the project. Description of the project. Description of the project. Description of the project. Description of the project. Description of the project. Description of the project. Description of the project. Description of the project. Description of the project. Description of the project. Description of the project. Description of the project. Description of the project. Description of the project. Description of the project. Description of the project. Description of the project. Description of the project. Description of the project. Description of the project. Description of the project. Description of the project.  -->
</div>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
