<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />




<title>Software &amp; Resources</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-4.5.0/css/font-awesome.min.css" rel="stylesheet" />

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}

.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->






<div class="navbar navbar-inverse  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Statistical Genomics @IMLS-UZH // Statistical Bioinformatics @SIB</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">
    <span class="fa fa-home"></span>
     
    Home
  </a>
</li>
<li>
  <a href="about.html">
    <span class="fa fa-user"></span>
     
    About
  </a>
</li>
<li>
  <a href="cv.html">
    <span class="fa fa-file-pdf-o"></span>
     
    CV
  </a>
</li>
<li>
  <a href="https://robinsonlabuzh.github.io/">
    <span class="fa fa-rss"></span>
     
    Blog
  </a>
</li>
<li>
  <a href="resources.html">
    <span class="fa fa-home"></span>
     
    Resources
  </a>
</li>
<li>
  <a href="more_info.html">
    <span class="fa fa-user"></span>
     
    More Info
  </a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="mailto:mark.robinson@imls.uzh.ch">
    <span class="fa fa-envelope-o"></span>
     
    Contact me
  </a>
</li>
<li>
  <a href="https://github.com/robinsonlabuzh">
    <span class="fa fa-github"></span>
     
    GitHub
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Software &amp; Resources</h1>

</div>


<p><img src="stickers/Bioconductor.png" width="94" /><img src="stickers/CATALYST.png" width="94" /><img src="stickers/countsimQC.png" width="94" /><img src="stickers/CrispRVariants.png" width="94" /><img src="stickers/DRIMSeq.png" width="94" /><img src="stickers/edgeR.png" width="94" /><img src="stickers/iCOBRA.png" width="94" /><img src="stickers/regsplice.png" width="94" /></p>
<div id="r-packages" class="section level3">
<h3><strong>R packages</strong></h3>
<table style="width:88%;">
<colgroup>
<col width="15%" />
<col width="22%" />
<col width="50%" />
</colgroup>
<thead>
<tr class="header">
<th>Name</th>
<th>Author(s)</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td><a href="http://bioconductor.org/packages/release/bioc/html/CATALYST.html">CATALYST</a></td>
<td><strong>Helena L Crowell</strong><br><strong>Mark D Robinson</strong><br>Vito RT Zanotelli<br>St&amp;#130phane Chevrier<br>Bernd Bodenmiller</td>
<td>Tools for preprocessing of cytometry data including<br>normalization, debarcoding, and compensation</td>
</tr>
<tr class="even">
<td><a href="http://bioconductor.org/packages/release/bioc/html/CrispRVariants.html">CrispRVariants</a></td>
<td><strong>Helen Lindsay</strong></td>
<td>Tools for counting and visualising mutations in a target location</td>
</tr>
<tr class="odd">
<td><a href="http://bioconductor.org/packages/release/bioc/html/DRIMSeq.html">DRIMSeq</a></td>
<td><strong>Malgorzata Nowicka</strong></td>
<td>Differential transcript usage and tuQTL analyses<br>with Dirichlet-multinomial model in RNA-seq</td>
</tr>
<tr class="even">
<td><a href="http://bioconductor.org/packages/release/bioc/html/edgeR.html">edgeR</a></td>
<td>Yunshun Chen<br>Aaron Lun<br>Davis McCarthy<br><strong>Xiaobei Zhou</strong><br><strong>Mark D Robinson</strong><br>Gordon Smyth</td>
<td>Empirical Analysis of Digital Gene Expression Data in R</td>
</tr>
<tr class="odd">
<td><a href="http://bioconductor.org/packages/release/bioc/html/flagme.html">flagme</a></td>
<td><strong>Mark D Robinson</strong><br>Riccardo Romoli</td>
<td>Analysis of Metabolomics GC/MS Data</td>
</tr>
<tr class="even">
<td><a href="http://bioconductor.org/packages/release/bioc/html/iCOBRA.html">iCOBRA</a></td>
<td><strong>Charlotte Soneson</strong></td>
<td>Comparison and Visualization of Ranking and Assignment Methods</td>
</tr>
<tr class="odd">
<td><a href="http://bioconductor.org/packages/release/bioc/html/regsplice.html">regsplice</a></td>
<td><strong>Lukas M Weber</strong></td>
<td>L1-regularization based methods for detection of differential splicing</td>
</tr>
<tr class="even">
<td><a href="http://bioconductor.org/packages/release/bioc/html/Repitools.html">Repitools</a></td>
<td><strong>Mark D Robinson</strong><br>Dario Strbenac<br>Aaron Statham<br>Andrea Riebler</td>
<td>Epigenomic tools</td>
</tr>
</tbody>
</table>
</div>
<div id="web-applications" class="section level3">
<h3><strong>Web applications</strong></h3>
<table style="width:88%;">
<colgroup>
<col width="15%" />
<col width="22%" />
<col width="50%" />
</colgroup>
<thead>
<tr class="header">
<th>Name</th>
<th>Author(s)</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td><a href="https://catalyst-project.github.io">CATALYST</a></td>
<td><strong>Helena L Crowell</strong></td>
<td>Tools for preprocessing of cytometry data, including<br>normalization, debarcoding, and compensation</td>
</tr>
<tr class="even">
<td><a href="http://imlspenticton.uzh.ch:3838/conquer/">conquer</a></td>
<td><strong>Charlotte Soneson</strong></td>
<td>Repository providing consistently processed<br>public single-cell RNA-seq data sets</td>
</tr>
<tr class="odd">
<td><a href="http://lmweber.github.io/CrispantCal/">CrispantCal</a></td>
<td><strong>Lukas M Weber</strong></td>
<td>R/Shiny web app to calculate injection mix volumes<br>for CRISPR-Cas9 experiments</td>
</tr>
<tr class="even">
<td><a href="http://imlspenticton.uzh.ch:3838/CrispRVariantsLite/">CrispRVariantsLite</a></td>
<td><strong>Helen Lindsay</strong></td>
<td>Graphical user interface to the R/Bioconductor CrispRVariants package</td>
</tr>
</tbody>
</table>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
