<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />




<title>Teaching</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/flatly.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.0.13/css/fa-svg-with-js.css" rel="stylesheet" />
<script src="site_libs/font-awesome-5.0.13/js/fontawesome-all.min.js"></script>
<script src="site_libs/font-awesome-5.0.13/js/fa-v4-shims.min.js"></script>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 60px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 65px;
  margin-top: -65px;
}

.section h2 {
  padding-top: 65px;
  margin-top: -65px;
}
.section h3 {
  padding-top: 65px;
  margin-top: -65px;
}
.section h4 {
  padding-top: 65px;
  margin-top: -65px;
}
.section h5 {
  padding-top: 65px;
  margin-top: -65px;
}
.section h6 {
  padding-top: 65px;
  margin-top: -65px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->






<div class="navbar navbar-inverse  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Statistical Bioinformatics @SIB @UZH</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="research.html">
    <span class="fa fa-area-chart"></span>
     
    research
  </a>
</li>
<li>
  <a href="publications.html">
    <span class="fa fa-file-text-o"></span>
     
    publications
  </a>
</li>
<li>
  <a href="tools.html">
    <span class="fa fa-code"></span>
     
    resources
  </a>
</li>
<li>
  <a href="people.html">
    <span class="fa fa-user"></span>
     
    people
  </a>
</li>
<li>
  <a href="teaching.html">
    <span class="fa fa-graduation-cap"></span>
     
    teaching
  </a>
</li>
<li>
  <a href="news.html">
    <span class="fa fa-newspaper-o"></span>
     
    news
  </a>
</li>
<li>
  <a href="more_info.html">
    <span class="fa fa-info-circle"></span>
     
    more info
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Teaching</h1>

</div>


<div id="courses-offered-at-uzh" class="section level3">
<h3><strong>Courses offered at UZH</strong></h3>
<table style="width:100%;">
<colgroup>
<col width="43%" />
<col width="44%" />
<col width="12%" />
</colgroup>
<thead>
<tr class="header">
<th><em>Lecture</em></th>
<th><em>Description</em></th>
<th><em>Semester</em></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td><strong>Statistical Analysis of High-Thoughput Genomic and Transcriptomic Data - STA426</strong></td>
<td>We will go through the most important statistical concepts employed in today’s High-Thoughput analysis methods, implemented as R packages. The student will propose and perform a project at the end of the course</td>
<td>Fall</td>
</tr>
<tr class="even">
<td><strong>Statistical Modelling - STA121</strong></td>
<td></td>
<td>Fall</td>
</tr>
<tr class="odd">
<td><strong>Introduction to Bionformatics I - BCH401 </strong></td>
<td>In each lecture we give an overview of the main Bioinformatic tools currently used</td>
<td>Fall</td>
</tr>
<tr class="even">
<td><strong>Topics in Evolutionary Biology - BIO554 </strong></td>
<td></td>
<td>Fall</td>
</tr>
<tr class="odd">
<td><strong>Modern Genetics and Genomics - BIO323 </strong></td>
<td>Key concepts and methods in genetics will be taught. You will conduct genetic and molecular studies in model systems and analyse data using bioinformatic tools</td>
<td>Fall</td>
</tr>
<tr class="even">
<td><strong>Cancer Epigenetics - BIO255</strong> (Block Course)</td>
<td>The lab gives a two week practical session along with other 5 different labs. The workload is divided in applying molecular biology techniques and a bioinformatic session</td>
<td>Fall</td>
</tr>
<tr class="odd">
<td><strong>Practical Bionformatics - BIO334</strong> (Block Course)</td>
<td>The lab gives a two week section of this course. We provide the student with the basics on performing straightforward data manipulations and statistical analyses in R</td>
<td>Spring</td>
</tr>
<tr class="even">
<td><strong>Data Analysis in Biology - BIO144</strong></td>
<td>This course is organized by the <a href="https://www.ieu.uzh.ch/en/staff/member/petchey_owen.html">Petchey lab</a>. Our lab provides support in the practical sessions</td>
<td>Spring</td>
</tr>
</tbody>
</table>
<p>For exact information on schedule, registration and location, please refer to the official UZH <a href="https://studentservices.uzh.ch/uzh/anonym/vvz/index.html?sap-language=EN&amp;sap-ui-language=EN">course catalogue</a></p>
</div>
<div id="upcoming-external-courses" class="section level3">
<h3><strong>Upcoming external courses</strong></h3>
</div>
<div id="past-external-courses" class="section level3">
<h3><strong>Past external courses</strong></h3>
</div>
<div id="section" class="section level3">
<h3>2018</h3>
<ul>
<li><a href="http://www-huber.embl.de/csama2018/">CSAMA 2018</a>. Brixen (Italy), July 8-13 2018</li>
<li><a href="https://www.ebi.ac.uk/training/events/2018/advanced-rna-seq-analysis">Advanced RNA-seq analysis</a>. Hinxton (UK), April 16-20 2018</li>
<li><a href="http://www.adaptation.ethz.ch/education/bag-winter-school-2018.html">Winter school on Bioinformatics for Adaptation Genomics:</a> Weggis (Switzerland) from the 11th to the 17th of February 2018</li>
</ul>
</div>
<div id="section-1" class="section level3">
<h3>2017</h3>
<ul>
<li><a href="https://pennekampster.github.io/2017-11-06-lugano/">Data Carpentry Ecology Workshop</a>. Lugano, Switzerland</li>
<li><a href="https://pennekampster.github.io/2017-11-08-lugano/">Data Carpentry Genomics Workshop</a>. Lugano, Switzerland</li>
<li><a href="https://www.denbi.de/22-training-cat/training-courses/278-de-nbi-summer-school-computational-genomics-and-rna-biology">de.NBI Summer School: Computational Genomics and RNA biology</a>. Berlin, Germany</li>
<li><a href="https://www.sib.swiss/training/course/stats-basel2017">Statistics for Life Scientists</a>. Basel, Switzerland</li>
<li><a href="http://www.huber.embl.de/csama2017/#home">Statistical data analysis for genome scale biology (CSAMA 2017)</a>. Brixen, Italy</li>
<li><a href="http://www.ebi.ac.uk/training/events/2017/advanced-rna-seq-and-chip-seq-data-analysis-1">Advanced RNA-seq and ChIP-seq data analysis</a>. EBI, Hinxton, UK</li>
</ul>
</div>
<div id="section-2" class="section level3">
<h3>2016</h3>
<ul>
<li><a href="https://lmweber.github.io/2016-05-30-Zurich/">Data Carpentry: Introduction to Data Analysis and Visualization in R (May 2016)</a>. University of Zurich</li>
<li><a href="https://markrobinsonuzh.github.io/2016-07-18-zurich/">Data Carpentry: Introduction to Data Analysis and Visualization in R (July 2016)</a>. University of Zurich</li>
<li><a href="http://compgen2016.mdc-berlin.de/">Computational Genomics: A hands-on course on data analysis</a>. Max-Delbrück Center for Molecular Medicine, Berlin, Germany</li>
<li>Advanced course on edgeR and DESeq2. Leuven, Belgium</li>
<li><a href="http://edu.isb-sib.ch/course/view.php?id=269">Statistics for Life Scientists</a>. Basel, Switzerland</li>
<li><a href="http://www.uni-regensburg.de/medicine/statistical-bioinformatics/method-course-oct-16/program/index.html">Method course - Analysis of NextGen RNA-seq data for expression profiling and protein-binding RNAs</a>. Regensburg, Germany</li>
<li><a href="http://events.embo.org/16-htp-seq/">Analysis of high-throughput sequencing data</a>. Cambridge, UK</li>
<li><a href="http://www.huber.embl.de/csama2016/about/">Statistics and computing in genome data science (CSAMA 2016)</a>. Brixen, Italy</li>
</ul>
<p>Other teaching activities are listed in members’ personal webpage or cv.</p>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
