---
title: "Software & Resources"
---

<style type="text/css">
#stickers {
  width: 840;
  padding: 60px 0;
  margin: 0 auto;
}
#stickers a {
  display: inline-block;
  height: 140px;
  width: 120px;
  margin-bottom: -36px;
}
#stickers a img {
  width: 120px;
  height: 140px;
}

#stickers a:nth-child(1) {
  margin-left: 60px;
}

@media (min-width: 900px) {
  #stickers {
    width: 840px;
  }
  #stickers a:nth-child(12n+1) {
    margin-left: 60px;
  }
}
@media (max-width: 899px) {
  #stickers a:nth-child(1) {
    margin-left: 30px;
  }  
  #stickers {
    width: 270px;
  }
  #stickers a {
    height: 70px;
    width: 60px;
    margin-bottom: -18px;
  }
  #stickers a img {
    width: 60px;
    height: 70px;
  }
  #stickers a:nth-child(8n+1) {
    margin-left: 30px;
  }
}
</style>

OMNIBENCHMARK FRAMEWORK: [![](https://omnibenchmark.org/img/logo_hu3bfc984ed46fa1b744ae58316fc8051b_18690_300x0_resize_box_3.png){width=15%}](https://omnibenchmark.org)

R PACKAGES:  
<div id="stickers">
[![](stickers/ARMOR.png)](https://github.com/csoneson/ARMOR)[![](stickers/ARPEGGIO.png)](https://github.com/supermaxiste/ARPEGGIO)[![](stickers/BANDITS.png)](https://github.com/SimoneTiberi/BANDITS)[![](stickers/CATALYST.png)](http://bioconductor.org/packages/CATALYST/)[![](stickers/countsimQC.png)](https://bioconductor.org/packages/countsimQC/)[![](stickers/CrispRVariants.png)](http://bioconductor.org/packages/CrispRVariants/)[![](stickers/DAMEfinder.png)](https://github.com/markrobinsonuzh/DAMEfinder)[![](stickers/diffcyt.png)](http://bioconductor.org/packages/diffcyt/)[![](stickers/distinct.png)](https://github.com/SimoneTiberi/distinct)[![](stickers/DRIMSeq.png)](http://bioconductor.org/packages/DRIMSeq/)[![](stickers/edgeR.png)](http://bioconductor.org/packages/edgeR/)[![](stickers/iCOBRA.png)](http://bioconductor.org/packages/iCOBRA/)[![](stickers/iSEE.png)](http://bioconductor.org/packages/iSEE/)[![](stickers/muscat.png)](https://github.com/HelenaLC/muscat)[![](stickers/pipeComp.png)](https://github.com/plger/pipeComp)[![](stickers/regsplice.png)](http://bioconductor.org/packages/regsplice/)[![](stickers/scDblFinder.png)](https://github.com/plger/scDblFinder)[![](https://github.com/Bioconductor/BiocStickers/raw/devel/SpatialExperiment/SpatialExperiment.png)](https://bioconductor.org/packages/SpatialExperiment)[![](stickers/treeclimbR.png)](https://github.com/fionarhuang/treeclimbR)[![](stickers/TreeHeatmap.png)](https://github.com/fionarhuang/TreeHeatmap)[![](stickers/TreeSumExp.png)](https://github.com/fionarhuang/TreeSummarizedExperiment)[![](stickers/DESpace.png)](https://github.com/peicai/DESpace)
</div>


<!-- [![](https://raw.githubusercontent.com/SimoneTiberi/distinct/master/inst/extdata/distinct.png)](https://bioconductor.org/packages/release/bioc/html/distinct.html) -->

### **R packages**

Name      | Author(s)               | Description 
----------|---------------|-----------------------------------
[aroma.affymetrix](http://aroma-project.org/) | Henrik Bengtsson<br>James Bullard<br>Kasper Hansen<br>Pierre Neuvial<br>Elizabeth Purdom<br>**Mark D Robinson**<br>Ken Simpson | An open-source R framework for microarray analysis
[BANDITS](http://bioconductor.org/packages/BANDITS/) | **Simone Tiberi**<br>**Mark D Robinson** | A Bayesian hierarchical model for differential splicing, via differential transcript usage, accounting for sample-to-sample variability and mapping uncertainty
[CATALYST](http://bioconductor.org/packages/CATALYST/) | **Helena L Crowell**<br>**Mark D Robinson**<br>Vito RT Zanotelli<br>Stéphane Chevrier<br>Bernd Bodenmiller | Pipeline for preprocessing of cytometry data<br>& visualization tools for differential analysis
[CellMixS](http://bioconductor.org/packages/release/bioc/html/CellMixS.html) | **Almut Lütge**<br>**Mark D Robinson** | Toolbox to evaluate Cellspecific Mixing Scores (CMS) for different batches in scRNA-seq data
[censcyt](https://bioconductor.org/packages/censcyt) | **Reto Gerber** | censored covariates in differential abundance analysis in cytometry
[countsimQC](https://bioconductor.org/packages/countsimQC/) | **Charlotte Soneson** | Comparison of characteristic features of count data sets
[CrispRVariants](http://bioconductor.org/packages/CrispRVariants/) | **Helen Lindsay** | Tools for counting and visualising mutations in a target location
[cytofWorkflow](https://www.bioconductor.org/packages/release/workflows/html/cytofWorkflow.html) | **Malgorzata Nowicka** | CyTOF workflow: differential discovery in high-throughput high-dimensional cytometry datasets
[DAMEfinder](https://github.com/markrobinsonuzh/DAMEfinder) | **Stephany Orjuela**<br>Dania Machlab<br>**Mark D Robinson** | Method to detect differential allele-specific methylation
[DESpace](https://bioconductor.org/packages/DESpace) | Peiying Cai<br>**Simone Tiberi**<br>**Mark D Robinson** | A framework to discover spatially variable genes via spatial clusters
[diffcyt](http://bioconductor.org/packages/diffcyt/) | **Lukas M Weber** | Statistical methods for differential discovery in high-dimensional cytometry data
[diffUTR](https://github.com/ETHZ-INS/diffUTR) | Stefan Gerber<br/>**Pierre-Luc Germain** | Streamlining differential exon and UTR usage analysis
[distinct](http://bioconductor.org/packages/distinct/) | **Simone Tiberi**<br>**Mark D Robinson** | A statistical method to perform differential analyses between groups of distributions, ideal for applications to high-throughput single-cell data. By comparing full distributions, *distinct* identifies both, differential patterns involving changes in the mean, as well as more subtle variations that do not involve the mean.
[DRIMSeq](http://bioconductor.org/packages/DRIMSeq/) | **Malgorzata Nowicka** | Differential transcript usage and tuQTL analyses<br>with Dirichlet-multinomial model in RNA-seq [Supplementary Page](http://imlspenticton.uzh.ch/robinson_lab/drimseq/)
[edgeR](http://bioconductor.org/packages/edgeR/) | Yunshun Chen<br>Aaron Lun<br>Davis McCarthy<br>**Xiaobei Zhou**<br>**Mark D Robinson**<br>Gordon Smyth | Empirical Analysis of Digital Gene Expression Data in R
[FIRMAGene](https://rdrr.io/rforge/FIRMAGene/) | **Mark D Robinson** | Analysis of differential splicing with Affymetrix Gene ST array data
[flagme](http://bioconductor.org/packages/flagme/) | **Mark D Robinson**<br>Riccardo Romoli | Analysis of Metabolomics GC/MS Data
[iCOBRA](http://bioconductor.org/packages/iCOBRA/) | **Charlotte Soneson** | Comparison and Visualization of Ranking and Assignment Methods
[iSEE](http://bioconductor.org/packages/iSEE/) | **Charlotte Soneson**<br>Aaron Lun<br>Federico Marini<br>Kevin Rue-Albrecht | Interactive SummarizedExperiment Explorer
[muscat](http://github.com/HelenaLC/muscat/) | **Helena L Crowell**<br>**Charlotte Soneson**<br>**Pierre-Luc Germain**<br>**Mark D Robinson** | Multi-sample multi-group scRNA-seq data analysis tools
[pipeComp](http://github.com/plger/pipeComp/) | **Pierre-Luc Germain**<br>**Anthony Sonrel**<br>**Mark D Robinson** | Pipeline benchmarking framework
[regsplice](http://bioconductor.org/packages/regsplice/) | **Lukas M Weber** | L1-regularization based methods for detection of differential splicing
[Repitools](http://bioconductor.org/packages/Repitools/) | **Mark D Robinson**<br>Dario Strbenac<br>Aaron Statham<br>Andrea Riebler | Epigenomic tools (BayMeth)
[scDblFinder](https://github.com/plger/scDblFinder) | **Pierre-Luc Germain** | Identifies doublets in single-cell sequencing data
[SpatialExperiment](https://github.com/drighelli/SpatialExperiment) | **Helena L Crowell** | SpatialExperiment package provides an S4 class for Spatial Omics data handling.
[treeclimbR](https://github.com/fionarhuang/treeclimbR) | **Ruizhu Huang** | Search for the optimal level of a tree to interpret signal by combining with edgeR or DESeq2
[TreeHeatmap](https://github.com/fionarhuang/TreeHeatmap) | **Ruizhu Huang** | Visualize heatmaps at arbitrary levels of a tree.
[TreeSummarizedExperiment](https://github.com/fionarhuang/TreeSummarizedExperiment) | **Ruizhu Huang** | SummarizedExperiment-based object to store data with hierarchical structure
---

### **Data resources**

Name      | Author(s)     | Description
----------|---------------|-----------------------------------
[Benchmark collection](http://imlspenticton.uzh.ch/robinson_lab/benchmark_collection) | **Robinson group** | Collection of high-throughput computational biology benchmarking data sets
[conquer](http://imlspenticton.uzh.ch:3838/conquer/) | **Charlotte Soneson** | Repository providing consistently processed<br>public single-cell RNA-seq data sets
[DuoClustering2018](https://bioconductor.org/packages/DuoClustering2018/) | **Angelo Duò**<br>**Charlotte Soneson** | Data package with processed data and clustering results from [Duò et al. (2018)](https://f1000research.com/articles/7-1141/v2)
[FlowRepository FR-FCM-ZZPH](http://flowrepository.org/id/FR-FCM-ZZPH) | **Lukas M Weber** | Data files for benchmark data sets used in *"Comparison of clustering methods for high-dimensional single-cell flow and mass cytometry data"* ([Weber and Robinson, 2016](https://www.ncbi.nlm.nih.gov/pubmed/27992111))
[FlowRepository FR-FCM-ZYL8](http://flowrepository.org/id/FR-FCM-ZYL8) | **Lukas M Weber** | Data files for benchmark data sets used in our paper introducing the 'diffcyt' framework ([Weber et al., 2018](https://www.biorxiv.org/content/early/2018/06/18/349738))
[HDCytoData](http://bioconductor.org/packages/release/data/experiment/html/HDCytoData.html) | **Lukas M Weber**<br>**Charlotte Soneson** | Data package of high-dimensional cytometry data sets in<br>SummarizedExperiment and flowSet formats
[TabulaMurisData](https://bioconductor.org/packages/TabulaMurisData/) | **Charlotte Soneson** | Bioconductor package providing access to scRNA-seq data from the [Tabula Muris Consortium](http://tabula-muris.ds.czbiohub.org/) 

---

### **Web applications**

Name      | Author(s)     | Description
----------|---------------|-----------------------------------
[CATALYST](https://catalyst-project.github.io) | **Helena L Crowell** | Tools for preprocessing of cytometry data including<br>normalization, debarcoding, and compensation
[CrispantCal](http://lmweber.github.io/CrispantCal/) | **Lukas M Weber** | R/Shiny web app to calculate injection mix volumes for<br>CRISPR-Cas9 experiments
[CrispRVariantsLite](http://imlspenticton.uzh.ch:3838/CrispRVariantsLite/) | **Helen Lindsay** | Graphical user interface to the R/Bioconductor CrispRVariants package
[iCOBRA](http://imlspenticton.uzh.ch:3838/iCOBRA/) | **Charlotte Soneson** | Graphical User interface to the R/Bioconductor iCOBRA package
[pubassistant](https://pubassistant.ch/general_oa_app) | **Reto Gerber** <br> **Mark D Robinson** | Interactive explorer of publications and open access
[scrnaseq_de_evaluation](http://imlspenticton.uzh.ch:3838/scrnaseq_de_evaluation/) | **Charlotte Soneson** |  Interactive browsing and exploration of the scRNA-seq differential expression results from [Soneson and Robinson, 2018](https://www.ncbi.nlm.nih.gov/pubmed/29481549)

---

### **Software & Workflows**

Name      | Author(s)     | Description
----------|---------------|-----------------------------------
[ARMOR](https://github.com/csoneson/ARMOR) | **Stephany Orjuela**<br>**Ruizhu Huang**<br>**Katharina Hembach**<br>**Mark D Robinson**<br>**Charlotte Soneson** |  Snakemake workflow to perform an end-to-end RNA-seq analysis.
[ARPEGGIO](https://github.com/supermaxiste/ARPEGGIO) | **Stefan Milosavljevic**<br>Tony Kuo<br>Samuele Decarli<br>Lucas Mohn<br>Jun Sese<br>Kentaro K. Shimizu<br>Rie Shimizu-Inatsugi<br>**Mark D Robinson** |  SnakeMake workflow to analyse whole genome bisulfite sequencing data from allopolyploids.
[gapless](https://github.com/schmeing/gapless) | **Stephan Schmeing** | Combined scaffolding, gap-closing and assembly correction with long reads.
[ReSeq](https://github.com/schmeing/ReSeq) | **Stephan Schmeing** | More realistic simulator for genomic DNA sequences from Illumina machines that achieves a similar k-mer spectrum as the original sequences.

---

### **Other resources**

Name      | Author(s)     | Description
----------|---------------|-----------------------------------
[ABCD-DNA Supplementary Page](http://imlspenticton.uzh.ch/robinson_lab/ABCD-DNA/) | **Mark D Robinson** |  Supplementary Page with Supplementary Data (semi-processed), R Code for all Figures and analyses ([Robinson et al., 2012](https://www.ncbi.nlm.nih.gov/pubmed/22879430)).
[conquer (build scripts)](https://github.com/markrobinsonuzh/conquer) | **Charlotte Soneson**<br>**Mark D Robinson** |  Repository containing the code used to prepare the conquer (consistent quantification of external RNA-seq data sets) repository ([Soneson and Robinson, 2018](https://www.ncbi.nlm.nih.gov/pubmed/29481549)).
[conquer (differential expression evaluation)](https://github.com/csoneson/conquer_comparison) | **Charlotte Soneson** | Repository containing the code used to perform the evalutaion of differential expression analysis methods in single-cell RNA-seq data ([Soneson and Robinson, 2018](https://www.ncbi.nlm.nih.gov/pubmed/29481549)) [Supplementary Page](http://imlspenticton.uzh.ch/robinson_lab/conquer_de_comparison/)
[CrispRVariants (shiny app code)](https://github.com/markrobinsonuzh/CrispRVariantsLite) | **Helen Lindsay** |  Repository containing the code underlying the CrispRVariantsLite Shiny-based web app ([Lindsay et al, 2016](https://www.ncbi.nlm.nih.gov/pubmed/27404876)).
[cyTOF compensation](https://github.com/BodenmillerGroup/cyTOFcompensation) | Vito RT Zanotelli<br>**Helena L Crowell**<br>**Mark D Robinson** | Repository containing scripts to reproduce a majority of the figures from *"Compensation of Signal Spillover in Suspension and Imaging Mass Cytometry"* ([Chevrier et al., 2018](https://doi.org/10.1016/j.cels.2018.02.010)).
[cytometry-clustering-comparison](https://github.com/lmweber/cytometry-clustering-comparison) | **Lukas M Weber** | Repository of scripts to reproduce and extend analyses and figures in *"Comparison of clustering methods for high-dimensional single-cell flow and mass cytometry data"* ([Weber and Robinson, 2016](https://www.ncbi.nlm.nih.gov/pubmed/27992111))
[diffcyt-evaluations](https://github.com/lmweber/diffcyt-evaluations) | **Lukas M Weber** | Repository of scripts to reproduce evaluations and figures in our paper introducing the 'diffcyt' framework ([Weber et al., 2018](https://www.biorxiv.org/content/early/2018/06/18/349738))
[muscat-comparison](https://github.com/HelenaLC/muscat-comparison/) | **Helena L Crowell** | Repository of scripts to reproduce comparisons and figures for 'muscat' framework ([Crowell et al., 2019](https://www.biorxiv.org/content/10.1101/713412v1))
[Differential splicing comparison](https://github.com/markrobinsonuzh/diff_splice_paper) | **Charlotte Soneson**<br>**Katharina Matthes** |  Repository containing the code for the comparison of various methods for differential isoform usage (differential splicing) detection ([Soneson, Matthes et al., 2016](https://www.ncbi.nlm.nih.gov/pubmed/26813113)) [Supplementary Page](http://imlspenticton.uzh.ch/robinson_lab/splicing_comparison/).
[edgeR robust](http://imlspenticton.uzh.ch/robinson_lab/edgeR_robust/) | **Xiaobei Zhou** |  Supplementary Page with Supplementary Data, R Code for simulations and analyses ([Zhou et al., 2014](https://www.ncbi.nlm.nih.gov/pubmed/24753412)).

