<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Software &amp; Resources</title>

<script src="site_libs/header-attrs-2.29/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/flatly.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-6.5.2/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.5.2/css/v4-shims.min.css" rel="stylesheet" />

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-inverse  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Statistical Bioinformatics @SIB @UZH</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="research.html">
    <span class="fa fa-area-chart"></span>
     
    research
  </a>
</li>
<li>
  <a href="publications.html">
    <span class="fa fa-file-text-o"></span>
     
    publications
  </a>
</li>
<li>
  <a href="tools.html">
    <span class="fa fa-code"></span>
     
    resources
  </a>
</li>
<li>
  <a href="people.html">
    <span class="fa fa-user"></span>
     
    people
  </a>
</li>
<li>
  <a href="teaching.html">
    <span class="fa fa-graduation-cap"></span>
     
    teaching
  </a>
</li>
<li>
  <a href="more_info.html">
    <span class="fa fa-info-circle"></span>
     
    more info
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Software &amp; Resources</h1>

</div>


<style type="text/css">
#stickers {
  width: 840;
  padding: 60px 0;
  margin: 0 auto;
}
#stickers a {
  display: inline-block;
  height: 140px;
  width: 120px;
  margin-bottom: -36px;
}
#stickers a img {
  width: 120px;
  height: 140px;
}

#stickers a:nth-child(1) {
  margin-left: 60px;
}

@media (min-width: 900px) {
  #stickers {
    width: 840px;
  }
  #stickers a:nth-child(12n+1) {
    margin-left: 60px;
  }
}
@media (max-width: 899px) {
  #stickers a:nth-child(1) {
    margin-left: 30px;
  }  
  #stickers {
    width: 270px;
  }
  #stickers a {
    height: 70px;
    width: 60px;
    margin-bottom: -18px;
  }
  #stickers a img {
    width: 60px;
    height: 70px;
  }
  #stickers a:nth-child(8n+1) {
    margin-left: 30px;
  }
}
</style>
<p><a href="https://omnibenchmark.org">Omnibenchmark</a>: <a
href="https://omnibenchmark.org"><img src="stickers/omnibenchmark.png"
style="width:12.0%" /></a></p>
R packages:<br />

<div id="stickers">
<p><a href="https://github.com/csoneson/ARMOR"><img
src="stickers/ARMOR.png" /></a><a
href="https://github.com/supermaxiste/ARPEGGIO"><img
src="stickers/ARPEGGIO.png" /></a><a
href="https://github.com/SimoneTiberi/BANDITS"><img
src="stickers/BANDITS.png" /></a><a
href="http://bioconductor.org/packages/CATALYST/"><img
src="stickers/CATALYST.png" /></a><a
href="https://bioconductor.org/packages/countsimQC/"><img
src="stickers/countsimQC.png" /></a><a
href="http://bioconductor.org/packages/CrispRVariants/"><img
src="stickers/CrispRVariants.png" /></a><a
href="https://github.com/markrobinsonuzh/DAMEfinder"><img
src="stickers/DAMEfinder.png" /></a><a
href="https://github.com/peicai/DESpace"><img
src="stickers/DESpace.png" /></a><a
href="http://bioconductor.org/packages/diffcyt/"><img
src="stickers/diffcyt.png" /></a><a
href="https://github.com/SimoneTiberi/distinct"><img
src="stickers/distinct.png" /></a><a
href="http://bioconductor.org/packages/DRIMSeq/"><img
src="stickers/DRIMSeq.png" /></a><a
href="http://bioconductor.org/packages/edgeR/"><img
src="stickers/edgeR.png" /></a><a
href="http://bioconductor.org/packages/iCOBRA/"><img
src="stickers/iCOBRA.png" /></a><a
href="http://bioconductor.org/packages/iSEE/"><img
src="stickers/iSEE.png" /></a><a
href="https://github.com/HelenaLC/muscat"><img
src="stickers/muscat.png" /></a><a
href="https://github.com/plger/pipeComp"><img
src="stickers/pipeComp.png" /></a><a
href="http://bioconductor.org/packages/regsplice/"><img
src="stickers/regsplice.png" /></a><a
href="https://github.com/plger/scDblFinder"><img
src="stickers/scDblFinder.png" /></a><a
href="https://bioconductor.org/packages/sosta"><img
src="stickers/sosta.png" /></a><a
href="https://bioconductor.org/packages/SpatialExperiment"><img
src="https://github.com/Bioconductor/BiocStickers/raw/devel/SpatialExperiment/SpatialExperiment.png" /></a><a
href="https://bioconductor.org/packages/spatialFDA"><img
src="stickers/spatialFDA.png" /></a><a
href="https://github.com/fionarhuang/treeclimbR"><img
src="stickers/treeclimbR.png" /></a><a
href="https://github.com/fionarhuang/TreeHeatmap"><img
src="stickers/TreeHeatmap.png" /></a><a
href="https://github.com/fionarhuang/TreeSummarizedExperiment"><img
src="stickers/TreeSumExp.png" /></a></p>
</div>
<!-- [![](https://raw.githubusercontent.com/SimoneTiberi/distinct/master/inst/extdata/distinct.png)](https://bioconductor.org/packages/release/bioc/html/distinct.html) -->
<div id="r-packages" class="section level3">
<h3><strong>R packages</strong></h3>
<table>
<colgroup>
<col width="16%" />
<col width="25%" />
<col width="58%" />
</colgroup>
<thead>
<tr class="header">
<th>Name</th>
<th>Author(s)</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td><a href="http://aroma-project.org/">aroma.affymetrix</a></td>
<td>Henrik Bengtsson<br>James Bullard<br>Kasper Hansen<br>Pierre
Neuvial<br>Elizabeth Purdom<br><strong>Mark D Robinson</strong><br>Ken
Simpson</td>
<td>An open-source R framework for microarray analysis</td>
</tr>
<tr class="even">
<td><a href="http://bioconductor.org/packages/BANDITS/">BANDITS</a></td>
<td><strong>Simone Tiberi</strong><br><strong>Mark D
Robinson</strong></td>
<td>A Bayesian hierarchical model for differential splicing, via
differential transcript usage, accounting for sample-to-sample
variability and mapping uncertainty</td>
</tr>
<tr class="odd">
<td><a
href="http://bioconductor.org/packages/CATALYST/">CATALYST</a></td>
<td><strong>Helena L Crowell</strong><br><strong>Mark D
Robinson</strong><br>Vito RT Zanotelli<br>Stéphane Chevrier<br>Bernd
Bodenmiller</td>
<td>Pipeline for preprocessing of cytometry data<br>&amp; visualization
tools for differential analysis</td>
</tr>
<tr class="even">
<td><a
href="http://bioconductor.org/packages/release/bioc/html/CellMixS.html">CellMixS</a></td>
<td><strong>Almut Lütge</strong><br><strong>Mark D
Robinson</strong></td>
<td>Toolbox to evaluate Cellspecific Mixing Scores (CMS) for different
batches in scRNA-seq data</td>
</tr>
<tr class="odd">
<td><a href="https://bioconductor.org/packages/censcyt">censcyt</a></td>
<td><strong>Reto Gerber</strong></td>
<td>censored covariates in differential abundance analysis in
cytometry</td>
</tr>
<tr class="even">
<td><a
href="https://bioconductor.org/packages/countsimQC/">countsimQC</a></td>
<td><strong>Charlotte Soneson</strong></td>
<td>Comparison of characteristic features of count data sets</td>
</tr>
<tr class="odd">
<td><a
href="http://bioconductor.org/packages/CrispRVariants/">CrispRVariants</a></td>
<td><strong>Helen Lindsay</strong></td>
<td>Tools for counting and visualising mutations in a target
location</td>
</tr>
<tr class="even">
<td><a
href="https://www.bioconductor.org/packages/release/workflows/html/cytofWorkflow.html">cytofWorkflow</a></td>
<td><strong>Malgorzata Nowicka</strong></td>
<td>CyTOF workflow: differential discovery in high-throughput
high-dimensional cytometry datasets</td>
</tr>
<tr class="odd">
<td><a
href="https://github.com/markrobinsonuzh/DAMEfinder">DAMEfinder</a></td>
<td><strong>Stephany Orjuela</strong><br>Dania Machlab<br><strong>Mark D
Robinson</strong></td>
<td>Method to detect differential allele-specific methylation</td>
</tr>
<tr class="even">
<td><a href="https://bioconductor.org/packages/DESpace">DESpace</a></td>
<td>Peiying Cai<br><strong>Simone Tiberi</strong><br><strong>Mark D
Robinson</strong></td>
<td>A framework to discover spatially variable genes via spatial
clusters</td>
</tr>
<tr class="odd">
<td><a href="http://bioconductor.org/packages/diffcyt/">diffcyt</a></td>
<td><strong>Lukas M Weber</strong></td>
<td>Statistical methods for differential discovery in high-dimensional
cytometry data</td>
</tr>
<tr class="even">
<td><a href="https://github.com/ETHZ-INS/diffUTR">diffUTR</a></td>
<td>Stefan Gerber<br/><strong>Pierre-Luc Germain</strong></td>
<td>Streamlining differential exon and UTR usage analysis</td>
</tr>
<tr class="odd">
<td><a
href="http://bioconductor.org/packages/distinct/">distinct</a></td>
<td><strong>Simone Tiberi</strong><br><strong>Mark D
Robinson</strong></td>
<td>A statistical method to perform differential analyses between groups
of distributions, ideal for applications to high-throughput single-cell
data. By comparing full distributions, <em>distinct</em> identifies
both, differential patterns involving changes in the mean, as well as
more subtle variations that do not involve the mean.</td>
</tr>
<tr class="even">
<td><a href="http://bioconductor.org/packages/DRIMSeq/">DRIMSeq</a></td>
<td><strong>Malgorzata Nowicka</strong></td>
<td>Differential transcript usage and tuQTL analyses<br>with
Dirichlet-multinomial model in RNA-seq <a
href="http://imlspenticton.uzh.ch/robinson_lab/drimseq/">Supplementary
Page</a></td>
</tr>
<tr class="odd">
<td><a href="http://bioconductor.org/packages/edgeR/">edgeR</a></td>
<td>Yunshun Chen<br>Aaron Lun<br>Davis McCarthy<br><strong>Xiaobei
Zhou</strong><br><strong>Mark D Robinson</strong><br>Gordon Smyth</td>
<td>Empirical Analysis of Digital Gene Expression Data in R</td>
</tr>
<tr class="even">
<td><a href="https://rdrr.io/rforge/FIRMAGene/">FIRMAGene</a></td>
<td><strong>Mark D Robinson</strong></td>
<td>Analysis of differential splicing with Affymetrix Gene ST array
data</td>
</tr>
<tr class="odd">
<td><a href="http://bioconductor.org/packages/flagme/">flagme</a></td>
<td><strong>Mark D Robinson</strong><br>Riccardo Romoli</td>
<td>Analysis of Metabolomics GC/MS Data</td>
</tr>
<tr class="even">
<td><a href="http://bioconductor.org/packages/iCOBRA/">iCOBRA</a></td>
<td><strong>Charlotte Soneson</strong></td>
<td>Comparison and Visualization of Ranking and Assignment Methods</td>
</tr>
<tr class="odd">
<td><a href="http://bioconductor.org/packages/iSEE/">iSEE</a></td>
<td><strong>Charlotte Soneson</strong><br>Aaron Lun<br>Federico
Marini<br>Kevin Rue-Albrecht</td>
<td>Interactive SummarizedExperiment Explorer</td>
</tr>
<tr class="even">
<td><a href="http://github.com/HelenaLC/muscat/">muscat</a></td>
<td><strong>Helena L Crowell</strong><br><strong>Charlotte
Soneson</strong><br><strong>Pierre-Luc Germain</strong><br><strong>Mark
D Robinson</strong></td>
<td>Multi-sample multi-group scRNA-seq data analysis tools</td>
</tr>
<tr class="odd">
<td><a href="http://github.com/plger/pipeComp/">pipeComp</a></td>
<td><strong>Pierre-Luc Germain</strong><br><strong>Anthony
Sonrel</strong><br><strong>Mark D Robinson</strong></td>
<td>Pipeline benchmarking framework</td>
</tr>
<tr class="even">
<td><a
href="http://bioconductor.org/packages/regsplice/">regsplice</a></td>
<td><strong>Lukas M Weber</strong></td>
<td>L1-regularization based methods for detection of differential
splicing</td>
</tr>
<tr class="odd">
<td><a
href="http://bioconductor.org/packages/Repitools/">Repitools</a></td>
<td><strong>Mark D Robinson</strong><br>Dario Strbenac<br>Aaron
Statham<br>Andrea Riebler</td>
<td>Epigenomic tools (BayMeth)</td>
</tr>
<tr class="even">
<td><a href="https://github.com/plger/scDblFinder">scDblFinder</a></td>
<td><strong>Pierre-Luc Germain</strong></td>
<td>Identifies doublets in single-cell sequencing data</td>
</tr>
<tr class="odd">
<td><a href="http://bioconductor.org/packages/sosta/">sosta</a></td>
<td><strong>Samuel Gunz</strong> <br><strong>Mark D
Robinson</strong></td>
<td>Analysis of anatomical tissue structures in spatial omics data</td>
</tr>
<tr class="even">
<td><a
href="http://bioconductor.org/packages/spatialFDA/">spatialFDA</a></td>
<td><strong>Martin Emons</strong><br><strong>Samuel
Gunz</strong><br>Fabian Scheipl<br><strong>Mark D Robinson</strong></td>
<td>Spatial multi-sample co-localisation analysis with spatial
statistics</td>
</tr>
<tr class="odd">
<td><a
href="https://github.com/drighelli/SpatialExperiment">SpatialExperiment</a></td>
<td><strong>Helena L Crowell</strong></td>
<td>SpatialExperiment package provides an S4 class for Spatial Omics
data handling.</td>
</tr>
<tr class="even">
<td><a
href="https://github.com/fionarhuang/treeclimbR">treeclimbR</a></td>
<td><strong>Ruizhu Huang</strong></td>
<td>Search for the optimal level of a tree to interpret signal by
combining with edgeR or DESeq2</td>
</tr>
<tr class="odd">
<td><a
href="https://github.com/fionarhuang/TreeHeatmap">TreeHeatmap</a></td>
<td><strong>Ruizhu Huang</strong></td>
<td>Visualize heatmaps at arbitrary levels of a tree.</td>
</tr>
<tr class="even">
<td><a
href="https://github.com/fionarhuang/TreeSummarizedExperiment">TreeSummarizedExperiment</a></td>
<td><strong>Ruizhu Huang</strong></td>
<td>SummarizedExperiment-based object to store data with hierarchical
structure</td>
</tr>
</tbody>
</table>
<hr />
</div>
<div id="data-resources" class="section level3">
<h3><strong>Data resources</strong></h3>
<table>
<colgroup>
<col width="16%" />
<col width="25%" />
<col width="58%" />
</colgroup>
<thead>
<tr class="header">
<th>Name</th>
<th>Author(s)</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td><a
href="http://imlspenticton.uzh.ch/robinson_lab/benchmark_collection">Benchmark
collection</a></td>
<td><strong>Robinson group</strong></td>
<td>Collection of high-throughput computational biology benchmarking
data sets</td>
</tr>
<tr class="even">
<td><a href="http://imlspenticton.uzh.ch:3838/conquer/">conquer</a></td>
<td><strong>Charlotte Soneson</strong></td>
<td>Repository providing consistently processed<br>public single-cell
RNA-seq data sets</td>
</tr>
<tr class="odd">
<td><a
href="https://bioconductor.org/packages/DuoClustering2018/">DuoClustering2018</a></td>
<td><strong>Angelo Duò</strong><br><strong>Charlotte
Soneson</strong></td>
<td>Data package with processed data and clustering results from <a
href="https://f1000research.com/articles/7-1141/v2">Duò et
al. (2018)</a></td>
</tr>
<tr class="even">
<td><a href="http://flowrepository.org/id/FR-FCM-ZZPH">FlowRepository
FR-FCM-ZZPH</a></td>
<td><strong>Lukas M Weber</strong></td>
<td>Data files for benchmark data sets used in <em>“Comparison of
clustering methods for high-dimensional single-cell flow and mass
cytometry data”</em> (<a
href="https://www.ncbi.nlm.nih.gov/pubmed/27992111">Weber and Robinson,
2016</a>)</td>
</tr>
<tr class="odd">
<td><a href="http://flowrepository.org/id/FR-FCM-ZYL8">FlowRepository
FR-FCM-ZYL8</a></td>
<td><strong>Lukas M Weber</strong></td>
<td>Data files for benchmark data sets used in our paper introducing the
‘diffcyt’ framework (<a
href="https://www.biorxiv.org/content/early/2018/06/18/349738">Weber et
al., 2018</a>)</td>
</tr>
<tr class="even">
<td><a
href="http://bioconductor.org/packages/release/data/experiment/html/HDCytoData.html">HDCytoData</a></td>
<td><strong>Lukas M Weber</strong><br><strong>Charlotte
Soneson</strong></td>
<td>Data package of high-dimensional cytometry data sets
in<br>SummarizedExperiment and flowSet formats</td>
</tr>
<tr class="odd">
<td><a
href="https://bioconductor.org/packages/TabulaMurisData/">TabulaMurisData</a></td>
<td><strong>Charlotte Soneson</strong></td>
<td>Bioconductor package providing access to scRNA-seq data from the <a
href="http://tabula-muris.ds.czbiohub.org/">Tabula Muris
Consortium</a></td>
</tr>
</tbody>
</table>
<hr />
</div>
<div id="web-applications" class="section level3">
<h3><strong>Web applications</strong></h3>
<table>
<colgroup>
<col width="16%" />
<col width="25%" />
<col width="58%" />
</colgroup>
<thead>
<tr class="header">
<th>Name</th>
<th>Author(s)</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td><a href="https://catalyst-project.github.io">CATALYST</a></td>
<td><strong>Helena L Crowell</strong></td>
<td>Tools for preprocessing of cytometry data
including<br>normalization, debarcoding, and compensation</td>
</tr>
<tr class="even">
<td><a href="http://lmweber.github.io/CrispantCal/">CrispantCal</a></td>
<td><strong>Lukas M Weber</strong></td>
<td>R/Shiny web app to calculate injection mix volumes
for<br>CRISPR-Cas9 experiments</td>
</tr>
<tr class="odd">
<td><a
href="http://imlspenticton.uzh.ch:3838/CrispRVariantsLite/">CrispRVariantsLite</a></td>
<td><strong>Helen Lindsay</strong></td>
<td>Graphical user interface to the R/Bioconductor CrispRVariants
package</td>
</tr>
<tr class="even">
<td><a href="http://imlspenticton.uzh.ch:3838/iCOBRA/">iCOBRA</a></td>
<td><strong>Charlotte Soneson</strong></td>
<td>Graphical User interface to the R/Bioconductor iCOBRA package</td>
</tr>
<tr class="odd">
<td><a
href="https://robinsonlabuzh.github.io/pasta/00-home.html">pasta</a></td>
<td><strong>Martin Emons</strong><br><strong>Samuel
Gunz</strong><br><strong>Helena Crowell</strong><br>Malte
Kuehl<br><strong>Izaskun Mallona</strong><br>Reinhard
Furrer<br><strong>Mark D. Robinson</strong></td>
<td>Webpage of vignettes on spatial statistics for spatial omics data in
<code>R</code> and <code>python</code></td>
</tr>
<tr class="even">
<td><a
href="https://pubassistant.ch/general_oa_app">pubassistant</a></td>
<td><strong>Reto Gerber</strong> <br> <strong>Mark D
Robinson</strong></td>
<td>Interactive explorer of publications and open access</td>
</tr>
<tr class="odd">
<td><a
href="http://imlspenticton.uzh.ch:3838/scrnaseq_de_evaluation/">scrnaseq_de_evaluation</a></td>
<td><strong>Charlotte Soneson</strong></td>
<td>Interactive browsing and exploration of the scRNA-seq differential
expression results from <a
href="https://www.ncbi.nlm.nih.gov/pubmed/29481549">Soneson and
Robinson, 2018</a></td>
</tr>
</tbody>
</table>
<hr />
</div>
<div id="software-workflows" class="section level3">
<h3><strong>Software &amp; Workflows</strong></h3>
<table>
<colgroup>
<col width="16%" />
<col width="25%" />
<col width="58%" />
</colgroup>
<thead>
<tr class="header">
<th>Name</th>
<th>Author(s)</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td><a href="https://github.com/csoneson/ARMOR">ARMOR</a></td>
<td><strong>Stephany Orjuela</strong><br><strong>Ruizhu
Huang</strong><br><strong>Katharina Hembach</strong><br><strong>Mark D
Robinson</strong><br><strong>Charlotte Soneson</strong></td>
<td>Snakemake workflow to perform an end-to-end RNA-seq analysis.</td>
</tr>
<tr class="even">
<td><a href="https://github.com/supermaxiste/ARPEGGIO">ARPEGGIO</a></td>
<td><strong>Stefan Milosavljevic</strong><br>Tony Kuo<br>Samuele
Decarli<br>Lucas Mohn<br>Jun Sese<br>Kentaro K. Shimizu<br>Rie
Shimizu-Inatsugi<br><strong>Mark D Robinson</strong></td>
<td>SnakeMake workflow to analyse whole genome bisulfite sequencing data
from allopolyploids.</td>
</tr>
<tr class="odd">
<td><a href="https://github.com/schmeing/gapless">gapless</a></td>
<td><strong>Stephan Schmeing</strong></td>
<td>Combined scaffolding, gap-closing and assembly correction with long
reads.</td>
</tr>
<tr class="even">
<td><a href="https://github.com/schmeing/ReSeq">ReSeq</a></td>
<td><strong>Stephan Schmeing</strong></td>
<td>More realistic simulator for genomic DNA sequences from Illumina
machines that achieves a similar k-mer spectrum as the original
sequences.</td>
</tr>
</tbody>
</table>
<hr />
</div>
<div id="other-resources" class="section level3">
<h3><strong>Other resources</strong></h3>
<table>
<colgroup>
<col width="16%" />
<col width="25%" />
<col width="58%" />
</colgroup>
<thead>
<tr class="header">
<th>Name</th>
<th>Author(s)</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td><a
href="http://imlspenticton.uzh.ch/robinson_lab/ABCD-DNA/">ABCD-DNA
Supplementary Page</a></td>
<td><strong>Mark D Robinson</strong></td>
<td>Supplementary Page with Supplementary Data (semi-processed), R Code
for all Figures and analyses (<a
href="https://www.ncbi.nlm.nih.gov/pubmed/22879430">Robinson et al.,
2012</a>).</td>
</tr>
<tr class="even">
<td><a href="https://github.com/markrobinsonuzh/conquer">conquer (build
scripts)</a></td>
<td><strong>Charlotte Soneson</strong><br><strong>Mark D
Robinson</strong></td>
<td>Repository containing the code used to prepare the conquer
(consistent quantification of external RNA-seq data sets) repository (<a
href="https://www.ncbi.nlm.nih.gov/pubmed/29481549">Soneson and
Robinson, 2018</a>).</td>
</tr>
<tr class="odd">
<td><a href="https://github.com/csoneson/conquer_comparison">conquer
(differential expression evaluation)</a></td>
<td><strong>Charlotte Soneson</strong></td>
<td>Repository containing the code used to perform the evalutaion of
differential expression analysis methods in single-cell RNA-seq data (<a
href="https://www.ncbi.nlm.nih.gov/pubmed/29481549">Soneson and
Robinson, 2018</a>) <a
href="http://imlspenticton.uzh.ch/robinson_lab/conquer_de_comparison/">Supplementary
Page</a></td>
</tr>
<tr class="even">
<td><a
href="https://github.com/markrobinsonuzh/CrispRVariantsLite">CrispRVariants
(shiny app code)</a></td>
<td><strong>Helen Lindsay</strong></td>
<td>Repository containing the code underlying the CrispRVariantsLite
Shiny-based web app (<a
href="https://www.ncbi.nlm.nih.gov/pubmed/27404876">Lindsay et al,
2016</a>).</td>
</tr>
<tr class="odd">
<td><a
href="https://github.com/BodenmillerGroup/cyTOFcompensation">cyTOF
compensation</a></td>
<td>Vito RT Zanotelli<br><strong>Helena L
Crowell</strong><br><strong>Mark D Robinson</strong></td>
<td>Repository containing scripts to reproduce a majority of the figures
from <em>“Compensation of Signal Spillover in Suspension and Imaging
Mass Cytometry”</em> (<a
href="https://doi.org/10.1016/j.cels.2018.02.010">Chevrier et al.,
2018</a>).</td>
</tr>
<tr class="even">
<td><a
href="https://github.com/lmweber/cytometry-clustering-comparison">cytometry-clustering-comparison</a></td>
<td><strong>Lukas M Weber</strong></td>
<td>Repository of scripts to reproduce and extend analyses and figures
in <em>“Comparison of clustering methods for high-dimensional
single-cell flow and mass cytometry data”</em> (<a
href="https://www.ncbi.nlm.nih.gov/pubmed/27992111">Weber and Robinson,
2016</a>)</td>
</tr>
<tr class="odd">
<td><a
href="https://github.com/lmweber/diffcyt-evaluations">diffcyt-evaluations</a></td>
<td><strong>Lukas M Weber</strong></td>
<td>Repository of scripts to reproduce evaluations and figures in our
paper introducing the ‘diffcyt’ framework (<a
href="https://www.biorxiv.org/content/early/2018/06/18/349738">Weber et
al., 2018</a>)</td>
</tr>
<tr class="even">
<td><a
href="https://github.com/HelenaLC/muscat-comparison/">muscat-comparison</a></td>
<td><strong>Helena L Crowell</strong></td>
<td>Repository of scripts to reproduce comparisons and figures for
‘muscat’ framework (<a
href="https://www.biorxiv.org/content/10.1101/713412v1">Crowell et al.,
2019</a>)</td>
</tr>
<tr class="odd">
<td><a
href="https://github.com/markrobinsonuzh/diff_splice_paper">Differential
splicing comparison</a></td>
<td><strong>Charlotte Soneson</strong><br><strong>Katharina
Matthes</strong></td>
<td>Repository containing the code for the comparison of various methods
for differential isoform usage (differential splicing) detection (<a
href="https://www.ncbi.nlm.nih.gov/pubmed/26813113">Soneson, Matthes et
al., 2016</a>) <a
href="http://imlspenticton.uzh.ch/robinson_lab/splicing_comparison/">Supplementary
Page</a>.</td>
</tr>
<tr class="even">
<td><a
href="http://imlspenticton.uzh.ch/robinson_lab/edgeR_robust/">edgeR
robust</a></td>
<td><strong>Xiaobei Zhou</strong></td>
<td>Supplementary Page with Supplementary Data, R Code for simulations
and analyses (<a
href="https://www.ncbi.nlm.nih.gov/pubmed/24753412">Zhou et al.,
2014</a>).</td>
</tr>
</tbody>
</table>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
